---
title: "Study pre-registration: ArHa - A systematic review of rodent-associated Arenaviruses and Hantaviruses and analysis of global sampling effort and taxanomic biases."
author:
  - name: "David Simons"
    url: https://www.dsimons.org
    orcid: 0000-0001-9655-1656
    affiliations:
    - id: psu
      name: Pennsylvania State University
      city: State College
      state: PA
      country: USA
      url: https://www.psu.edu
    corresponding: true
  - name: "Ricardo Rivero"
    url: https://scholar.google.com/citations?hl=es&user=tz01w90AAAAJ&view_op=list_works&sortby=pubdate
    orcid: 0000-0003-2284-3335
    affiliations:
    - id: wsu
      name: Washington State University
      city: Spokane
      state: WA
      country: USA
      url: https://www.wsu.edu
  - name: "Sagan Friant"
    url: https://www.saganfriant.com/
    orcid: 0000-0003-1664-5180
    affiliations:
      - ref: psu
  - name: "Stephanie Seifert"
    url: https://labs.wsu.edu/mezap/
    orcid: 0000-0002-4397-6156
    affiliations:
      - ref: wsu
date: "2023-10-30"
format: 
  html: 
    toc: true
    toc-location: left
    code-fold: true
    code-summary: "Code"
    page-layout: full
crossref:
  fig-title: '**Figure**'
categories:
  - Arenavirus
  - Hantavirus
---

## Background

Rodents are global hosts of zoonotic pathogens and potential hosts of novel pathogens of epidemic potential [ecke, han]. Existing efforts to catalogue host-pathogen associations in these species are limited by global datasets which lack temporal and geographic specificity [ref]. Current research is hindered by spatial-, host taxa- and temporal biases within these datasets that are challenging to quantify. 

Two viral taxa, Arenaviridae and Hantaviridae include both zoonoses and non-zoonotic rodent associated viruses. **Describe differences and similarities between the genomic structure of these viral families.**

Recombination is dampened in negative sense RNA viruses relative to positive sense RNA viruses which undergo homologous recombination. The segmented genomic architecture of Bunyavirales allows for reassortment of viral genomic segments that facilitates rapid acquisition of mutations that may contribute to host breadth, replication dynamics, transmission and stability, or pathogenesis. Viral reassortment is driven by both ecological factors (e.g. host overlap) and molecular compatibility (e.g. coinfection of a single cell with two virus particles, functional compatibility of reassorted segments). Using a Bayesian stochastic search variable selection and GLM framework implemented in BEAST v2, we will determine the relative contributions of ecological, geographic, and molecular factors contributing to cross-species transmission and reassortment in rodent-associated arenaviruses and hantaviruses. We will perform co-phylogenetic analysis on available pathogen and host sequences to investigate adaptation of these viruses to rodent hosts.

These pathogens have a global distribution with members of these viral families found everywhere their rodent and other small-mammal hosts have been detected. Medically important Arenviridae include *Lassa mammarenavirus*, the causative agent of Lassa fever, *Junin* and *LCMV*. These Arenaviridae have been detected in multiple rodent and non-rodent small mammal species (i.e., *Soricidae spp.*), however, the contribution of these other species to viral maintenance, transmission and spillover into human populations is not well understood. Geographic and taxanomic sampling biases limits an understanding of the geographically heterogeneous hazard to human populations in endemic regions. 

Similarly to Arenaviridae, Hantaviridae are widely distributed in rodent and other small-mammal hosts. Medically important hantaviridae include Puumala virus, Seoul virus and Sin Nombre virus.

These pathogens are considered to have strong co-evolutionary history with their rodent and small-mammal hosts which may affect their geographic distribution and potential for cross species transmission. Whether these co-evolutionary histories are shared across all species within each of these viral families is not known.

To investigate whether strict co-evolutionary history is consistent across Arenaviridae and Hantaviridae species we constructed a dataset of global sampling effort for these viral species in rodent and other small-mammal hosts. We use this constructed dataset to enrich currently available sequence data stored in GenBank to assess the evidence for co-evolution of viral and host species for these two important rodent-associated zoonoses pathogens.

The database produced will provide a novel and accessible resource to explore a range of questions at a global scale about rodents and their pathogens. The produced data will be deposited within the Global Biodiversity Information Facility to support wider re-use as they continue to expand their GBIF health programme.

## Research questions

### Primary research questions

How many arenaviridae and hantaviridae have been detected in small-mammal hosts globally?

Is there evidence of spatial, temporal or taxanomic biases in current sampling for arenaviridae or hantaviridae in small-mammal hosts?

### Secondary research questions

Is there evidence for host-pathogen co-evolution in arenaviruses or hantaviruses that have been well sampled?

Does incorporating sampling biases in phylogenetic analysis improve models of pathogen host-switching or radiation?

### Expectations/Hypotheses

It is expected that arenaviruses and hantaviruses will be detected in all regions where sampling has been conducted.

It is expected that sampling effort will be greatest in areas with known zoonotic arenaviruses and hantaviruses and in areas where host species distributions overlap areas with high human population density.

## Study design

### Setting and population of interest

Meta-analysis - A systematic review of published studies

Blinding - No blinding is involved in this study. However, for the subset of citations and articles double screened against inclusion and exclusion criteria, individuals will be blinded to decisions of other team members. Similarly, for the subset of included articles where data extraction is repeated, individuals will be blinded to the data extracted by other team members.

### Search strategy

Databases - PubMed, Web of Science, BioOne, bioRxiv, ecoevoRxiv, medRxiv

Grey literature - 

Inclusion criteria

1. Include data on rodent or shrew species detection with associated geographic locations of detections
2. Include data on the detection of acute infection with Arenaviridae or Hantaviridae in these species OR detection of antibodies to Arenaviridae or Hantaviridae

Exclusion criteria

1. Does not include information that can be used to infer the location of detection of rodent or shrew species and their associated pathogens

Query strings

1. Rodent* OR (Shrew OR Soricid*)
2. Arenaviridae
3. Hantaviridae
4. 2 AND 3
5. 1 AND 4

Variables of interest

General study descriptives:
+ Authors
+ Year of publication
+ Study title
+ Publication
+ Countries surveyed


Procedures to contact authors

De-duplication

Screening

Screening reliability and reconciliation

Software

### Data extraction

Entities to extract

Extraction reliability and reconciliation

Data management and sharing

### Synthesis and quality assessment

Planned data transformations

Missing data

Data validation

Quality assessment

### Analysis plan

Descriptive analysis

Measures, including specification of predictors, outcome variables, moderators, mediators and covariates

Statistical analysis

Inference criteria

Exploratory analysis

Limitations

## Other

Funding

Conflicts of interest

Ethical approval