---
title: "Study protocol: Production of Arenavirus and Hantavirus host-pathogen database."
author:
  - name: "David Simons"
  - name: "Ricardo Rivero"
  - name: "Harry Gordon"
  - name: "Anna Martinez-Checa"
  - name: "David Redding"
  - name: "Stephanie Seifert"
date: "`r Sys.Date()`"
format: 
  html: 
    toc: true
    toc-location: left
    code-fold: true
    code-summary: "Code"
    page-layout: full
crossref:
  fig-title: '**Figure**'
categories:
  - Arenavirus
  - Hantavirus
bibliography: references.bib
---

``` {r setup, include = FALSE}

if (!require("pacman")) install.packages("pacman")

pkgs =
  c("caret",
    "colorspace",
    "here",
    "htmltools",
    "leaflet",
    "plotly",
    "sf",
    "tidyverse"
  )

pacman::p_load(pkgs, character.only = T)

```

Target journals:    

  1. Scientific Data unclear if they accept protocols but may be worth a try?   
  2. Gigabyte?    
  3. Open Research Europe/F1000 or similar

# Motivation

Current host-pathogen association datasets provide synthesised information on hosts and their pathogens but do not contain temporal or geographic information. These resources often provide linking information to publications reporting the association but information including accession numbers of archived sequences, number of individuals tested and measures of prevalence among sampled populations are not immediately retrievable. Using these resources for inference beyond host-pathogen associations is therefore limited. Here, we aim to produce a database of host-pathogen associations for two viral families of small mammals which contain several known zoonoses, namely Arenaviridae and Hantaviridae. This database can be used to explore the distribution of small mammal hosts of known and suspected pathogens and the extent to which they have been sampled. Further, linkage to sequence data of known zoonoses will support analysis of the risk of viral reassortment between viral species within geographically co-located host species.

# Introduction

Arenaviruses and Hantaviruses are globally distributed pathogens primarily infecting rodents and other small mammals [@childs1993ecology; @jonsson2010global]. Several of these pathogens sporadically spillover into human populations where they are responsible for substantial morbidity and mortality within their endemic regions [@smith2024health; @vapalahti2003hantavirus]. Arenaviruses, including *Mammarenavirus lassaense* (LASV) and *Mammarenavirus juninense* (JUNV), cause viral haemorrhagic fevers such as Lassa fever in West Africa and Argentine haemorrhagic fever [@gibb2017understanding; @gallo2022virus]. In contrast, Mammarenavirus choriomeningitidis has a global distribution and causes Lymphocytic choriomeningitis, an aseptic meningitis [@vilibic2021lymphocytic]. Hantaviruses, such as *Orthohantavirus puumalaense* (PUUV) and *Orthohantavirus seoulense* (SEOV) cause haemorrhagic fever with renal syndrome, while in the Americas *Orthohantavirus sinnombreense* (SNV) is the causative agent of hantavirus pulmonary syndrome [@makary2010disease; @park2019epidemiologic; @goodfellow2021tracing]. The distribution and human health impact of hantaviruses in Africa is poorly understood [@kang2014expanded]. 

Novel Arenaviruses and Hantaviruses continue to be described, with new species evolving and emerging in the Anthropocene [@yanagihara2014hantaviruses; @mcmahon2018ecosystem]. Deforestation, urbanization, and agricultural expansion bring humans into closer contact with rodent reservoirs, increasing the likelihood of spillover events [@plowright2017pathways; @dearing2010ecology]. Changes in climate, land use, rodent ecology, and human behaviour alter human-reservoir contact, which is a component of a complex human-animal-environment nexus driving spillover and maintenance of novel pathogens [@keesing2010impacts; @ecke2022population]. Although these pathogens remain endemic diseases with limited human-to-human transmission, there is concern about the potential emergence of novel strains that could sustain human-to-human transmission, leading to LASVâ€™s classification as a priority disease by the WHO [ref].

A broader understanding of the economic and societal impacts of these pathogens is also required. The economic burden of arenavirus and hantavirus infections extends beyond healthcare costs to include long-term productivity losses and the economic strain on already vulnerable populations [@smith2024health]. The Yosemite National Park outbreak of HPS led to significant economic disruption in affected communities, demonstrating the potentially far-reaching consequences of these diseases [@bach2017social]. Lasting impacts on an individuals health and wellbeing following infection may also occur. While infection with LASV is only presumed to be symptomatic in 20% of infections, the true scale of symptomatic disease or long term sequelae are unknown, for example sensorineural hearing defects [@ficenec2020lassa]. Difficulty in diagnosing acute arenavirus and hantavirus infections likely means most cases are not detected through passive disease surveillance, likely leading to substantial under-reporting in endemic regions.

Small mammals, primarily rodents (Rodentia), but also shrews (Soricidae) are reservoirs of zoonotic arenaviruses and hantaviruses [@jonsson2010global]. Traditionally these pathogens, particularly hantaviruses have been considered single reservoir species but this paradigm is changing with increased pathogen surveillance in endemic settings [@reusken2013factors]. Multiple potential hosts have been identified for pathogens including LASV, SEOV and SNV [@simons2023rodent; @clement2019wild; @goodfellow2021tracing]. Similarly, a single host species may be a host of multiple arenaviruses and hantaviruses throughout its geographic distribution, for example *Mastomys natalensis*, the primary host of LASV is the host of at least 7 distinct arenaviruses throughout its range [@de2020dhati].

The ecology of these hosts, including their population dynamics, behaviour, habitat preferences, and interactions within their communities, directly influences pathogen transmission dynamics [@voutilainen2016temporal]. Linking ecological and genomic data allows for the identification of genetic markers associated with increased virulence or transmissibility, providing critical insights into the evolutionary trajectories of arenaviruses and hantaviruses [@kozakiewicz2018pathogens; @napflin2019genomics]. Small-mammal sampling to quantify these parameters, is vital to better understand ecological factors driving pathogen spillover [ref]. Existing data on rodent-pathogen associations from these studies is typically at a local scale and often lack comprehensive geographic and temporal information which hampers research into the full scope of host-pathogen interactions which requires synthesised and standardised data sources [@simons2023rodent]. 

Additionally, human disease data is often reported at national or subnational scales, to preserve patient anonymity,  making it challenging to directly link ecological findings with human infection risk. A unified dataset that includes detailed spatial and temporal data could serve as a critical tool for connecting local ecological data with broader-scale public health patterns, enhancing the ability to identify hotspots of transmission risk and inform targeted interventions [ref].

Research efforts have also been unevenly distributed, with high-risk but understudied regions often lacking sufficient sampling intensity, particularly in Africa and Southeast Asia [ref]. Identifying these undersampled areas is essential for guiding future viral discovery and research prioritisation.  This requires a comprehensive understanding of global sampling efforts within and between different host and pathogen taxa, helping to pinpoint gaps in knowledge and inform future research priorities.

Local-scale, high-intensity research can provide valuable insights into host-pathogen dynamics, but data fragmentation limits the ability to connect this high-resolution information to broader-scale patterns (i.e., human infection) [ref]. This impairs efforts to identify larger-scale trends that could inform public health strategies and disease management. For example, a better linkage of ecological and genomic data could facilitate a better understanding of the drivers of viral evolution, reassortment, and the emergence of novel pathogens with zoonotic potential. A unified dataset could enable better predictive modelling and outbreak response strategies, addressing current data fragmentation and improving risk assessments and public health interventions.

A detailed dataset on the current state of sampling for arenaviruses and hantaviruses in wild caught small-mammals is therefore urgently needed. Synthesising spatial, temporal and genomic data from previously published small-mammal sampling studies will provide the most comprehensive index of host-pathogen associations for these viral families in addition to identifying locations and potential hosts that have been relatively undersampled and which may benefit from targeted viral discovery. This dataset is expected to provide a key resource for the development of targeted public health interventions, risk mapping and predictive modelling of future outbreaks.

Following Open Science principles, we will share research tools, data extraction methods, and processing code on suitable platforms, ensuring adherence to the FAIR principles (Findable, Accessible, Interoperable, Reusable). The dataset will be accessible to researchers, policymakers, and public health officials worldwide, fostering evidence-based decision-making and international collaboration in disease surveillance. It will be submitted to a suitable data repository (e.g., the Global Biodiversity Information Facility) [@wilkinson2016fair; @gbif].

# Method

## Search strategy

We searched NCBI PubMed and Clarivate Web of Science. We did not use any limits for publication date, language or geographic locations.

| Search term | PubMed | Web of Science |
|--|--|--|
| 1. rodent* | 175,084 | 175,258 |
| 2. shrew | - | - |
| 3. arenavir* | 2,506 | 1,941 |
| 4. hantavir* | 4,274 | 5,016 |
| 5. 1 OR 2 | - | - |
| 6. 2 OR 3 | 6,689 | 6,809 |
| 7. 1 AND 4 | 1,956 | 2,064 |

Citations returned by the searches were downloaded and de-duplicated by matching on titles, authors, publication identifiers or digital object identifiers. The initial search (conducted 2023-10-16) resulted in 2,755 unique publications. These citations were uploaded to the Rayyan platform to assess against inclusion and exclusion criteria [ref].

## Study screening

We screened studies against the following inclusion criteria. Studies were included if they reported:

  1. Rodent OR Shrew sampling AND
  2. Direct or indirect detection of microorganisms in the Arenaviridae and Hantaviridae families AND
  3. Information on the sampling location of small mammals

Studies were excluded if they did not contain information on the host species of the sample for direct or indirect detection of Arenaviridae or Hantaviridae. Studies reporting experimental infections or solely human infections were also excluded. Studies reported as abstract or conference proceedings were also excluded if they did not provide sufficient description of the method of animal sampling or microorganism detection.
  
Direct detection was defined as PCR or culture of virus and indirect detection was defined as detection of antibodies against an Arenaviridae or Hantaviridae.

We first screened study title and abstracts, for manuscripts not published in English we used the search datasets translation of titles and abstracts to assess against the inclusion and exclusion criteria. Assessments were made by two members of the study team, studies deemed to meet inclusion criteria by at least one member proceeded to full text review. For articles published in languages other than English automated translation services (i.e., Google Translate) were used prior to assessing the full text manuscript for inclusion.

Reference chaining was performed on studies considered for inclusion at the full text screening stage and relevant publications were added as manually identified relevant articles. Figure 1. shows the PRISMA flow chart for the search process. We intend to re-run the search when data extraction from the current round has been completed.

## Data extraction

We aimed to extract included study meta-data and to produce three linked datasets that could used to assess sampling for a) small-mammal hosts, b) viral prevalence and c) genetic variability. We have developed and refined the current data extraction tools through data extraction from 10% of the studies included at title and abstract stage (815 articles). Progress towards final dataset development will be shared through a dedicated `RShiny` web-application [ref].

### Included studies

Included study meta-data was abstracted to ensure appropriate attribution for the underlying data that would be extracted from the study. Each included study was provided an internal study_id unique identifier which provides linkage to the full text manuscript citation. We extracted reported sampling effort, where available. For articles not reporting sampling effort at the study level we inferred total sampling effort from individual trapping session effort. Studies not reporting any measure of sampling effort we recorded a value of not reported. Finally, we record the level of data aggregation of rodent or pathogen sampling, classified as individual (data were provided at the level of individual small-mammals) or summarised (data were aggregated, e.g., at site or sampling session level).

| Column name | Description |
| ---- | :-------- |
| study_id | An internal unique identifier for the included study |
| full_text_id | The internal ID for the full text manuscript and its associated citation |
| datasetName | The title of the manuscript, report or book section |
| sampling_effort | A free text entry to capture the effort of sampling, ideally in trap nights for rodent studies |
| data_access | Whether data is available for individual small mammals or whether it is aggregated |
| data_resolution | The level of aggregation available |
| linked_manuscripts | The DOI or weblink to other studies including the same dataset either in its entirety or a subset, this will be used to attempt to de-duplicate data |
| notes | Details that may be important for interpreting the extracted data |

: Study information extraction sheet {#tbl-description}

### Small-mammal sampling

Data is extracted into a `rodent` sheet. A record is produced at the highest level of temporal or spatial resolution for detected small-mammals. For example a study reporting on the detection of a single species at four spatially distinct sampling sites would be associated with four records. If this same study reported detections from four distinct trapping sessions at each of these sites there would be 16 records associated with the study. Studies presenting data on *N* individual rodents will have *N* records with each record associated with the detection of a single rodent. 

Identification of a detected small mammal was recorded as reported within the article, we will not systematically assess the method of identification and assume accurate identification as reported within the articles. Where identification is to genus or higher taxanomic level we report to this level (i.e., *Rattus spp.*, Rodentia). See data processing below for the method used to harmonise of species names.

Included studies were expected to variably report location data associated with sampling. We extract locality as the highest resolution of location data that could be matched to administrative levels within a country (e.g., city, county), verbatimLocality is used where higher resolution spatial data is available which may not map to administrative levels (e.g., Site A). 

| Column name | Description |
| ---- | :-------- |
| rodent_record_id | An internal unique identifier for the rodent record, the resolution of a record is dependent on the level of aggregation reported in the study |
| study_id | A unique identifier to link a record to the study from which it originated |
| scientificName | The species name of the small-mammal as reported in the study, ideally binomial names |
| eventDate | The period in which sampling was conducted |
| locality | The location of sampling effort, reported to the highest spatial resolution available |
| country | Country where trapping occurred, for multinational studies where counts are not disaggregated by country, all countries sampled are reported |
| verbatimLocality | High level habitat type will be recorded here at the scale for which trapping is recorded, particularly useful when locality does not discriminate between multiple sampling sites |
| coordinate_resolution | The description of coordinate levels provided in the study |
| decimalLatitude | Latitude in decimal format, converted from coordinates reported as required |
| decimalLongitude | Longitude in decimal format, converted from coordinates reported as required |
| individualCount | The number of detected individuals associated with a record |
| trapEffort | Trap effort, recorded in trap nights associated with a record |
| trapEffortResolution | The resolution of trapping effort for the record |

: Rodent information extraction sheet {#tbl-rodent}

### Pathogen sampling

Assays conducted for arenaviruses and hantavrisues are extracted in the `pathogen` sheet. A record is produced at the highest resolution of pathogen sampling. Each pathogen record is associated with a rodent record, one-to-many matching at this stage is possible. For example, a single rodent may be tested for a pathogen using both antibody and PCR based assays, hence a single sample may be associated with multiple pathogen records. 

The species or family targeted by the assay was recorded as reported within the relevant article, we did not make any assessment of the suitability, sensitivity or specificity of an assay. We recorded the number of samples tested using the assay for a specific pathogen. This may differ from the individual count of the small-mammals sampled in the associated record as not all samples may not have been suitable for testing, or the study authors may have decided to subset available samples. The number of positive and negative samples are related to this tested number. Where reported we extracted the number of inconclusive samples. Similarly to above, we recorded positives and negatives as reported by study authors and make no assessment of these classifications.

| Column name | Description |
| ---- | :-------- |
| pathogen_record_id | A unique identifier for the group of samples from the same rodent species, at a specific location or timepoint, tested for the same pathogen using the same method |
| associated_rodent_record_id | |
| study_id | A unique identifier to link a study to the `descriptive` sheet entry for that study |
| associatedTaxa | The scientificName of the rodent from the associated_rodent_record_id |
| family | The family of the pathogen, either Arenaviridae or Hantaviridae |
| scientificName | The species name of the pathogen assayed for, if available. Some assays are not specific to a species. |
| assay | Whether the assay is attempting to detect antibody, direct detection of virus (i.e. pcr), or other |
| number_tested | The number of distinct samples tested |
| number_negative | The number of reported negative samples |
| number_positive | The number of reported positive samples |
| number_inconclusive | The number of samples with inconclusive results |
| note | Notes relevant to this record |

: Pathogen information extraction sheet {#tbl-pathogen}

### Sequences

If studies include complete or partial sequences of hosts or viruses archived in NCBI they will be linked through the `sequences` sheet. A record will be produced for each accession available. A sequence_record_id will be associated with each accession and depending on whether the sequence relates to a pathogen or host each record will be associated with one or both of these. Many-to-one matching of sequences may occur for several reasons, first, hantaviruses and arenaviruses contain multiple segments and so three or two records respectively will be produced for each acutely infected small mammal. Second, reporting of pathogen sampling may be aggregated with multiple sequences obtained from a single reported assay. Pathogen sequences from human infections will be extracted but will only be linked at study level.

| Column name | Description |
| ---- | :-------- |
| sequence_record_id | A unique identifier for the sequence record |
| sequenceType | One of Pathogen or Host |
| associated_pathogen_record_id | An associated pathogen record for this sequence |
| associated_rodent_record_id | An associated rodent record for this sequence |
| study_id | Study ID associated with this sequence |
| associatedTaxa | Species name of small mammal sampled/sequenced |
| scientificName | Species name of pathogen sequenced |
| accession_number | NCBI nucleotide accession |
| note | Notes relevant to this record |

: Pathogen sequences extraction sheet {#tbl-sequence}

We are not integrating NCBI with our dataset beyond incorporating the NCBI accession. We will develop a method to integrate the enhanced metadata produced with current metadata available in the GenBank NCBI format.

## Data processing

We describe the processes in place to clean and harmonise extracted data below. These processes have been developed on a pilot sample of 10% of included articles. All data processing will be conducted in `R` with scripts retained in a version controlled git repository [ref]. Raw data will be downloaded from Google Sheets using the `googledrive` API in `R`, with date stamped files stored locally [ref].

### Species name harmonisation

Small mammal and rodent taxonomy has and will continue to change over time. To standardise reported species we will match reported names to both the GBIF backbone taxonomy and the NCBI taxonomy database [ref]. The `taxize` `R` package is used to query the APIs of these platforms [ref]. Unmatched names are retained as reported by study authors, corrected to a matching name or allocated to a higher taxa (e.g., "*Mus/Nannomys*" becoming identified at the genus level as *Mus*). 

### Location of sampling

For studies not reporting geographic coordinates of sampling but including some information that describes the location (i.e., the name of the village sampled) we will locate coordinates through several methods. Locations will be searched for in Google Maps, Wikipedia or the  Geographic Names Server provided by the National Geospatial-Intelligence Agency USA. Locations that represent administrative divisions will be matched using the Database of Global Administrative Areas (GADM) accessed through the `geodata` `R` package [ref].

The coordinate resolution for coordinates labelled, site or study site will be set as 100 meters. For locations associated with an adminstrative area but no higher resolution data we will use the radius of the administrative region to represent uncertainty in these coordinates.

### Imputed non-detections

Non-detection of a small mammal species in a location it may be expected to be is of interest to researchers. We will enrich detection only data from included studies by imputing non-detections where they are not given. We restrict this imputation to species that have been detected elsewhere in the study. Imputed non-detections will be labelled as imputed in the final data product.

# Discussion

This novel dataset provides spatially and temporally explicit data on small mammal sampling for arenaviruses and hantaviruses, offering a more comprehensive view of host-pathogen associations than is currently available. By including sampling effort and explicit spatial data, this dataset enhances the ability to quantify sampling biases, assess geographic gaps, and better understand the spatial ecology of these pathogens. The insights gained from this dataset could improve our understanding of how environmental changes, such as habitat fragmentation and urbanisation, influence pathogen dynamics and spillover risks.

Importantly, this resource could serve as a foundational tool for predictive modelling, helping identify areas at heightened risk for emerging zoonosis spillover. The ability to link ecological data with human health outcomes could inform targeted public health interventions, enhancing outbreak preparedness and response. For instance, identifying hotspots of high pathogen prevalence in rodents could guide targeted development of ecologically-based rodent control measures in regions identified as vulnerable.

Existing host-pathogen datasets lack detailed spatial or temporal information, limiting their use in ecological and epidemiological modelling [ref]. By addressing these gaps, our dataset provides information about host-pathogen interactions that span multiple spatial and temporal scales. Moreover, the explicit reporting of sampling effort allows for more robust analyses of detection probability, which is crucial for understanding the true distribution of pathogens [ref]. Current global host-pathogen datasets are unable to account for sampling biases due to their method of synthesis.

This dataset also offers the unique benefit of explicitly reporting the extent of sampling for pathogens within their host species, providing critical insights into spatial sampling biases and detection efforts. For instance, while it is expected that Mastomys natalensis sampling occurs throughout its range, the detection of specific pathogens such as Lassa virus (LASV) or Morogoro virus may be confined to only a subset of that range. By detailing both presence and absence data, the dataset allows for a more nuanced understanding of pathogen distribution within host species, beyond mere occurrence records. This capability to quantify the detection effort for specific pathogens is vital for assessing spatial sampling biases, identifying under-sampled regions, and refining ecological and epidemiological models.

Despite its strengths, this dataset has several limitations. First, the inherent variability in the quality and detail of reported data from included studies means that some records may lack critical information, such as exact coordinates or specific sampling dates. The imputation of non-detections, introduces assumptions that could affect the interpretation of absence data. Additionally, the dataset is inherently static and will become outdated as new data emerge.

Furthermore, the reliance on published literature introduces publication bias, as studies with significant findings are more likely to be published than those with negative results. This could skew the dataset towards areas and hosts with known or previously detected pathogens, potentially underrepresenting regions where sampling has occurred but without positive detections. We are not routinely contacting study authors to obtain data reported within publications. If data are not available within the article or as supplementary material we are not incorporating it into this dataset. An associated project is currently being planned where we hope to contact individual study authors to obtain, standardise and support submission to available repositories. To address limitations associated with extracting data without access to the raw data we encourage scientists with ongoing field studies and pathogen surveillance programmes, to submit their data to dynamic repositories. 

Developing predictive models based on this dataset could provide valuable insights into the factors driving zoonotic spillover, enabling targeted surveillance and control measures. Such models could also be used to forecast the emergence of novel strains with epidemic potential, guiding resource allocation and intervention strategies. Beyond research applications, the dataset could serve as a tool for policymakers, helping to identify priority areas for viral discovery and public health interventions.

# Conclusion

Overall, this dataset will be a valuable resource for understanding the ecology of arenaviruses and hantaviruses in their natural reservoirs. By bridging the gap between local-scale ecological studies and broader public health needs, it has the potential to enhance our ability to predict and mitigate the risks posed by these emerging pathogens. Continued efforts to update and expand this resource will be crucial for maintaining its utility in a rapidly changing epidemiological landscape.

# References

---

### Zoonosis status

Finally, an additional sheet `known_zoonoses` will be produced containing all of the viral species sampled and whether they are known to cause disease in humans.

| Column name | Description |
| ---- | :-------- |
| pathogen_id | A unique identifier for the pathogen species |
| pathogen_family | The family of pathogen |
| pathogen_species | The species of pathogen |
| known_zoonosis | A logical statement of pathogenicity among humans |
| disease_name | The disease name caused by the viral species, if multiple diseases they are entered with a comma separator |
| icd_10 | The ICD-10 associated name of the disease (if known) |
| disease_reference | A DOI of a publication that supports the `known_zoonosis` statement |

: Zoonosis status extraction sheet {#tbl-sequence}

### Rodent sampling

An example of a map produce from rodent sampling data included in the initial 9 studies is shown below.

```{r leaflet-rodents, warning = FALSE}
#| fig-height: 8
#| label: rodent-map
#| fig-cap: An interactive map displaying the location of detected small species in included studies. Selecting points will expand the number of data points at those coordinates. Point colour indicates small mammal genus with size of the point varying by the number of individuals detected. Hovering over a point or selecting it will show the species name, the number detected and the time period surveyed at the location. Data is currently shown for 9 studies.
# 
# combined_data <- read_rds(here("data", "raw_data", paste0(search_date, "_data.rds")))
# 
# project_crs <- "EPSG:4326"
# 
# rodent_locations <- combined_data$rodent %>%
#   select(genus, species, latitude, longitude, number) %>%
#   mutate(number = as.numeric(number)) %>%
#   drop_na(number) %>%
#   st_as_sf(coords = c("longitude", "latitude"), crs = project_crs) %>%
#   group_by(geometry, species) %>%
#   mutate(number = sum(number)) %>%
#   ungroup() %>%
#   distinct() %>%
#   rowwise() %>%
#   mutate(labels = HTML(paste0("Species: ", species, "<br>", "Number detected: ", number)))
# 
# pal <- colorFactor(diverging_hcl(n = length(sort(unique(rodent_locations$genus))), palette = "Berlin"),
#                    domain = sort(unique(rodent_locations$genus)))
# 
# leaflet(data = rodent_locations) %>%
#   addTiles() %>%
#   addCircleMarkers(radius = ~ifelse(sqrt(number) == 0, 1, sqrt(number)),
#                    color = ~pal(genus),
#                    stroke = FALSE,
#                    fillOpacity = 0.8,
#                    clusterOptions = markerClusterOptions(spiderfyOnMaxZoom = TRUE,
#                                                          spiderLegPolylineOptions = list(length = 6),
#                                                          spiderfyDistanceMultiplier = 2),
#                    label = ~labels,
#                    popup = ~labels) %>%
#   addLegend("bottomright", pal = pal, values = ~genus, title = "Host genus", opacity = 0.8)

```


### Pathogen detection

A similar map can be produced to map pathogen detection with separate layers for acute infection (i.e. PCR positive samples) and evidence of prior infection (i.e. antibody positive samples)

``` {r leaflet-pathogens, warning = FALSE}
#| label: pathogen-map
#| fig-cap: An interactive map showing the locations of pathogen sampling. Selecting clustered points will expand the data. Each individual circle represents a single host species pathogen combination at that sampling location. The number of samples tested is associated with circle size. Selecting the point will display information about the host-pathogen association, including number of samples tested and number of positive samples. A white border of the circle designates host-pathogen associations with at least one positive result. The colour of the circle indicates the pathogen species (where known). Two layers are available, for viral detection and antibody detection.


# pathogen_locations <- combined_data$pathogen %>%
#   select(detection_method, pathogen_family, pathogen_species, host_species, latitude, longitude, number_tested, number_positive) %>%
#   mutate(number_tested = as.numeric(number_tested),
#          number_positive = as.numeric(number_positive)) %>%
#   drop_na(number_tested, number_positive) %>%
#   st_as_sf(coords = c("longitude", "latitude"), crs = project_crs) %>%
#   group_by(geometry, detection_method, pathogen_family, pathogen_species, host_species) %>%
#   mutate(number_tested = sum(number_tested),
#          number_positive = sum(number_positive)) %>%
#   ungroup() %>%
#   distinct() %>%
#   mutate(percent_positive = case_when(is.nan(number_positive/number_tested * 100) ~ "NA",
#                                       number_positive == 0 ~ "NA",
#                                       number_positive/number_tested * 100 < 1 ~ "< 1%",
#                                       TRUE ~ paste0(as.character(round(number_positive/number_tested * 100, 2)), "%")),
#          detection_method = case_when(detection_method == "pcr" ~ "Viral",
#                                       detection_method == "antibody" ~ "Antibody"),
#          detection = case_when(number_positive >= 1 ~ "Detected",
#                                TRUE ~ "Not detected")) %>%
#   rowwise() %>%
#   mutate(labels = HTML(paste0("Pathogen family: ", pathogen_family, "<br>",
#                               "Pathogen species: ", pathogen_species, "<br>",
#                               "Host species: ", host_species, "<br>",
#                               "Number tested: ", number_tested, "<br>",
#                               "Number positive (%): ", number_positive, " (", percent_positive, ")", "<br>")))
# 
# 
# path_pal <- colorFactor(diverging_hcl(n = length(unique(pathogen_locations$pathogen_species)), palette = "Berlin"),
#                         domain = sort(unique(pathogen_locations$pathogen_species)))
# stroke_pal <- colorFactor(palette = c("white", "gray"), levels = c("Detected", "Not detected"))
# 
# leaflet(data = pathogen_locations) %>%
#   addTiles() %>%
#   addCircleMarkers(data = pathogen_locations %>%
#                      filter(detection_method == "Viral") %>%
#                      filter(number_tested != 0),
#                    radius = ~ifelse(sqrt(number_tested) == 0, 1, sqrt(number_tested)),
#                    fill = TRUE,
#                    fillColor = ~path_pal(pathogen_species),
#                    stroke = TRUE,
#                    color = ~stroke_pal(detection),
#                    weight = 2,
#                    opacity = 1,
#                    fillOpacity = 0.8,
#                    clusterOptions = markerClusterOptions(spiderfyOnMaxZoom = TRUE,
#                                                          spiderLegPolylineOptions = list(length = 6),
#                                                          spiderfyDistanceMultiplier = 2),
#                    label = ~labels,
#                    popup = ~labels,
#                    group = "Viral detection") %>%
#   addCircleMarkers(data = pathogen_locations %>%
#                      filter(detection_method == "Antibody") %>%
#                      filter(number_tested != 0),
#                    radius = ~ifelse(sqrt(number_tested) == 0, 1, sqrt(number_tested)),
#                    fill = TRUE,
#                    fillColor = ~path_pal(pathogen_species),
#                    stroke = TRUE,
#                    color = ~stroke_pal(detection),
#                    weight = 2,
#                    opacity = 1,
#                    fillOpacity = 0.8,
#                    clusterOptions = markerClusterOptions(spiderfyOnMaxZoom = TRUE,
#                                                          spiderLegPolylineOptions = list(length = 6),
#                                                          spiderfyDistanceMultiplier = 2),
#                    label = ~labels,
#                    popup = ~labels,
#                    group = "Antibody detection") %>%
#   addLayersControl(overlayGroups = c("Viral detection", "Antibody detection"),
#                    options = layersControlOptions(collapsed = FALSE)) %>%
#   addLegend("bottomright", pal = path_pal, values = ~pathogen_species, title = "Pathogen species", opacity = 0.8)

```

### Host pathogen associations

``` {r  plotly-hp, warning = FALSE}
#| fig-width: 9
#| fig-height: 12
#| label: hp-associations
#| fig-cap: An interactive plot displaying observed host pathogen associations. Facets are produced for antibody based assays and viral detection assays. The rodent species are listed in alphabetical order on the y-axis, detected pathogens are displayed in alphabetical order on the x-axis. Purple tiles represent host-pathogen associations that were not observed for that detection type, yellow tiles represent observed host-pathogen associations. The strength of the colour (the alpha) is scaled to the number of assays performed for that host-pathogen pair. A black line surrounding the tile indicates the pathogen as a known zoonosis. Hovering over the relavent tile will highlight the number of assays and the number of positive samples.

# pathogen_summary <- combined_data$pathogen %>%
#   select(genus = host_genus,
#          species = host_species,
#          pathogen_family, pathogen_species,
#          number_tested, number_positive,
#          detection_method) %>%
#   mutate(number_tested = as.numeric(number_tested)) %>%
#   filter(number_tested >= 1) %>%
#   left_join(combined_data$zoonoses %>%
#               select(pathogen_species, known_zoonosis),
#             by = c("pathogen_species"))
# 
# hp_associations <- combined_data$rodent %>%
#   select(genus, species) %>%
#   left_join(pathogen_summary %>%
#               group_by(detection_method, species, pathogen_species, known_zoonosis) %>%
#               mutate(number_tested = sum(number_tested),
#                      number_positive = sum(number_positive)), by = c("genus", "species")) %>%
#   drop_na(number_tested) %>%
#   distinct() %>%
#   filter(!pathogen_species %in% c("Multiple", "Puumala virus/Saaremaa virus")) %>%
#   arrange(pathogen_family, pathogen_species, genus, species) %>%
#   mutate(association = case_when(number_positive >= 1 ~ TRUE,
#                                  TRUE ~ FALSE),
#          detection_method = case_when(detection_method == "pcr" ~ "Viral",
#                                       detection_method == "antibody" ~ "Antibody"))
# 
# scaled_n <- preProcess(as.data.frame(log10(hp_associations$number_tested)), method = "range")
# hp_associations$alpha <- predict(scaled_n, as.data.frame(log10(hp_associations$number_tested)))[, 1]
# 
# host_pathogen_plot <- hp_associations %>%
#   ggplot(aes(x = pathogen_species, y = species)) +
#   geom_tile(aes(fill = association, colour = known_zoonosis, alpha = alpha,
#                 text = paste0("Pathogen species: ", pathogen_species, "<br>",
#                               "Host species: ", species, "<br>",
#                               "Number tested: ", number_tested, "<br>",
#                               "Number positive: ", number_positive, "<br>",
#                               "Known zoonosis: ", known_zoonosis))) +
#   scale_fill_viridis_d() +
#   scale_colour_manual(values = c("white", "black")) +
#   facet_wrap(~ detection_method) +
#   guides(alpha = "none",
#          colour = "none") +
#   labs(fill = "H-P association",
#        x = element_blank(),
#        y = element_blank()) +
#   theme_bw() +
#   theme(axis.text.x = element_text(angle = 45, vjust = 1, hjust = 1))
# 
# ggplotly(host_pathogen_plot, tooltip = "text")

```