<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.2.247">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="David Simons">
<meta name="dcterms.date" content="2023-01-10">

<title>Study protocol: Production of Arenavirus and Hantavirus host-pathogen database.</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1.6em;
  vertical-align: middle;
}
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>


<script src="protocol_files/libs/clipboard/clipboard.min.js"></script>
<script src="protocol_files/libs/quarto-html/quarto.js"></script>
<script src="protocol_files/libs/quarto-html/popper.min.js"></script>
<script src="protocol_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="protocol_files/libs/quarto-html/anchor.min.js"></script>
<link href="protocol_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="protocol_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="protocol_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="protocol_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="protocol_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script src="protocol_files/libs/htmlwidgets-1.5.4/htmlwidgets.js"></script>
<script src="protocol_files/libs/jquery-3.5.1/jquery.min.js"></script>
<link href="protocol_files/libs/leaflet-1.3.1/leaflet.css" rel="stylesheet">
<script src="protocol_files/libs/leaflet-1.3.1/leaflet.js"></script>
<link href="protocol_files/libs/leafletfix-1.0.0/leafletfix.css" rel="stylesheet">
<script src="protocol_files/libs/proj4-2.6.2/proj4.min.js"></script>
<script src="protocol_files/libs/Proj4Leaflet-1.0.1/proj4leaflet.js"></script>
<link href="protocol_files/libs/rstudio_leaflet-1.3.1/rstudio_leaflet.css" rel="stylesheet">
<script src="protocol_files/libs/leaflet-binding-2.1.1/leaflet.js"></script>
<link href="protocol_files/libs/leaflet-markercluster-1.0.5/MarkerCluster.css" rel="stylesheet">
<link href="protocol_files/libs/leaflet-markercluster-1.0.5/MarkerCluster.Default.css" rel="stylesheet">
<script src="protocol_files/libs/leaflet-markercluster-1.0.5/leaflet.markercluster.js"></script>
<script src="protocol_files/libs/leaflet-markercluster-1.0.5/leaflet.markercluster.freezable.js"></script>
<script src="protocol_files/libs/leaflet-markercluster-1.0.5/leaflet.markercluster.layersupport.js"></script>
<script src="protocol_files/libs/plotly-binding-4.10.1/plotly.js"></script>
<script src="protocol_files/libs/typedarray-0.1/typedarray.min.js"></script>
<link href="protocol_files/libs/crosstalk-1.2.0/css/crosstalk.min.css" rel="stylesheet">
<script src="protocol_files/libs/crosstalk-1.2.0/js/crosstalk.min.js"></script>
<link href="protocol_files/libs/plotly-htmlwidgets-css-2.11.1/plotly-htmlwidgets.css" rel="stylesheet">
<script src="protocol_files/libs/plotly-main-2.11.1/plotly-latest.min.js"></script>


</head>

<body>

<div id="quarto-content" class="page-columns page-rows-contents page-layout-full toc-left">
<div id="quarto-sidebar-toc-left" class="sidebar toc-left">
  <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#motivation" id="toc-motivation" class="nav-link active" data-scroll-target="#motivation">Motivation</a></li>
  <li><a href="#introduction" id="toc-introduction" class="nav-link" data-scroll-target="#introduction">Introduction</a></li>
  <li><a href="#project-aims" id="toc-project-aims" class="nav-link" data-scroll-target="#project-aims">Project aims</a></li>
  <li><a href="#method" id="toc-method" class="nav-link" data-scroll-target="#method">Method</a>
  <ul class="collapse">
  <li><a href="#search-strategy" id="toc-search-strategy" class="nav-link" data-scroll-target="#search-strategy">Search strategy</a></li>
  <li><a href="#data-extraction" id="toc-data-extraction" class="nav-link" data-scroll-target="#data-extraction">Data extraction</a>
  <ul class="collapse">
  <li><a href="#included-studies" id="toc-included-studies" class="nav-link" data-scroll-target="#included-studies">Included studies</a></li>
  <li><a href="#rodent-sampling" id="toc-rodent-sampling" class="nav-link" data-scroll-target="#rodent-sampling">Rodent sampling</a></li>
  <li><a href="#pathogen-sampling" id="toc-pathogen-sampling" class="nav-link" data-scroll-target="#pathogen-sampling">Pathogen sampling</a></li>
  <li><a href="#pathogen-sequences" id="toc-pathogen-sequences" class="nav-link" data-scroll-target="#pathogen-sequences">Pathogen sequences</a></li>
  <li><a href="#zoonosis-status" id="toc-zoonosis-status" class="nav-link" data-scroll-target="#zoonosis-status">Zoonosis status</a></li>
  </ul></li>
  <li><a href="#data-processing" id="toc-data-processing" class="nav-link" data-scroll-target="#data-processing">Data processing</a></li>
  </ul></li>
  <li><a href="#outputs" id="toc-outputs" class="nav-link" data-scroll-target="#outputs">Outputs</a>
  <ul class="collapse">
  <li><a href="#data-visualisation" id="toc-data-visualisation" class="nav-link" data-scroll-target="#data-visualisation">Data visualisation</a>
  <ul class="collapse">
  <li><a href="#rodent-sampling-1" id="toc-rodent-sampling-1" class="nav-link" data-scroll-target="#rodent-sampling-1">Rodent sampling</a></li>
  <li><a href="#pathogen-detection" id="toc-pathogen-detection" class="nav-link" data-scroll-target="#pathogen-detection">Pathogen detection</a></li>
  <li><a href="#host-pathogen-associations" id="toc-host-pathogen-associations" class="nav-link" data-scroll-target="#host-pathogen-associations">Host pathogen associations</a></li>
  </ul></li>
  </ul></li>
  </ul>
</nav>
</div>
<div id="quarto-margin-sidebar" class="sidebar margin-sidebar zindex-bottom">
</div>
<main class="content column-page-right" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Study protocol: Production of Arenavirus and Hantavirus host-pathogen database.</h1>
  <div class="quarto-categories">
    <div class="quarto-category">Arenavirus</div>
    <div class="quarto-category">Hantavirus</div>
  </div>
  </div>


<div class="quarto-title-meta-author">
  <div class="quarto-title-meta-heading">Author</div>
  <div class="quarto-title-meta-heading"></div>
  
    <div class="quarto-title-meta-contents">
    <a href="https://www.dsimons.org">David Simons</a> 
  </div>
    <div class="quarto-title-meta-contents">
        <p class="affiliation">
            The Royal Veterinary College, London, UK
          </p>
      </div>
    </div>

<div class="quarto-title-meta">

      
    <div>
    <div class="quarto-title-meta-heading">Published</div>
    <div class="quarto-title-meta-contents">
      <p class="date">January 10, 2023</p>
    </div>
  </div>
  
    
  </div>
  

</header>

<section id="motivation" class="level1">
<h1>Motivation</h1>
<p>Current host-pathogen association datasets provide synthesised information on hosts and their pathogens but do not contain temporal or geographic information. These resources often provide linking information to publications reporting the association but information including accession numbers of archived sequences, number of individuals tested and measures of prevalence among sampled populations are not immediately retrievable. Using these resources for inference beyond host-pathogen associations is therefore limited. Here, we aim to produce a database of host-pathogen associations for two viral families of small mammals which contain several known zoonoses, namely Arenaviridae and Hantaviridae. This database can be used to explore the distribution of small mammal hosts of known and suspected pathogens and the extent to which they have been sampled. Further, linkage to sequence data of known zoonoses will support analysis of the risk of viral reassortment between viral species within geographically co-located host species.</p>
</section>
<section id="introduction" class="level1">
<h1>Introduction</h1>
</section>
<section id="project-aims" class="level1">
<h1>Project aims</h1>
<ol type="1">
<li>Review the literature to produce a synthesised dataset on Arenaviridae and Hantaviridae among small mammals</li>
<li>Implement visualisation tools to explore sampling extent for hosts and pathogens to further understanding of host-pathogen associations</li>
<li>Investigate the relative contributions of ecological, geographic, and genomic factors leading to cross-species transmission and reassortment in rodent-associated arenaviruses and hantaviruses</li>
</ol>
</section>
<section id="method" class="level1">
<h1>Method</h1>
<section id="search-strategy" class="level2">
<h2 class="anchored" data-anchor-id="search-strategy">Search strategy</h2>
<p>An initial search was run on NCBI Pubmed 2023-01-06.</p>
<ol type="1">
<li>rodent* - 165,513</li>
<li>arenavir* - 2,367</li>
<li>hantavir*.mp. - 4,022</li>
<li>2 OR 3 - 6,308</li>
<li>1 AND 4 - 1,842</li>
</ol>
<p>Citations were downloaded as a text file and imported into <code>R</code> for processing. Deduplication by Pubmed ID resulted in 1821 distinct citations.</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a>search_date <span class="ot">&lt;-</span> <span class="st">"2023-01-06"</span></span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a>articles <span class="ot">&lt;-</span> <span class="fu">read_rds</span>(<span class="fu">here</span>(<span class="st">"data"</span>, <span class="st">"search"</span>, <span class="fu">paste0</span>(<span class="st">"citations_"</span>, search_date, <span class="st">".rds"</span>)))</span>
<span id="cb1-4"><a href="#cb1-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-5"><a href="#cb1-5" aria-hidden="true" tabindex="-1"></a>articles</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stdout">
<pre><code># A tibble: 1,821 x 14
# Groups:   pmid [1,821]
   pmid     doi   title abstr~1 year  month day   jabbrv journal keywo~2 lastn~3
   &lt;chr&gt;    &lt;chr&gt; &lt;chr&gt; &lt;chr&gt;   &lt;chr&gt; &lt;chr&gt; &lt;chr&gt; &lt;chr&gt;  &lt;chr&gt;   &lt;chr&gt;   &lt;chr&gt;  
 1 10030054 ""    [Ven~ "Venez~ 1998  1     1     Acta ~ Acta c~ &lt;NA&gt;    Salas  
 2 10071675 "10.~ Hant~ "We pr~ 1999  2     1     South~ Southe~ &lt;NA&gt;    Schuss~
 3 10073292 "10.~ Spec~ "Hanta~ 1999  1     1     Annu ~ Annual~ &lt;NA&gt;    Peters 
 4 10073697 "10.~ Isol~ "Dobra~ 1999  2     1     J Gen~ The Jo~ &lt;NA&gt;    Nemirov
 5 10081674 "10.~ Gene~ "The 1~ 1999  1     1     Emerg~ Emergi~ &lt;NA&gt;    Monroe 
 6 10081676 "10.~ Long~ "Hanta~ 1999  1     1     Emerg~ Emergi~ &lt;NA&gt;    Mills  
 7 10081677 "10.~ Long~ "For 3~ 1999  1     1     Emerg~ Emergi~ &lt;NA&gt;    Abbott 
 8 10081678 "10.~ A lo~ "We de~ 1999  1     1     Emerg~ Emergi~ &lt;NA&gt;    Kuenzi 
 9 10081679 "10.~ Stat~ "A lon~ 1999  1     1     Emerg~ Emergi~ &lt;NA&gt;    Parmen~
10 10081680 "10.~ Natu~ "A mar~ 1999  1     1     Emerg~ Emergi~ &lt;NA&gt;    Calish~
# ... with 1,811 more rows, 3 more variables: firstname &lt;chr&gt;, address &lt;chr&gt;,
#   email &lt;chr&gt;, and abbreviated variable names 1: abstract, 2: keywords,
#   3: lastname</code></pre>
</div>
</div>
<p>An initial search of the returned citations was conducted to identify nine studies to trial data extraction.</p>
</section>
<section id="data-extraction" class="level2">
<h2 class="anchored" data-anchor-id="data-extraction">Data extraction</h2>
<section id="included-studies" class="level3">
<h3 class="anchored" data-anchor-id="included-studies">Included studies</h3>
<p>Information about the included study is extracted in a <code>descriptive</code> sheet.</p>
<div id="tbl-description" class="anchored">
<table class="table">
<caption>Table&nbsp;1: Study information extraction sheet</caption>
<colgroup>
<col style="width: 30%">
<col style="width: 69%">
</colgroup>
<thead>
<tr class="header">
<th>Column name</th>
<th style="text-align: left;">Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>study_id</td>
<td style="text-align: left;">A unique identifier for the included study</td>
</tr>
<tr class="even">
<td>pubmed_id</td>
<td style="text-align: left;">The pubmed ID of the included study (if available)</td>
</tr>
<tr class="odd">
<td>DOI</td>
<td style="text-align: left;">The digital object identifier of the study (if available), if a DOI is not available a weblink to a persistent page for the reference will be included</td>
</tr>
<tr class="even">
<td>first_author_surname</td>
<td style="text-align: left;">The first authors surname</td>
</tr>
<tr class="odd">
<td>title</td>
<td style="text-align: left;">The title of the manuscript, report or book section</td>
</tr>
<tr class="even">
<td>journal</td>
<td style="text-align: left;">The name of the journal, report or book</td>
</tr>
<tr class="odd">
<td>year</td>
<td style="text-align: left;">The year of publication</td>
</tr>
<tr class="even">
<td>study_design</td>
<td style="text-align: left;">A free text entry succinctly describing the study design for the rodents or pathogens</td>
</tr>
<tr class="odd">
<td>sampling_effort</td>
<td style="text-align: left;">A free text entry to capture the effort of sampling, ideally in trap nights for rodent studies</td>
</tr>
<tr class="even">
<td>data_access</td>
<td style="text-align: left;">Whether study data is available in complete form or whether summarise data only are available</td>
</tr>
<tr class="odd">
<td>linked_manuscripts</td>
<td style="text-align: left;">The DOI or weblink to other studies including the same dataset either in its entirety or a subset, this will be used to attempt to de-duplicate data</td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="rodent-sampling" class="level3">
<h3 class="anchored" data-anchor-id="rodent-sampling">Rodent sampling</h3>
<p>Rodent data is extracted in a <code>rodent</code> sheet. This will include information on the timing of data collection, the location of data collected, the small mammal species detected either as detection/non-detection or number detected. For studies not reporting the detection of an individual species at a location that has been detected at other locations the species will be entered as not-detected at that location.</p>
<div id="tbl-rodent" class="anchored">
<table class="table">
<caption>Table&nbsp;2: Rodent information extraction sheet</caption>
<colgroup>
<col style="width: 30%">
<col style="width: 69%">
</colgroup>
<thead>
<tr class="header">
<th>Column name</th>
<th style="text-align: left;">Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>rodent_record_id</td>
<td style="text-align: left;">A unique identifier for the rodent species, at a specific location or timepoint reported by the study dependent on the level of aggregation reported in the study</td>
</tr>
<tr class="even">
<td>study_id</td>
<td style="text-align: left;">A unique identifier to link a study to the <code>descriptive</code> sheet entry for that study</td>
</tr>
<tr class="odd">
<td>date</td>
<td style="text-align: left;">The period in which data collection of rodent samples occurred, this will be extracted at the highest temporal resolution provided</td>
</tr>
<tr class="even">
<td>genus</td>
<td style="text-align: left;">The genus of the small-mammal as reported in the study</td>
</tr>
<tr class="odd">
<td>species</td>
<td style="text-align: left;">The species of the small-mammal as reported in the study</td>
</tr>
<tr class="even">
<td>location</td>
<td style="text-align: left;">The location of sampling effort, depending on how data are presented in the study this will match to the coordinates given for trapping effort. i.e.&nbsp;if trapping is aggregated at village level, village names will be used</td>
</tr>
<tr class="odd">
<td>country</td>
<td style="text-align: left;">Country where trapping occurred, for multinational studies where numbers are not disaggregated by country, all countries will be included</td>
</tr>
<tr class="even">
<td>habitat_type</td>
<td style="text-align: left;">High level habitat type will be recorded here at the scale for which trapping is recorded</td>
</tr>
<tr class="odd">
<td>coordinate_resolution</td>
<td style="text-align: left;">The description of coordinate levels provided in the study, i.e.&nbsp;aggregated at study site or study region</td>
</tr>
<tr class="even">
<td>latitude</td>
<td style="text-align: left;">Latitude will be converted from coordinates presented to EPSG:4326</td>
</tr>
<tr class="odd">
<td>longitude</td>
<td style="text-align: left;">Longitude will be converted from coordinates presented to EPSG:4326</td>
</tr>
<tr class="even">
<td>number</td>
<td style="text-align: left;">The number of detected individuals, for capture-mark-recapture studies the number of distinct individuals will be entered. For studies not explicitly reporting non-detection, values of 0 for a species or genus will be entered if it is detected elsewhere in the study</td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="pathogen-sampling" class="level3">
<h3 class="anchored" data-anchor-id="pathogen-sampling">Pathogen sampling</h3>
<p>Pathogen assays are extracted in the <code>pathogen</code> sheet. This includes information on the host the sample originated from, the pathogen family and species being assayed for and the method of the assay. For studies conducting multiple assays on the same samples for different pathogens additional records will be added for each assay. Similarly, if antibody and direct detection for the same pathogen on the same samples is performed additional records will be added.</p>
<div id="tbl-pathogen" class="anchored">
<table class="table">
<caption>Table&nbsp;3: Pathogen information extraction sheet</caption>
<colgroup>
<col style="width: 30%">
<col style="width: 69%">
</colgroup>
<thead>
<tr class="header">
<th>Column name</th>
<th style="text-align: left;">Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>pathogen_record_id</td>
<td style="text-align: left;">A unique identifier for the group of samples from the same rodent species, at a specific location or timepoint, tested for the same pathogen using the same method</td>
</tr>
<tr class="even">
<td>study_id</td>
<td style="text-align: left;">A unique identifier to link a study to the <code>descriptive</code> sheet entry for that study</td>
</tr>
<tr class="odd">
<td>date</td>
<td style="text-align: left;">The period in which data collection of rodent samples occurred, this will be extracted at the highest temporal resolution provided</td>
</tr>
<tr class="even">
<td>host_genus</td>
<td style="text-align: left;">The genus of the small-mammal from which the sample originated as reported in the study</td>
</tr>
<tr class="odd">
<td>host_species</td>
<td style="text-align: left;">The species of the small-mammal from which the sample originated as reported in the study</td>
</tr>
<tr class="even">
<td>location</td>
<td style="text-align: left;">The location of sampling effort, depending on how data are presented in the study this will match to the coordinates given for trapping effort. i.e.&nbsp;if trapping is aggregated at village level, village names will be used</td>
</tr>
<tr class="odd">
<td>country</td>
<td style="text-align: left;">Country where trapping occurred, for multinational studies where numbers are not disaggregated by country, all countries will be included</td>
</tr>
<tr class="even">
<td>habitat_type</td>
<td style="text-align: left;">High level habitat type will be recorded here at the scale for which trapping is recorded</td>
</tr>
<tr class="odd">
<td>coordinate_resolution</td>
<td style="text-align: left;">The description of coordinate levels provided in the study, i.e.&nbsp;aggregated at study site or study region</td>
</tr>
<tr class="even">
<td>latitude</td>
<td style="text-align: left;">Latitude will be converted from coordinates presented to EPSG:4326</td>
</tr>
<tr class="odd">
<td>longitude</td>
<td style="text-align: left;">Longitude will be converted from coordinates presented to EPSG:4326</td>
</tr>
<tr class="even">
<td>pathogen_family</td>
<td style="text-align: left;">The family of virus being assayed for (i.e.&nbsp;Arenaviridae or Hantaviridae)</td>
</tr>
<tr class="odd">
<td>pathogen_species</td>
<td style="text-align: left;">The species of virus being assayed for if a specific test is being used. For assays unable to differentiate between multiple viral species <code>Multiple</code> will be entered</td>
</tr>
<tr class="even">
<td>detection_method</td>
<td style="text-align: left;">Whether the assay is attempting to detect antibody, direct detection of virus (i.e.&nbsp;pcr), or other</td>
</tr>
<tr class="odd">
<td>number_tested</td>
<td style="text-align: left;">The number of distinct samples tested</td>
</tr>
<tr class="even">
<td>number_negative</td>
<td style="text-align: left;">The number of reported negative samples</td>
</tr>
<tr class="odd">
<td>number_positive</td>
<td style="text-align: left;">The number of reported positive samples</td>
</tr>
<tr class="even">
<td>number_inconclusive</td>
<td style="text-align: left;">The number of samples with inconclusive results</td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="pathogen-sequences" class="level3">
<h3 class="anchored" data-anchor-id="pathogen-sequences">Pathogen sequences</h3>
<p>If studies include linkage to complete or partial sequences of viruses archived in NCBI they will be lined through the <code>pathogen_sequences</code> sheet.</p>
<div id="tbl-sequence" class="anchored">
<table class="table">
<caption>Table&nbsp;4: Pathogen sequences extraction sheet</caption>
<colgroup>
<col style="width: 30%">
<col style="width: 69%">
</colgroup>
<thead>
<tr class="header">
<th>Column name</th>
<th style="text-align: left;">Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>sequence_record_id</td>
<td style="text-align: left;">A unique identifier for the sequence record</td>
</tr>
<tr class="even">
<td>study_id</td>
<td style="text-align: left;">A unique identifier to link a study to the <code>descriptive</code> sheet entry for that study</td>
</tr>
<tr class="odd">
<td>host_genus</td>
<td style="text-align: left;">The genus of the small-mammal from which the sample originated as reported in the study</td>
</tr>
<tr class="even">
<td>host_species</td>
<td style="text-align: left;">The species of the small-mammal from which the sample originated as reported in the study</td>
</tr>
<tr class="odd">
<td>pathogen_species</td>
<td style="text-align: left;">The species of the pathogen</td>
</tr>
<tr class="even">
<td>accession_number</td>
<td style="text-align: left;">The accession number for each record archived by the study</td>
</tr>
</tbody>
</table>
</div>
</section>
<section id="zoonosis-status" class="level3">
<h3 class="anchored" data-anchor-id="zoonosis-status">Zoonosis status</h3>
<p>Finally, an additional sheet <code>known_zoonoses</code> will be produced containing all of the viral species sampled and whether they are known to cause disease in humans.</p>
<div id="tbl-sequence" class="anchored">
<table class="table">
<caption>Table&nbsp;5: Zoonosis status extraction sheet</caption>
<colgroup>
<col style="width: 30%">
<col style="width: 69%">
</colgroup>
<thead>
<tr class="header">
<th>Column name</th>
<th style="text-align: left;">Description</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>pathogen_id</td>
<td style="text-align: left;">A unique identifier for the pathogen species</td>
</tr>
<tr class="even">
<td>pathogen_family</td>
<td style="text-align: left;">The family of pathogen</td>
</tr>
<tr class="odd">
<td>pathogen_species</td>
<td style="text-align: left;">The species of pathogen</td>
</tr>
<tr class="even">
<td>known_zoonosis</td>
<td style="text-align: left;">A logical statement of pathogenicity among humans</td>
</tr>
<tr class="odd">
<td>disease_name</td>
<td style="text-align: left;">The disease name caused by the viral species, if multiple diseases they are entered with a comma separator</td>
</tr>
<tr class="even">
<td>icd_10</td>
<td style="text-align: left;">The ICD-10 associated name of the disease (if known)</td>
</tr>
<tr class="odd">
<td>disease_reference</td>
<td style="text-align: left;">A DOI of a publication that supports the <code>known_zoonosis</code> statement</td>
</tr>
</tbody>
</table>
</div>
</section>
</section>
<section id="data-processing" class="level2">
<h2 class="anchored" data-anchor-id="data-processing">Data processing</h2>
<p>Raw data will be downloaded from Google Sheets using the <code>googledrive</code> API in <code>R</code>, with date stamped files stored locally. Data will be imported into <code>R</code> for processing, cleaning and formatting to produce a dataset suitable for further analysis.</p>
</section>
</section>
<section id="outputs" class="level1">
<h1>Outputs</h1>
<section id="data-visualisation" class="level2">
<h2 class="anchored" data-anchor-id="data-visualisation">Data visualisation</h2>
<p>An <code>RShiny</code> web-based application will be produced to visualise the database and support future analysis.</p>
<section id="rodent-sampling-1" class="level3">
<h3 class="anchored" data-anchor-id="rodent-sampling-1">Rodent sampling</h3>
<p>An example of a map produce from rodent sampling data included in the initial 9 studies is shown below.</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb3"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a>combined_data <span class="ot">&lt;-</span> <span class="fu">read_rds</span>(<span class="fu">here</span>(<span class="st">"data"</span>, <span class="st">"raw_data"</span>, <span class="fu">paste0</span>(search_date, <span class="st">"_data.rds"</span>)))</span>
<span id="cb3-2"><a href="#cb3-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-3"><a href="#cb3-3" aria-hidden="true" tabindex="-1"></a>project_crs <span class="ot">&lt;-</span> <span class="st">"EPSG:4326"</span></span>
<span id="cb3-4"><a href="#cb3-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-5"><a href="#cb3-5" aria-hidden="true" tabindex="-1"></a>rodent_locations <span class="ot">&lt;-</span> combined_data<span class="sc">$</span>rodent <span class="sc">%&gt;%</span></span>
<span id="cb3-6"><a href="#cb3-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(genus, species, latitude, longitude, number) <span class="sc">%&gt;%</span></span>
<span id="cb3-7"><a href="#cb3-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">number =</span> <span class="fu">as.numeric</span>(number)) <span class="sc">%&gt;%</span></span>
<span id="cb3-8"><a href="#cb3-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">drop_na</span>(number) <span class="sc">%&gt;%</span></span>
<span id="cb3-9"><a href="#cb3-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">st_as_sf</span>(<span class="at">coords =</span> <span class="fu">c</span>(<span class="st">"longitude"</span>, <span class="st">"latitude"</span>), <span class="at">crs =</span> project_crs) <span class="sc">%&gt;%</span></span>
<span id="cb3-10"><a href="#cb3-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(geometry, species) <span class="sc">%&gt;%</span></span>
<span id="cb3-11"><a href="#cb3-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">number =</span> <span class="fu">sum</span>(number)) <span class="sc">%&gt;%</span></span>
<span id="cb3-12"><a href="#cb3-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ungroup</span>() <span class="sc">%&gt;%</span></span>
<span id="cb3-13"><a href="#cb3-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">distinct</span>() <span class="sc">%&gt;%</span></span>
<span id="cb3-14"><a href="#cb3-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">rowwise</span>() <span class="sc">%&gt;%</span></span>
<span id="cb3-15"><a href="#cb3-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">labels =</span> <span class="fu">HTML</span>(<span class="fu">paste0</span>(<span class="st">"Species: "</span>, species, <span class="st">"&lt;br&gt;"</span>, <span class="st">"Number detected: "</span>, number)))</span>
<span id="cb3-16"><a href="#cb3-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-17"><a href="#cb3-17" aria-hidden="true" tabindex="-1"></a>pal <span class="ot">&lt;-</span> <span class="fu">colorFactor</span>(<span class="fu">diverging_hcl</span>(<span class="at">n =</span> <span class="fu">length</span>(<span class="fu">sort</span>(<span class="fu">unique</span>(rodent_locations<span class="sc">$</span>genus))), <span class="at">palette =</span> <span class="st">"Berlin"</span>),</span>
<span id="cb3-18"><a href="#cb3-18" aria-hidden="true" tabindex="-1"></a>                   <span class="at">domain =</span> <span class="fu">sort</span>(<span class="fu">unique</span>(rodent_locations<span class="sc">$</span>genus)))</span>
<span id="cb3-19"><a href="#cb3-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-20"><a href="#cb3-20" aria-hidden="true" tabindex="-1"></a><span class="fu">leaflet</span>(<span class="at">data =</span> rodent_locations) <span class="sc">%&gt;%</span></span>
<span id="cb3-21"><a href="#cb3-21" aria-hidden="true" tabindex="-1"></a>  <span class="fu">addTiles</span>() <span class="sc">%&gt;%</span></span>
<span id="cb3-22"><a href="#cb3-22" aria-hidden="true" tabindex="-1"></a>  <span class="fu">addCircleMarkers</span>(<span class="at">radius =</span> <span class="sc">~</span><span class="fu">ifelse</span>(<span class="fu">sqrt</span>(number) <span class="sc">==</span> <span class="dv">0</span>, <span class="dv">1</span>, <span class="fu">sqrt</span>(number)),</span>
<span id="cb3-23"><a href="#cb3-23" aria-hidden="true" tabindex="-1"></a>                   <span class="at">color =</span> <span class="sc">~</span><span class="fu">pal</span>(genus),</span>
<span id="cb3-24"><a href="#cb3-24" aria-hidden="true" tabindex="-1"></a>                   <span class="at">stroke =</span> <span class="cn">FALSE</span>,</span>
<span id="cb3-25"><a href="#cb3-25" aria-hidden="true" tabindex="-1"></a>                   <span class="at">fillOpacity =</span> <span class="fl">0.8</span>,</span>
<span id="cb3-26"><a href="#cb3-26" aria-hidden="true" tabindex="-1"></a>                   <span class="at">clusterOptions =</span> <span class="fu">markerClusterOptions</span>(<span class="at">spiderfyOnMaxZoom =</span> <span class="cn">TRUE</span>,</span>
<span id="cb3-27"><a href="#cb3-27" aria-hidden="true" tabindex="-1"></a>                                                         <span class="at">spiderLegPolylineOptions =</span> <span class="fu">list</span>(<span class="at">length =</span> <span class="dv">6</span>),</span>
<span id="cb3-28"><a href="#cb3-28" aria-hidden="true" tabindex="-1"></a>                                                         <span class="at">spiderfyDistanceMultiplier =</span> <span class="dv">2</span>),</span>
<span id="cb3-29"><a href="#cb3-29" aria-hidden="true" tabindex="-1"></a>                   <span class="at">label =</span> <span class="sc">~</span>labels,</span>
<span id="cb3-30"><a href="#cb3-30" aria-hidden="true" tabindex="-1"></a>                   <span class="at">popup =</span> <span class="sc">~</span>labels) <span class="sc">%&gt;%</span></span>
<span id="cb3-31"><a href="#cb3-31" aria-hidden="true" tabindex="-1"></a>  <span class="fu">addLegend</span>(<span class="st">"bottomright"</span>, <span class="at">pal =</span> pal, <span class="at">values =</span> <span class="sc">~</span>genus, <span class="at">title =</span> <span class="st">"Host genus"</span>, <span class="at">opacity =</span> <span class="fl">0.8</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div id="htmlwidget-d4f04ced807fd13e10a8" style="width:100%;height:742px;" class="leaflet html-widget"></div>
<script type="application/json" data-for="htmlwidget-d4f04ced807fd13e10a8">{"x":{"options":{"crs":{"crsClass":"L.CRS.EPSG3857","code":null,"proj4def":null,"projectedBounds":null,"options":{}}},"calls":[{"method":"addTiles","args":["https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png",null,null,{"minZoom":0,"maxZoom":18,"tileSize":256,"subdomains":"abc","errorTileUrl":"","tms":false,"noWrap":false,"zoomOffset":0,"zoomReverse":false,"opacity":1,"zIndex":1,"detectRetina":false,"attribution":"&copy; <a href=\"https://openstreetmap.org\">OpenStreetMap<\/a> contributors, <a href=\"https://creativecommons.org/licenses/by-sa/2.0/\">CC-BY-SA<\/a>"}]},{"method":"addCircleMarkers","args":[[31.833135,31.833135,31.833135,31.833135,31.833135,31.833135,31.833135,31.833135,31.833135,31.833135,31.833135,33.446953,33.446953,33.446953,33.446953,33.446953,33.446953,33.446953,33.446953,33.446953,33.446953,33.446953,33.446953,33.446953,33.446953,33.446953,33.4469529999999,33.4469529999999,33.4469529999999,33.4469529999999,33.4469529999999,33.4469529999999,33.4469529999999,33.4469529999999,33.4469529999999,33.4469529999999,33.4469529999999,33.4469529999998,33.4469529999998,33.4469529999998,33.4469529999998,33.4469529999998,33.4469529999998,33.4469529999998,33.4469529999998,33.4469529999998,33.4469529999998,33.4469529999998,33.4469529999997,33.4469529999997,33.4469529999997,33.4469529999997,33.4469529999997,33.4469529999997,33.4469529999997,48.719,48.719,48.0466,63.8258,40.75,40.75,40.75,40.75,40.75,41,41,41,41,41,41,41.0167,41.0167,41.0167,41.0167,41.0167,41.0167,36.204,36.204,36.204,36.204,36.204,38.11002,38.11002,38.11002,38.11002,38.11002,38.11002,38.11002,38.11002,38.11002,38.051592,38.051592,38.051592,38.051592,38.051592,38.051592,38.051592,38.051592,38.051592,49.283333,49.283333,7.945028,7.945028,7.945028,7.945028,7.945028,7.945028,7.945028,7.945028,7.945028,7.945028,7.945028,7.945028,7.945028,7.945028,7.945028,7.945028,8.005569,8.005569,8.005569,8.005569,8.005569,8.005569,8.005569,8.005569,8.005569,8.005569,8.005569,8.005569,8.005569,8.005569,8.005569,8.005569,7.757033,7.757033,7.757033,7.757033,7.757033,7.757033,7.75703299999999,7.75703299999999,7.75703299999999,7.75703299999999,7.75703299999999,7.75703299999999,7.75703299999999,7.75703299999999,7.75703299999999,7.75703299999998,7.762217,7.762217,7.762217,7.762217,7.762217,7.762217,7.762217,7.762217,7.762217,7.762217,7.762217,7.762217,7.762217,7.762217,7.762217,7.762217,7.771753,7.771753,7.771753,7.771753,7.771753,7.771753,7.771753,7.771753,7.771753,7.771753,7.771753,7.771753,7.771753,7.771753,7.771753,7.771753,7.973069,7.973069,7.973069,7.973069,7.973069,7.973069,7.973069,7.973069,7.973069,7.973069,7.973069,7.973069,7.973069,7.973069,7.973069,7.973069],[-110.852966,-110.852966,-110.852966,-110.852966,-110.852966,-110.852966,-110.852966,-110.852966,-110.852966,-110.852966,-110.852966,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,21.2611,21.2611,17.34802,20.263,39.616,39.616,39.616,39.616,39.616,41.05,41.05,41.05,41.05,41.05,41.05,40.983,40.983,40.983,40.983,40.983,40.983,138.252,138.252,138.252,138.252,138.252,127.07705,127.07705,127.07705,127.07705,127.07705,127.07705,127.07705,127.07705,127.07705,127.1069191,127.1069191,127.1069191,127.1069191,127.1069191,127.1069191,127.1069191,127.1069191,127.1069191,-123.1,-123.1,-11.431461,-11.431461,-11.431461,-11.431461,-11.431461,-11.431461,-11.431461,-11.431461,-11.431461,-11.431461,-11.431461,-11.431461,-11.431461,-11.431461,-11.431461,-11.431461,-11.427842,-11.427842,-11.427842,-11.427842,-11.427842,-11.427842,-11.427842,-11.427842,-11.427842,-11.427842,-11.427842,-11.427842,-11.427842,-11.427842,-11.427842,-11.427842,-11.770189,-11.770189,-11.770189,-11.770189,-11.770189,-11.770189,-11.770189,-11.770189,-11.770189,-11.770189,-11.770189,-11.770189,-11.770189,-11.770189,-11.770189,-11.770189,-11.742564,-11.742564,-11.742564,-11.742564,-11.742564,-11.742564,-11.742564,-11.742564,-11.742564,-11.742564,-11.742564,-11.742564,-11.742564,-11.742564,-11.742564,-11.742564,-11.726172,-11.726172,-11.726172,-11.726172,-11.726172,-11.726172,-11.726172,-11.726172,-11.726172,-11.726172,-11.726172,-11.726172,-11.726172,-11.726172,-11.726172,-11.726172,-11.3957,-11.3957,-11.3957,-11.3957,-11.3957,-11.3957,-11.3957,-11.3957,-11.3957,-11.3957,-11.3957,-11.3957,-11.3957,-11.3957,-11.3957,-11.3957],[26.5329983228432,11.2249721603218,12.2882057274445,11.7046999107196,2.44948974278318,5.3851648071345,4,3.3166247903554,5,1,2.64575131106459,3.46410161513775,1.4142135623731,4.12310562561766,5.29150262212918,8.94427190999916,7.48331477354788,6.08276253029822,13.6747943311773,11.5758369027902,22.4722050542442,14.1067359796659,1,1,1.73205080756888,1,1,1.4142135623731,5.65685424949238,1,1,1,1,1,6.6332495807108,3.74165738677394,2.44948974278318,1,1,1,2.82842712474619,1,4.58257569495584,8.77496438739212,2.64575131106459,5.65685424949238,1.73205080756888,4.69041575982343,6.40312423743285,18.5741756210067,13.8202749610853,6.78232998312527,1.4142135623731,4,1,17.4928556845359,7.41619848709566,1,46.6154480832267,1.73205080756888,3.16227766016838,8.48528137423857,2,2.82842712474619,5.29150262212918,3.16227766016838,9.05538513813742,1,3.3166247903554,2.64575131106459,1,2.64575131106459,7.21110255092798,5,1,3,1.73205080756888,2.23606797749979,2.44948974278318,1,2.44948974278318,22.8035085019828,1,2.44948974278318,2,1,1,1,1,4.89897948556636,25.4361946839538,2,1.73205080756888,3.3166247903554,1,2.44948974278318,1.4142135623731,1,7.48331477354788,26.1725046566048,6.32455532033676,6.48074069840786,1,1,1,1,2.82842712474619,1,6.40312423743285,1,2,7.74596669241483,2,2.64575131106459,6.6332495807108,7.81024967590665,1,4.89897948556636,1,1,1,1.73205080756888,2.23606797749979,1,3.74165738677394,1,1,9.79795897113271,1.73205080756888,1.4142135623731,6.2449979983984,5.09901951359278,1,5.19615242270663,1.4142135623731,1,1,2,1.4142135623731,1,5.74456264653803,1.73205080756888,1.4142135623731,4,1.4142135623731,1,4.58257569495584,6.32455532033676,1,7.34846922834953,1,1,1,1.73205080756888,1,1,5,2,1.4142135623731,10,1,1.73205080756888,10.0995049383621,7.61577310586391,1,5.91607978309962,2,1,1,1,1,1,5.56776436283002,2.44948974278318,1.73205080756888,5.8309518948453,1,1,6.92820323027551,6.48074069840786,1.4142135623731,5.29150262212918,1,1,1,2.23606797749979,2.44948974278318,1,5.47722557505166,3,1.4142135623731,7.14142842854285,1,3.3166247903554,9.59166304662544,5.65685424949238,1],null,null,{"interactive":true,"className":"","stroke":false,"color":["#5EAEEA","#3B1917","#532521","#532521","#532521","#532521","#813D38","#934641","#5EAEEA","#0072A9","#471E1C","#5EAEEA","#5EAEEA","#111111","#1C1110","#3B1917","#3B1917","#532521","#532521","#532521","#532521","#813D38","#713430","#713430","#BA5C56","#CF6862","#5EAEEA","#111111","#1C1110","#3B1917","#3B1917","#532521","#532521","#532521","#532521","#813D38","#713430","#713430","#BA5C56","#CF6862","#5EAEEA","#5EAEEA","#111111","#1C1110","#3B1917","#3B1917","#532521","#532521","#532521","#532521","#813D38","#713430","#713430","#BA5C56","#CF6862","#7FBFF5","#7FBFF5","#7FBFF5","#309FE0","#7FBFF5","#7FBFF5","#7FBFF5","#111111","#111111","#7FBFF5","#7FBFF5","#7FBFF5","#006496","#111111","#111111","#7FBFF5","#7FBFF5","#7FBFF5","#111111","#111111","#111111","#A6514B","#A6514B","#A6514B","#0090D3","#F8A29E","#7FBFF5","#0D141B","#1C1110","#713430","#713430","#DE7A74","#111111","#261211","#0090D3","#7FBFF5","#0D141B","#1C1110","#713430","#713430","#DE7A74","#111111","#261211","#0090D3","#713430","#713430","#0090D3","#0080BD","#005784","#004B73","#004164","#003755","#002E48","#00263C","#021F31","#081926","#081926","#301513","#301513","#612C28","#713430","#EB8E88","#0090D3","#0080BD","#005784","#004B73","#004164","#003755","#002E48","#00263C","#021F31","#081926","#081926","#301513","#301513","#612C28","#713430","#EB8E88","#0090D3","#0080BD","#005784","#004B73","#004164","#003755","#002E48","#00263C","#021F31","#081926","#081926","#301513","#301513","#612C28","#713430","#EB8E88","#0090D3","#0080BD","#005784","#004B73","#004164","#003755","#002E48","#00263C","#021F31","#081926","#081926","#301513","#301513","#612C28","#713430","#EB8E88","#0090D3","#0080BD","#005784","#004B73","#004164","#003755","#002E48","#00263C","#021F31","#081926","#081926","#301513","#301513","#612C28","#713430","#EB8E88","#0090D3","#0080BD","#005784","#004B73","#004164","#003755","#002E48","#00263C","#021F31","#081926","#081926","#301513","#301513","#612C28","#713430","#EB8E88"],"weight":5,"opacity":0.5,"fill":true,"fillColor":["#5EAEEA","#3B1917","#532521","#532521","#532521","#532521","#813D38","#934641","#5EAEEA","#0072A9","#471E1C","#5EAEEA","#5EAEEA","#111111","#1C1110","#3B1917","#3B1917","#532521","#532521","#532521","#532521","#813D38","#713430","#713430","#BA5C56","#CF6862","#5EAEEA","#111111","#1C1110","#3B1917","#3B1917","#532521","#532521","#532521","#532521","#813D38","#713430","#713430","#BA5C56","#CF6862","#5EAEEA","#5EAEEA","#111111","#1C1110","#3B1917","#3B1917","#532521","#532521","#532521","#532521","#813D38","#713430","#713430","#BA5C56","#CF6862","#7FBFF5","#7FBFF5","#7FBFF5","#309FE0","#7FBFF5","#7FBFF5","#7FBFF5","#111111","#111111","#7FBFF5","#7FBFF5","#7FBFF5","#006496","#111111","#111111","#7FBFF5","#7FBFF5","#7FBFF5","#111111","#111111","#111111","#A6514B","#A6514B","#A6514B","#0090D3","#F8A29E","#7FBFF5","#0D141B","#1C1110","#713430","#713430","#DE7A74","#111111","#261211","#0090D3","#7FBFF5","#0D141B","#1C1110","#713430","#713430","#DE7A74","#111111","#261211","#0090D3","#713430","#713430","#0090D3","#0080BD","#005784","#004B73","#004164","#003755","#002E48","#00263C","#021F31","#081926","#081926","#301513","#301513","#612C28","#713430","#EB8E88","#0090D3","#0080BD","#005784","#004B73","#004164","#003755","#002E48","#00263C","#021F31","#081926","#081926","#301513","#301513","#612C28","#713430","#EB8E88","#0090D3","#0080BD","#005784","#004B73","#004164","#003755","#002E48","#00263C","#021F31","#081926","#081926","#301513","#301513","#612C28","#713430","#EB8E88","#0090D3","#0080BD","#005784","#004B73","#004164","#003755","#002E48","#00263C","#021F31","#081926","#081926","#301513","#301513","#612C28","#713430","#EB8E88","#0090D3","#0080BD","#005784","#004B73","#004164","#003755","#002E48","#00263C","#021F31","#081926","#081926","#301513","#301513","#612C28","#713430","#EB8E88","#0090D3","#0080BD","#005784","#004B73","#004164","#003755","#002E48","#00263C","#021F31","#081926","#081926","#301513","#301513","#612C28","#713430","#EB8E88"],"fillOpacity":0.8},{"showCoverageOnHover":true,"zoomToBoundsOnClick":true,"spiderfyOnMaxZoom":true,"removeOutsideVisibleBounds":true,"spiderLegPolylineOptions":{"length":6},"freezeAtZoom":false,"spiderfyDistanceMultiplier":2},null,["Species: Chaetodipus baileyi<br>Number detected: 704","Species: Neotoma albigula<br>Number detected: 126","Species: Peromyscus eremicus<br>Number detected: 151","Species: Peromyscus boylii<br>Number detected: 137","Species: Peromyscus maniculatus<br>Number detected: 6","Species: Peromyscus leucopus<br>Number detected: 29","Species: Reithrodontomys fulvescens<br>Number detected: 16","Species: Sigmodon ochrognathus<br>Number detected: 11","Species: Chaetodipus penicillatus<br>Number detected: 25","Species: Dipodomys merriami<br>Number detected: 1","Species: Onychomys torridue<br>Number detected: 7","Species: Chaetodipus californicus<br>Number detected: 12","Species: Chaetodipus fallax<br>Number detected: 2","Species: Microtus californicus<br>Number detected: 17","Species: Mus musculus<br>Number detected: 28","Species: Neotoma fiscipes<br>Number detected: 80","Species: Neotoma lepida<br>Number detected: 56","Species: Peromyscus boylii<br>Number detected: 37","Species: Peromyscus californicus<br>Number detected: 187","Species: Peromyscus eremicus<br>Number detected: 134","Species: Peromyscus maniculatus<br>Number detected: 505","Species: Reithrodontomys megalotis<br>Number detected: 199","Species: Rattus rattus<br>Number detected: 1","Species: Rattus norvegicus<br>Number detected: 0","Species: Spermophilus beecheyi<br>Number detected: 3","Species: Thomomys bottae<br>Number detected: 1","Species: Chaetodipus fallax<br>Number detected: 0","Species: Microtus californicus<br>Number detected: 2","Species: Mus musculus<br>Number detected: 32","Species: Neotoma fiscipes<br>Number detected: 0","Species: Neotoma lepida<br>Number detected: 0","Species: Peromyscus boylii<br>Number detected: 0","Species: Peromyscus californicus<br>Number detected: 0","Species: Peromyscus eremicus<br>Number detected: 0","Species: Peromyscus maniculatus<br>Number detected: 44","Species: Reithrodontomys megalotis<br>Number detected: 14","Species: Rattus rattus<br>Number detected: 6","Species: Rattus norvegicus<br>Number detected: 0","Species: Spermophilus beecheyi<br>Number detected: 0","Species: Thomomys bottae<br>Number detected: 0","Species: Chaetodipus californicus<br>Number detected: 8","Species: Chaetodipus fallax<br>Number detected: 0","Species: Microtus californicus<br>Number detected: 21","Species: Mus musculus<br>Number detected: 77","Species: Neotoma fiscipes<br>Number detected: 7","Species: Neotoma lepida<br>Number detected: 32","Species: Peromyscus boylii<br>Number detected: 3","Species: Peromyscus californicus<br>Number detected: 22","Species: Peromyscus eremicus<br>Number detected: 41","Species: Peromyscus maniculatus<br>Number detected: 345","Species: Reithrodontomys megalotis<br>Number detected: 191","Species: Rattus rattus<br>Number detected: 46","Species: Rattus norvegicus<br>Number detected: 2","Species: Spermophilus beecheyi<br>Number detected: 16","Species: Thomomys bottae<br>Number detected: 0","Species: Apodemus agrarius<br>Number detected: 306","Species: Apodemus flavicollis<br>Number detected: 55","Species: Apodemus agrarius<br>Number detected: 0","Species: Clethrionomys glareolus<br>Number detected: 2173","Species: Apodemus flavicollis<br>Number detected: 3","Species: Apodemus mystacinus<br>Number detected: 10","Species: Apodemus sylvaticus s.l.<br>Number detected: 72","Species: Microtus roberti<br>Number detected: 4","Species: Microtus rossiaemeridonalis<br>Number detected: 8","Species: Apodemus flavicollis<br>Number detected: 28","Species: Apodemus mystacinus<br>Number detected: 10","Species: Apodemus sylvaticus s.l.<br>Number detected: 82","Species: Dryomys nitedula<br>Number detected: 1","Species: Microtus roberti<br>Number detected: 11","Species: Microtus rossiaemeridonalis<br>Number detected: 7","Species: Apodemus flavicollis<br>Number detected: 0","Species: Apodemus mystacinus<br>Number detected: 7","Species: Apodemus sylvaticus s.l.<br>Number detected: 52","Species: Microtus guentheri lydius<br>Number detected: 25","Species: Microtus roberti<br>Number detected: 1","Species: Microtus rossiaemeridonalis<br>Number detected: 9","Species: Sorex caecutiens<br>Number detected: 3","Species: Sorex gracillimus<br>Number detected: 5","Species: Sorex unguiculatus<br>Number detected: 6","Species: Crocidura dsinezumi<br>Number detected: 1","Species: Urotrichis talpoides<br>Number detected: 6","Species: Apodemus agrarius<br>Number detected: 520","Species: Micromys minutus<br>Number detected: 1","Species: Mus musculus<br>Number detected: 6","Species: Rattus norvegicus<br>Number detected: 4","Species: Rattus rattus<br>Number detected: 0","Species: Tscherskia triton<br>Number detected: 0","Species: Microtus fortis<br>Number detected: 1","Species: Myodes regulus<br>Number detected: 1","Species: Crocidura lasiura<br>Number detected: 24","Species: Apodemus agrarius<br>Number detected: 647","Species: Micromys minutus<br>Number detected: 4","Species: Mus musculus<br>Number detected: 3","Species: Rattus norvegicus<br>Number detected: 11","Species: Rattus rattus<br>Number detected: 1","Species: Tscherskia triton<br>Number detected: 6","Species: Microtus fortis<br>Number detected: 2","Species: Myodes regulus<br>Number detected: 1","Species: Crocidura lasiura<br>Number detected: 56","Species: Rattus norvegicus<br>Number detected: 685","Species: Rattus rattus<br>Number detected: 40","Species: Crocidura sp.<br>Number detected: 42","Species: Dasymys rufulus<br>Number detected: 0","Species: Grammomys buntingi<br>Number detected: 1","Species: Graphiurus sp.<br>Number detected: 1","Species: Hybomys sp.<br>Number detected: 0","Species: Hylomyscus simus<br>Number detected: 8","Species: Lemniscomys striatus<br>Number detected: 1","Species: Lophuromys sikapusi<br>Number detected: 41","Species: Malacomys edwardsi<br>Number detected: 1","Species: Mastomys erythroleucus<br>Number detected: 4","Species: Mastomys natalensis<br>Number detected: 60","Species: Nannomys minutoides<br>Number detected: 4","Species: Nannomys setulosus<br>Number detected: 7","Species: Praomys rostratus<br>Number detected: 44","Species: Rattus rattus<br>Number detected: 61","Species: Uranomys ruddi<br>Number detected: 0","Species: Crocidura sp.<br>Number detected: 24","Species: Dasymys rufulus<br>Number detected: 0","Species: Grammomys buntingi<br>Number detected: 0","Species: Graphiurus sp.<br>Number detected: 0","Species: Hybomys sp.<br>Number detected: 3","Species: Hylomyscus simus<br>Number detected: 5","Species: Lemniscomys striatus<br>Number detected: 0","Species: Lophuromys sikapusi<br>Number detected: 14","Species: Malacomys edwardsi<br>Number detected: 0","Species: Mastomys erythroleucus<br>Number detected: 1","Species: Mastomys natalensis<br>Number detected: 96","Species: Nannomys minutoides<br>Number detected: 3","Species: Nannomys setulosus<br>Number detected: 2","Species: Praomys rostratus<br>Number detected: 39","Species: Rattus rattus<br>Number detected: 26","Species: Uranomys ruddi<br>Number detected: 0","Species: Crocidura sp.<br>Number detected: 27","Species: Dasymys rufulus<br>Number detected: 2","Species: Grammomys buntingi<br>Number detected: 1","Species: Graphiurus sp.<br>Number detected: 0","Species: Hybomys sp.<br>Number detected: 4","Species: Hylomyscus simus<br>Number detected: 2","Species: Lemniscomys striatus<br>Number detected: 0","Species: Lophuromys sikapusi<br>Number detected: 33","Species: Malacomys edwardsi<br>Number detected: 3","Species: Mastomys erythroleucus<br>Number detected: 2","Species: Mastomys natalensis<br>Number detected: 16","Species: Nannomys minutoides<br>Number detected: 2","Species: Nannomys setulosus<br>Number detected: 1","Species: Praomys rostratus<br>Number detected: 21","Species: Rattus rattus<br>Number detected: 40","Species: Uranomys ruddi<br>Number detected: 0","Species: Crocidura sp.<br>Number detected: 54","Species: Dasymys rufulus<br>Number detected: 0","Species: Grammomys buntingi<br>Number detected: 0","Species: Graphiurus sp.<br>Number detected: 0","Species: Hybomys sp.<br>Number detected: 3","Species: Hylomyscus simus<br>Number detected: 1","Species: Lemniscomys striatus<br>Number detected: 1","Species: Lophuromys sikapusi<br>Number detected: 25","Species: Malacomys edwardsi<br>Number detected: 4","Species: Mastomys erythroleucus<br>Number detected: 2","Species: Mastomys natalensis<br>Number detected: 100","Species: Nannomys minutoides<br>Number detected: 1","Species: Nannomys setulosus<br>Number detected: 3","Species: Praomys rostratus<br>Number detected: 102","Species: Rattus rattus<br>Number detected: 58","Species: Uranomys ruddi<br>Number detected: 0","Species: Crocidura sp.<br>Number detected: 35","Species: Dasymys rufulus<br>Number detected: 4","Species: Grammomys buntingi<br>Number detected: 0","Species: Graphiurus sp.<br>Number detected: 0","Species: Hybomys sp.<br>Number detected: 1","Species: Hylomyscus simus<br>Number detected: 1","Species: Lemniscomys striatus<br>Number detected: 1","Species: Lophuromys sikapusi<br>Number detected: 31","Species: Malacomys edwardsi<br>Number detected: 6","Species: Mastomys erythroleucus<br>Number detected: 3","Species: Mastomys natalensis<br>Number detected: 34","Species: Nannomys minutoides<br>Number detected: 0","Species: Nannomys setulosus<br>Number detected: 0","Species: Praomys rostratus<br>Number detected: 48","Species: Rattus rattus<br>Number detected: 42","Species: Uranomys ruddi<br>Number detected: 2","Species: Crocidura sp.<br>Number detected: 28","Species: Dasymys rufulus<br>Number detected: 1","Species: Grammomys buntingi<br>Number detected: 0","Species: Graphiurus sp.<br>Number detected: 0","Species: Hybomys sp.<br>Number detected: 5","Species: Hylomyscus simus<br>Number detected: 6","Species: Lemniscomys striatus<br>Number detected: 1","Species: Lophuromys sikapusi<br>Number detected: 30","Species: Malacomys edwardsi<br>Number detected: 9","Species: Mastomys erythroleucus<br>Number detected: 2","Species: Mastomys natalensis<br>Number detected: 51","Species: Nannomys minutoides<br>Number detected: 1","Species: Nannomys setulosus<br>Number detected: 11","Species: Praomys rostratus<br>Number detected: 92","Species: Rattus rattus<br>Number detected: 32","Species: Uranomys ruddi<br>Number detected: 0"],null,["Species: Chaetodipus baileyi<br>Number detected: 704","Species: Neotoma albigula<br>Number detected: 126","Species: Peromyscus eremicus<br>Number detected: 151","Species: Peromyscus boylii<br>Number detected: 137","Species: Peromyscus maniculatus<br>Number detected: 6","Species: Peromyscus leucopus<br>Number detected: 29","Species: Reithrodontomys fulvescens<br>Number detected: 16","Species: Sigmodon ochrognathus<br>Number detected: 11","Species: Chaetodipus penicillatus<br>Number detected: 25","Species: Dipodomys merriami<br>Number detected: 1","Species: Onychomys torridue<br>Number detected: 7","Species: Chaetodipus californicus<br>Number detected: 12","Species: Chaetodipus fallax<br>Number detected: 2","Species: Microtus californicus<br>Number detected: 17","Species: Mus musculus<br>Number detected: 28","Species: Neotoma fiscipes<br>Number detected: 80","Species: Neotoma lepida<br>Number detected: 56","Species: Peromyscus boylii<br>Number detected: 37","Species: Peromyscus californicus<br>Number detected: 187","Species: Peromyscus eremicus<br>Number detected: 134","Species: Peromyscus maniculatus<br>Number detected: 505","Species: Reithrodontomys megalotis<br>Number detected: 199","Species: Rattus rattus<br>Number detected: 1","Species: Rattus norvegicus<br>Number detected: 0","Species: Spermophilus beecheyi<br>Number detected: 3","Species: Thomomys bottae<br>Number detected: 1","Species: Chaetodipus fallax<br>Number detected: 0","Species: Microtus californicus<br>Number detected: 2","Species: Mus musculus<br>Number detected: 32","Species: Neotoma fiscipes<br>Number detected: 0","Species: Neotoma lepida<br>Number detected: 0","Species: Peromyscus boylii<br>Number detected: 0","Species: Peromyscus californicus<br>Number detected: 0","Species: Peromyscus eremicus<br>Number detected: 0","Species: Peromyscus maniculatus<br>Number detected: 44","Species: Reithrodontomys megalotis<br>Number detected: 14","Species: Rattus rattus<br>Number detected: 6","Species: Rattus norvegicus<br>Number detected: 0","Species: Spermophilus beecheyi<br>Number detected: 0","Species: Thomomys bottae<br>Number detected: 0","Species: Chaetodipus californicus<br>Number detected: 8","Species: Chaetodipus fallax<br>Number detected: 0","Species: Microtus californicus<br>Number detected: 21","Species: Mus musculus<br>Number detected: 77","Species: Neotoma fiscipes<br>Number detected: 7","Species: Neotoma lepida<br>Number detected: 32","Species: Peromyscus boylii<br>Number detected: 3","Species: Peromyscus californicus<br>Number detected: 22","Species: Peromyscus eremicus<br>Number detected: 41","Species: Peromyscus maniculatus<br>Number detected: 345","Species: Reithrodontomys megalotis<br>Number detected: 191","Species: Rattus rattus<br>Number detected: 46","Species: Rattus norvegicus<br>Number detected: 2","Species: Spermophilus beecheyi<br>Number detected: 16","Species: Thomomys bottae<br>Number detected: 0","Species: Apodemus agrarius<br>Number detected: 306","Species: Apodemus flavicollis<br>Number detected: 55","Species: Apodemus agrarius<br>Number detected: 0","Species: Clethrionomys glareolus<br>Number detected: 2173","Species: Apodemus flavicollis<br>Number detected: 3","Species: Apodemus mystacinus<br>Number detected: 10","Species: Apodemus sylvaticus s.l.<br>Number detected: 72","Species: Microtus roberti<br>Number detected: 4","Species: Microtus rossiaemeridonalis<br>Number detected: 8","Species: Apodemus flavicollis<br>Number detected: 28","Species: Apodemus mystacinus<br>Number detected: 10","Species: Apodemus sylvaticus s.l.<br>Number detected: 82","Species: Dryomys nitedula<br>Number detected: 1","Species: Microtus roberti<br>Number detected: 11","Species: Microtus rossiaemeridonalis<br>Number detected: 7","Species: Apodemus flavicollis<br>Number detected: 0","Species: Apodemus mystacinus<br>Number detected: 7","Species: Apodemus sylvaticus s.l.<br>Number detected: 52","Species: Microtus guentheri lydius<br>Number detected: 25","Species: Microtus roberti<br>Number detected: 1","Species: Microtus rossiaemeridonalis<br>Number detected: 9","Species: Sorex caecutiens<br>Number detected: 3","Species: Sorex gracillimus<br>Number detected: 5","Species: Sorex unguiculatus<br>Number detected: 6","Species: Crocidura dsinezumi<br>Number detected: 1","Species: Urotrichis talpoides<br>Number detected: 6","Species: Apodemus agrarius<br>Number detected: 520","Species: Micromys minutus<br>Number detected: 1","Species: Mus musculus<br>Number detected: 6","Species: Rattus norvegicus<br>Number detected: 4","Species: Rattus rattus<br>Number detected: 0","Species: Tscherskia triton<br>Number detected: 0","Species: Microtus fortis<br>Number detected: 1","Species: Myodes regulus<br>Number detected: 1","Species: Crocidura lasiura<br>Number detected: 24","Species: Apodemus agrarius<br>Number detected: 647","Species: Micromys minutus<br>Number detected: 4","Species: Mus musculus<br>Number detected: 3","Species: Rattus norvegicus<br>Number detected: 11","Species: Rattus rattus<br>Number detected: 1","Species: Tscherskia triton<br>Number detected: 6","Species: Microtus fortis<br>Number detected: 2","Species: Myodes regulus<br>Number detected: 1","Species: Crocidura lasiura<br>Number detected: 56","Species: Rattus norvegicus<br>Number detected: 685","Species: Rattus rattus<br>Number detected: 40","Species: Crocidura sp.<br>Number detected: 42","Species: Dasymys rufulus<br>Number detected: 0","Species: Grammomys buntingi<br>Number detected: 1","Species: Graphiurus sp.<br>Number detected: 1","Species: Hybomys sp.<br>Number detected: 0","Species: Hylomyscus simus<br>Number detected: 8","Species: Lemniscomys striatus<br>Number detected: 1","Species: Lophuromys sikapusi<br>Number detected: 41","Species: Malacomys edwardsi<br>Number detected: 1","Species: Mastomys erythroleucus<br>Number detected: 4","Species: Mastomys natalensis<br>Number detected: 60","Species: Nannomys minutoides<br>Number detected: 4","Species: Nannomys setulosus<br>Number detected: 7","Species: Praomys rostratus<br>Number detected: 44","Species: Rattus rattus<br>Number detected: 61","Species: Uranomys ruddi<br>Number detected: 0","Species: Crocidura sp.<br>Number detected: 24","Species: Dasymys rufulus<br>Number detected: 0","Species: Grammomys buntingi<br>Number detected: 0","Species: Graphiurus sp.<br>Number detected: 0","Species: Hybomys sp.<br>Number detected: 3","Species: Hylomyscus simus<br>Number detected: 5","Species: Lemniscomys striatus<br>Number detected: 0","Species: Lophuromys sikapusi<br>Number detected: 14","Species: Malacomys edwardsi<br>Number detected: 0","Species: Mastomys erythroleucus<br>Number detected: 1","Species: Mastomys natalensis<br>Number detected: 96","Species: Nannomys minutoides<br>Number detected: 3","Species: Nannomys setulosus<br>Number detected: 2","Species: Praomys rostratus<br>Number detected: 39","Species: Rattus rattus<br>Number detected: 26","Species: Uranomys ruddi<br>Number detected: 0","Species: Crocidura sp.<br>Number detected: 27","Species: Dasymys rufulus<br>Number detected: 2","Species: Grammomys buntingi<br>Number detected: 1","Species: Graphiurus sp.<br>Number detected: 0","Species: Hybomys sp.<br>Number detected: 4","Species: Hylomyscus simus<br>Number detected: 2","Species: Lemniscomys striatus<br>Number detected: 0","Species: Lophuromys sikapusi<br>Number detected: 33","Species: Malacomys edwardsi<br>Number detected: 3","Species: Mastomys erythroleucus<br>Number detected: 2","Species: Mastomys natalensis<br>Number detected: 16","Species: Nannomys minutoides<br>Number detected: 2","Species: Nannomys setulosus<br>Number detected: 1","Species: Praomys rostratus<br>Number detected: 21","Species: Rattus rattus<br>Number detected: 40","Species: Uranomys ruddi<br>Number detected: 0","Species: Crocidura sp.<br>Number detected: 54","Species: Dasymys rufulus<br>Number detected: 0","Species: Grammomys buntingi<br>Number detected: 0","Species: Graphiurus sp.<br>Number detected: 0","Species: Hybomys sp.<br>Number detected: 3","Species: Hylomyscus simus<br>Number detected: 1","Species: Lemniscomys striatus<br>Number detected: 1","Species: Lophuromys sikapusi<br>Number detected: 25","Species: Malacomys edwardsi<br>Number detected: 4","Species: Mastomys erythroleucus<br>Number detected: 2","Species: Mastomys natalensis<br>Number detected: 100","Species: Nannomys minutoides<br>Number detected: 1","Species: Nannomys setulosus<br>Number detected: 3","Species: Praomys rostratus<br>Number detected: 102","Species: Rattus rattus<br>Number detected: 58","Species: Uranomys ruddi<br>Number detected: 0","Species: Crocidura sp.<br>Number detected: 35","Species: Dasymys rufulus<br>Number detected: 4","Species: Grammomys buntingi<br>Number detected: 0","Species: Graphiurus sp.<br>Number detected: 0","Species: Hybomys sp.<br>Number detected: 1","Species: Hylomyscus simus<br>Number detected: 1","Species: Lemniscomys striatus<br>Number detected: 1","Species: Lophuromys sikapusi<br>Number detected: 31","Species: Malacomys edwardsi<br>Number detected: 6","Species: Mastomys erythroleucus<br>Number detected: 3","Species: Mastomys natalensis<br>Number detected: 34","Species: Nannomys minutoides<br>Number detected: 0","Species: Nannomys setulosus<br>Number detected: 0","Species: Praomys rostratus<br>Number detected: 48","Species: Rattus rattus<br>Number detected: 42","Species: Uranomys ruddi<br>Number detected: 2","Species: Crocidura sp.<br>Number detected: 28","Species: Dasymys rufulus<br>Number detected: 1","Species: Grammomys buntingi<br>Number detected: 0","Species: Graphiurus sp.<br>Number detected: 0","Species: Hybomys sp.<br>Number detected: 5","Species: Hylomyscus simus<br>Number detected: 6","Species: Lemniscomys striatus<br>Number detected: 1","Species: Lophuromys sikapusi<br>Number detected: 30","Species: Malacomys edwardsi<br>Number detected: 9","Species: Mastomys erythroleucus<br>Number detected: 2","Species: Mastomys natalensis<br>Number detected: 51","Species: Nannomys minutoides<br>Number detected: 1","Species: Nannomys setulosus<br>Number detected: 11","Species: Praomys rostratus<br>Number detected: 92","Species: Rattus rattus<br>Number detected: 32","Species: Uranomys ruddi<br>Number detected: 0"],{"interactive":false,"permanent":false,"direction":"auto","opacity":1,"offset":[0,0],"textsize":"10px","textOnly":false,"className":"","sticky":true},null]},{"method":"addLegend","args":[{"colors":["#7FBFF5","#5EAEEA","#309FE0","#0090D3","#0080BD","#0072A9","#006496","#005784","#004B73","#004164","#003755","#002E48","#00263C","#021F31","#081926","#0D141B","#111111","#1C1110","#261211","#301513","#3B1917","#471E1C","#532521","#612C28","#713430","#813D38","#934641","#A6514B","#BA5C56","#CF6862","#DE7A74","#EB8E88","#F8A29E"],"labels":["Apodemus","Chaetodipus","Clethrionomys","Crocidura","Dasymys","Dipodomys","Dryomys","Grammomys","Graphiurus","Hybomys","Hylomyscus","Lemniscomys","Lophuromys","Malacomys","Mastomys","Micromys","Microtus","Mus","Myodes","Nannoyms","Neotoma","Onychomys","Peromyscus","Praomys","Rattus","Reithrodontomys","Sigmodon","Sorex","Spermophilus","Thomomys","Tscherskia","Uranomys","Urotrichus"],"na_color":null,"na_label":"NA","opacity":0.8,"position":"bottomright","type":"factor","title":"Host genus","extra":null,"layerId":null,"className":"info legend","group":null}]}],"limits":{"lat":[7.75703299999998,63.8258],"lng":[-123.1,138.252]}},"evals":[],"jsHooks":[]}</script>
<p>An interactive map displaying the location of detected small species in included studies. Selecting points will expand the number of data points at those coordinates. Point colour indicates small mammal genus with size of the point varying by the number of individuals detected. Hovering over a point or selecting it will show the species name, the number detected and the time period surveyed at the location. Data is currently shown for 9 studies.</p>
</div>
</div>
</section>
<section id="pathogen-detection" class="level3">
<h3 class="anchored" data-anchor-id="pathogen-detection">Pathogen detection</h3>
<p>A similar map can be produced to map pathogen detection with separate layers for acute infection (i.e.&nbsp;PCR positive samples) and evidence of prior infection (i.e.&nbsp;antibody positive samples)</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb4"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1" aria-hidden="true" tabindex="-1"></a>pathogen_locations <span class="ot">&lt;-</span> combined_data<span class="sc">$</span>pathogen <span class="sc">%&gt;%</span></span>
<span id="cb4-2"><a href="#cb4-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(detection_method, pathogen_family, pathogen_species, host_species, latitude, longitude, number_tested, number_positive) <span class="sc">%&gt;%</span></span>
<span id="cb4-3"><a href="#cb4-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">number_tested =</span> <span class="fu">as.numeric</span>(number_tested),</span>
<span id="cb4-4"><a href="#cb4-4" aria-hidden="true" tabindex="-1"></a>         <span class="at">number_positive =</span> <span class="fu">as.numeric</span>(number_positive)) <span class="sc">%&gt;%</span></span>
<span id="cb4-5"><a href="#cb4-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">drop_na</span>(number_tested, number_positive) <span class="sc">%&gt;%</span></span>
<span id="cb4-6"><a href="#cb4-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">st_as_sf</span>(<span class="at">coords =</span> <span class="fu">c</span>(<span class="st">"longitude"</span>, <span class="st">"latitude"</span>), <span class="at">crs =</span> project_crs) <span class="sc">%&gt;%</span></span>
<span id="cb4-7"><a href="#cb4-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(geometry, detection_method, pathogen_family, pathogen_species, host_species) <span class="sc">%&gt;%</span></span>
<span id="cb4-8"><a href="#cb4-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">number_tested =</span> <span class="fu">sum</span>(number_tested),</span>
<span id="cb4-9"><a href="#cb4-9" aria-hidden="true" tabindex="-1"></a>         <span class="at">number_positive =</span> <span class="fu">sum</span>(number_positive)) <span class="sc">%&gt;%</span></span>
<span id="cb4-10"><a href="#cb4-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ungroup</span>() <span class="sc">%&gt;%</span></span>
<span id="cb4-11"><a href="#cb4-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">distinct</span>() <span class="sc">%&gt;%</span></span>
<span id="cb4-12"><a href="#cb4-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">percent_positive =</span> <span class="fu">case_when</span>(<span class="fu">is.nan</span>(number_positive<span class="sc">/</span>number_tested <span class="sc">*</span> <span class="dv">100</span>) <span class="sc">~</span> <span class="st">"NA"</span>,</span>
<span id="cb4-13"><a href="#cb4-13" aria-hidden="true" tabindex="-1"></a>                                      number_positive <span class="sc">==</span> <span class="dv">0</span> <span class="sc">~</span> <span class="st">"NA"</span>,</span>
<span id="cb4-14"><a href="#cb4-14" aria-hidden="true" tabindex="-1"></a>                                      number_positive<span class="sc">/</span>number_tested <span class="sc">*</span> <span class="dv">100</span> <span class="sc">&lt;</span> <span class="dv">1</span> <span class="sc">~</span> <span class="st">"&lt; 1%"</span>,</span>
<span id="cb4-15"><a href="#cb4-15" aria-hidden="true" tabindex="-1"></a>                                      <span class="cn">TRUE</span> <span class="sc">~</span> <span class="fu">paste0</span>(<span class="fu">as.character</span>(<span class="fu">round</span>(number_positive<span class="sc">/</span>number_tested <span class="sc">*</span> <span class="dv">100</span>, <span class="dv">2</span>)), <span class="st">"%"</span>)),</span>
<span id="cb4-16"><a href="#cb4-16" aria-hidden="true" tabindex="-1"></a>         <span class="at">detection_method =</span> <span class="fu">case_when</span>(detection_method <span class="sc">==</span> <span class="st">"pcr"</span> <span class="sc">~</span> <span class="st">"Viral"</span>,</span>
<span id="cb4-17"><a href="#cb4-17" aria-hidden="true" tabindex="-1"></a>                                      detection_method <span class="sc">==</span> <span class="st">"antibody"</span> <span class="sc">~</span> <span class="st">"Antibody"</span>),</span>
<span id="cb4-18"><a href="#cb4-18" aria-hidden="true" tabindex="-1"></a>         <span class="at">detection =</span> <span class="fu">case_when</span>(number_positive <span class="sc">&gt;=</span> <span class="dv">1</span> <span class="sc">~</span> <span class="st">"Detected"</span>,</span>
<span id="cb4-19"><a href="#cb4-19" aria-hidden="true" tabindex="-1"></a>                               <span class="cn">TRUE</span> <span class="sc">~</span> <span class="st">"Not detected"</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb4-20"><a href="#cb4-20" aria-hidden="true" tabindex="-1"></a>  <span class="fu">rowwise</span>() <span class="sc">%&gt;%</span></span>
<span id="cb4-21"><a href="#cb4-21" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">labels =</span> <span class="fu">HTML</span>(<span class="fu">paste0</span>(<span class="st">"Pathogen family: "</span>, pathogen_family, <span class="st">"&lt;br&gt;"</span>,</span>
<span id="cb4-22"><a href="#cb4-22" aria-hidden="true" tabindex="-1"></a>                              <span class="st">"Pathogen species: "</span>, pathogen_species, <span class="st">"&lt;br&gt;"</span>,</span>
<span id="cb4-23"><a href="#cb4-23" aria-hidden="true" tabindex="-1"></a>                              <span class="st">"Host species: "</span>, host_species, <span class="st">"&lt;br&gt;"</span>,</span>
<span id="cb4-24"><a href="#cb4-24" aria-hidden="true" tabindex="-1"></a>                              <span class="st">"Number tested: "</span>, number_tested, <span class="st">"&lt;br&gt;"</span>,</span>
<span id="cb4-25"><a href="#cb4-25" aria-hidden="true" tabindex="-1"></a>                              <span class="st">"Number positive (%): "</span>, number_positive, <span class="st">" ("</span>, percent_positive, <span class="st">")"</span>, <span class="st">"&lt;br&gt;"</span>)))</span>
<span id="cb4-26"><a href="#cb4-26" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-27"><a href="#cb4-27" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-28"><a href="#cb4-28" aria-hidden="true" tabindex="-1"></a>path_pal <span class="ot">&lt;-</span> <span class="fu">colorFactor</span>(<span class="fu">diverging_hcl</span>(<span class="at">n =</span> <span class="fu">length</span>(<span class="fu">unique</span>(pathogen_locations<span class="sc">$</span>pathogen_species)), <span class="at">palette =</span> <span class="st">"Berlin"</span>),</span>
<span id="cb4-29"><a href="#cb4-29" aria-hidden="true" tabindex="-1"></a>                        <span class="at">domain =</span> <span class="fu">sort</span>(<span class="fu">unique</span>(pathogen_locations<span class="sc">$</span>pathogen_species)))</span>
<span id="cb4-30"><a href="#cb4-30" aria-hidden="true" tabindex="-1"></a>stroke_pal <span class="ot">&lt;-</span> <span class="fu">colorFactor</span>(<span class="at">palette =</span> <span class="fu">c</span>(<span class="st">"white"</span>, <span class="st">"gray"</span>), <span class="at">levels =</span> <span class="fu">c</span>(<span class="st">"Detected"</span>, <span class="st">"Not detected"</span>))</span>
<span id="cb4-31"><a href="#cb4-31" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb4-32"><a href="#cb4-32" aria-hidden="true" tabindex="-1"></a><span class="fu">leaflet</span>(<span class="at">data =</span> pathogen_locations) <span class="sc">%&gt;%</span></span>
<span id="cb4-33"><a href="#cb4-33" aria-hidden="true" tabindex="-1"></a>  <span class="fu">addTiles</span>() <span class="sc">%&gt;%</span></span>
<span id="cb4-34"><a href="#cb4-34" aria-hidden="true" tabindex="-1"></a>  <span class="fu">addCircleMarkers</span>(<span class="at">data =</span> pathogen_locations <span class="sc">%&gt;%</span></span>
<span id="cb4-35"><a href="#cb4-35" aria-hidden="true" tabindex="-1"></a>                     <span class="fu">filter</span>(detection_method <span class="sc">==</span> <span class="st">"Viral"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb4-36"><a href="#cb4-36" aria-hidden="true" tabindex="-1"></a>                     <span class="fu">filter</span>(number_tested <span class="sc">!=</span> <span class="dv">0</span>),</span>
<span id="cb4-37"><a href="#cb4-37" aria-hidden="true" tabindex="-1"></a>                   <span class="at">radius =</span> <span class="sc">~</span><span class="fu">ifelse</span>(<span class="fu">sqrt</span>(number_tested) <span class="sc">==</span> <span class="dv">0</span>, <span class="dv">1</span>, <span class="fu">sqrt</span>(number_tested)),</span>
<span id="cb4-38"><a href="#cb4-38" aria-hidden="true" tabindex="-1"></a>                   <span class="at">fill =</span> <span class="cn">TRUE</span>,</span>
<span id="cb4-39"><a href="#cb4-39" aria-hidden="true" tabindex="-1"></a>                   <span class="at">fillColor =</span> <span class="sc">~</span><span class="fu">path_pal</span>(pathogen_species),</span>
<span id="cb4-40"><a href="#cb4-40" aria-hidden="true" tabindex="-1"></a>                   <span class="at">stroke =</span> <span class="cn">TRUE</span>,</span>
<span id="cb4-41"><a href="#cb4-41" aria-hidden="true" tabindex="-1"></a>                   <span class="at">color =</span> <span class="sc">~</span><span class="fu">stroke_pal</span>(detection),</span>
<span id="cb4-42"><a href="#cb4-42" aria-hidden="true" tabindex="-1"></a>                   <span class="at">weight =</span> <span class="dv">2</span>,</span>
<span id="cb4-43"><a href="#cb4-43" aria-hidden="true" tabindex="-1"></a>                   <span class="at">opacity =</span> <span class="dv">1</span>,</span>
<span id="cb4-44"><a href="#cb4-44" aria-hidden="true" tabindex="-1"></a>                   <span class="at">fillOpacity =</span> <span class="fl">0.8</span>,</span>
<span id="cb4-45"><a href="#cb4-45" aria-hidden="true" tabindex="-1"></a>                   <span class="at">clusterOptions =</span> <span class="fu">markerClusterOptions</span>(<span class="at">spiderfyOnMaxZoom =</span> <span class="cn">TRUE</span>,</span>
<span id="cb4-46"><a href="#cb4-46" aria-hidden="true" tabindex="-1"></a>                                                         <span class="at">spiderLegPolylineOptions =</span> <span class="fu">list</span>(<span class="at">length =</span> <span class="dv">6</span>),</span>
<span id="cb4-47"><a href="#cb4-47" aria-hidden="true" tabindex="-1"></a>                                                         <span class="at">spiderfyDistanceMultiplier =</span> <span class="dv">2</span>),</span>
<span id="cb4-48"><a href="#cb4-48" aria-hidden="true" tabindex="-1"></a>                   <span class="at">label =</span> <span class="sc">~</span>labels,</span>
<span id="cb4-49"><a href="#cb4-49" aria-hidden="true" tabindex="-1"></a>                   <span class="at">popup =</span> <span class="sc">~</span>labels,</span>
<span id="cb4-50"><a href="#cb4-50" aria-hidden="true" tabindex="-1"></a>                   <span class="at">group =</span> <span class="st">"Viral detection"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb4-51"><a href="#cb4-51" aria-hidden="true" tabindex="-1"></a>  <span class="fu">addCircleMarkers</span>(<span class="at">data =</span> pathogen_locations <span class="sc">%&gt;%</span></span>
<span id="cb4-52"><a href="#cb4-52" aria-hidden="true" tabindex="-1"></a>                     <span class="fu">filter</span>(detection_method <span class="sc">==</span> <span class="st">"Antibody"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb4-53"><a href="#cb4-53" aria-hidden="true" tabindex="-1"></a>                     <span class="fu">filter</span>(number_tested <span class="sc">!=</span> <span class="dv">0</span>),</span>
<span id="cb4-54"><a href="#cb4-54" aria-hidden="true" tabindex="-1"></a>                   <span class="at">radius =</span> <span class="sc">~</span><span class="fu">ifelse</span>(<span class="fu">sqrt</span>(number_tested) <span class="sc">==</span> <span class="dv">0</span>, <span class="dv">1</span>, <span class="fu">sqrt</span>(number_tested)),</span>
<span id="cb4-55"><a href="#cb4-55" aria-hidden="true" tabindex="-1"></a>                   <span class="at">fill =</span> <span class="cn">TRUE</span>,</span>
<span id="cb4-56"><a href="#cb4-56" aria-hidden="true" tabindex="-1"></a>                   <span class="at">fillColor =</span> <span class="sc">~</span><span class="fu">path_pal</span>(pathogen_species),</span>
<span id="cb4-57"><a href="#cb4-57" aria-hidden="true" tabindex="-1"></a>                   <span class="at">stroke =</span> <span class="cn">TRUE</span>,</span>
<span id="cb4-58"><a href="#cb4-58" aria-hidden="true" tabindex="-1"></a>                   <span class="at">color =</span> <span class="sc">~</span><span class="fu">stroke_pal</span>(detection),</span>
<span id="cb4-59"><a href="#cb4-59" aria-hidden="true" tabindex="-1"></a>                   <span class="at">weight =</span> <span class="dv">2</span>,</span>
<span id="cb4-60"><a href="#cb4-60" aria-hidden="true" tabindex="-1"></a>                   <span class="at">opacity =</span> <span class="dv">1</span>,</span>
<span id="cb4-61"><a href="#cb4-61" aria-hidden="true" tabindex="-1"></a>                   <span class="at">fillOpacity =</span> <span class="fl">0.8</span>,</span>
<span id="cb4-62"><a href="#cb4-62" aria-hidden="true" tabindex="-1"></a>                   <span class="at">clusterOptions =</span> <span class="fu">markerClusterOptions</span>(<span class="at">spiderfyOnMaxZoom =</span> <span class="cn">TRUE</span>,</span>
<span id="cb4-63"><a href="#cb4-63" aria-hidden="true" tabindex="-1"></a>                                                         <span class="at">spiderLegPolylineOptions =</span> <span class="fu">list</span>(<span class="at">length =</span> <span class="dv">6</span>),</span>
<span id="cb4-64"><a href="#cb4-64" aria-hidden="true" tabindex="-1"></a>                                                         <span class="at">spiderfyDistanceMultiplier =</span> <span class="dv">2</span>),</span>
<span id="cb4-65"><a href="#cb4-65" aria-hidden="true" tabindex="-1"></a>                   <span class="at">label =</span> <span class="sc">~</span>labels,</span>
<span id="cb4-66"><a href="#cb4-66" aria-hidden="true" tabindex="-1"></a>                   <span class="at">popup =</span> <span class="sc">~</span>labels,</span>
<span id="cb4-67"><a href="#cb4-67" aria-hidden="true" tabindex="-1"></a>                   <span class="at">group =</span> <span class="st">"Antibody detection"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb4-68"><a href="#cb4-68" aria-hidden="true" tabindex="-1"></a>  <span class="fu">addLayersControl</span>(<span class="at">overlayGroups =</span> <span class="fu">c</span>(<span class="st">"Viral detection"</span>, <span class="st">"Antibody detection"</span>),</span>
<span id="cb4-69"><a href="#cb4-69" aria-hidden="true" tabindex="-1"></a>                   <span class="at">options =</span> <span class="fu">layersControlOptions</span>(<span class="at">collapsed =</span> <span class="cn">FALSE</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb4-70"><a href="#cb4-70" aria-hidden="true" tabindex="-1"></a>  <span class="fu">addLegend</span>(<span class="st">"bottomright"</span>, <span class="at">pal =</span> path_pal, <span class="at">values =</span> <span class="sc">~</span>pathogen_species, <span class="at">title =</span> <span class="st">"Pathogen species"</span>, <span class="at">opacity =</span> <span class="fl">0.8</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div id="htmlwidget-c32525d17d3a4229c07b" style="width:100%;height:464px;" class="leaflet html-widget"></div>
<script type="application/json" data-for="htmlwidget-c32525d17d3a4229c07b">{"x":{"options":{"crs":{"crsClass":"L.CRS.EPSG3857","code":null,"proj4def":null,"projectedBounds":null,"options":{}}},"calls":[{"method":"addTiles","args":["https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png",null,null,{"minZoom":0,"maxZoom":18,"tileSize":256,"subdomains":"abc","errorTileUrl":"","tms":false,"noWrap":false,"zoomOffset":0,"zoomReverse":false,"opacity":1,"zIndex":1,"detectRetina":false,"attribution":"&copy; <a href=\"https://openstreetmap.org\">OpenStreetMap<\/a> contributors, <a href=\"https://creativecommons.org/licenses/by-sa/2.0/\">CC-BY-SA<\/a>"}]},{"method":"addCircleMarkers","args":[[48.719,48.719,36.204,36.204,36.204,36.204,36.204,34.470556,34.470219,49.283333,7.945028,7.945028,7.945028,7.945028,7.945028,7.945028,7.945028,7.945028,7.945028,7.945028,7.945028,7.945028,7.945028,8.005569,8.005569,8.005569,8.005569,8.005569,8.005569,8.005569,8.005569,8.005569,8.005569,7.757033,7.757033,7.757033,7.757033,7.757033,7.75703299999999,7.75703299999999,7.75703299999999,7.75703299999999,7.75703299999999,7.75703299999999,7.75703299999999,7.75703299999999,7.762217,7.762217,7.762217,7.762217,7.762217,7.762217,7.762217,7.762217,7.762217,7.762217,7.762217,7.762217,7.771753,7.771753,7.771753,7.771753,7.771753,7.771753,7.771753,7.771753,7.771753,7.771753,7.771753,7.771753,7.973069,7.973069,7.973069,7.973069,7.973069,7.973069,7.973069,7.973069,7.973069,7.973069,7.973069,7.973069,7.973069],[21.2611,21.2611,138.252,138.252,138.252,138.252,138.252,136.762833,136.762725,-123.1,-11.431461,-11.431461,-11.431461,-11.431461,-11.431461,-11.431461,-11.431461,-11.431461,-11.431461,-11.431461,-11.431461,-11.431461,-11.431461,-11.427842,-11.427842,-11.427842,-11.427842,-11.427842,-11.427842,-11.427842,-11.427842,-11.427842,-11.427842,-11.770189,-11.770189,-11.770189,-11.770189,-11.770189,-11.770189,-11.770189,-11.770189,-11.770189,-11.770189,-11.770189,-11.770189,-11.770189,-11.742564,-11.742564,-11.742564,-11.742564,-11.742564,-11.742564,-11.742564,-11.742564,-11.742564,-11.742564,-11.742564,-11.742564,-11.726172,-11.726172,-11.726172,-11.726172,-11.726172,-11.726172,-11.726172,-11.726172,-11.726172,-11.726172,-11.726172,-11.726172,-11.3957,-11.3957,-11.3957,-11.3957,-11.3957,-11.3957,-11.3957,-11.3957,-11.3957,-11.3957,-11.3957,-11.3957,-11.3957],[6.48074069840786,1.4142135623731,1.73205080756888,2.23606797749979,2.44948974278318,1,1,1.4142135623731,1.73205080756888,5.47722557505166,6.48074069840786,1,1,2.82842712474619,1,6.40312423743285,1,2,7.74596669241483,2,2.64575131106459,6.6332495807108,7.81024967590665,4.89897948556636,1.73205080756888,2.23606797749979,3.74165738677394,1,9.79795897113271,1.73205080756888,1.4142135623731,6.2449979983984,5.09901951359278,5.19615242270663,1.4142135623731,1,2,1.4142135623731,5.74456264653803,1.73205080756888,1.4142135623731,4,1.4142135623731,1,4.58257569495584,6.32455532033676,7.34846922834953,1.73205080756888,1,1,5,2,1.4142135623731,10,1,1.73205080756888,10.0995049383621,7.61577310586391,5.91607978309962,2,1,1,1,5.56776436283002,2.44948974278318,1.73205080756888,5.8309518948453,6.92820323027551,6.48074069840786,1.4142135623731,5.29150262212918,1,2.23606797749979,2.44948974278318,1,5.47722557505166,3,1.4142135623731,7.14142842854285,1,3.3166247903554,9.59166304662544,5.65685424949238],null,"Viral detection",{"interactive":true,"className":"","stroke":true,"color":["#FFFFFF","#FFFFFF","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#FFFFFF","#FFFFFF","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#FFFFFF","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#FFFFFF","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE"],"weight":2,"opacity":1,"fill":true,"fillColor":["#0087C7","#0087C7","#7FBFF5","#7FBFF5","#7FBFF5","#7FBFF5","#7FBFF5","#7FBFF5","#7FBFF5","#C3625B","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C"],"fillOpacity":0.8},{"showCoverageOnHover":true,"zoomToBoundsOnClick":true,"spiderfyOnMaxZoom":true,"removeOutsideVisibleBounds":true,"spiderLegPolylineOptions":{"length":6},"freezeAtZoom":false,"spiderfyDistanceMultiplier":2},null,["Pathogen family: Hantaviridae<br>Pathogen species: Dobrava-Belgrade orthohantavirus<br>Host species: Apodemus agrarius<br>Number tested: 42<br>Number positive (%): 8 (19.05%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Dobrava-Belgrade orthohantavirus<br>Host species: Apodemus flavicollis<br>Number tested: 2<br>Number positive (%): 1 (50%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Asama virus<br>Host species: Sorex caecutiens<br>Number tested: 3<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Asama virus<br>Host species: Sorex gracillimus<br>Number tested: 5<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Asama virus<br>Host species: Sorex unguiculatus<br>Number tested: 6<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Asama virus<br>Host species: Crocidura dsinezumi<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Asama virus<br>Host species: Urotrichis talpoides<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Asama virus<br>Host species: Urotrichis talpoides<br>Number tested: 2<br>Number positive (%): 1 (50%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Asama virus<br>Host species: Urotrichis talpoides<br>Number tested: 3<br>Number positive (%): 2 (66.67%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Seoul hantavirus<br>Host species: Rattus sp.<br>Number tested: 30<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Crocidura sp.<br>Number tested: 42<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Grammomys buntingi<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Graphiurus sp.<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hylomyscus simus<br>Number tested: 8<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lemniscomys striatus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lophuromys sikapusi<br>Number tested: 41<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Malacomys edwardsi<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys erythroleucus<br>Number tested: 4<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys natalensis<br>Number tested: 60<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys minutoides<br>Number tested: 4<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys setulosus<br>Number tested: 7<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Praomys rostratus<br>Number tested: 44<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Rattus rattus<br>Number tested: 61<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Crocidura sp.<br>Number tested: 24<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hybomys sp.<br>Number tested: 3<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hylomyscus simus<br>Number tested: 5<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lophuromys sikapusi<br>Number tested: 14<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys erythroleucus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys natalensis<br>Number tested: 96<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys minutoides<br>Number tested: 3<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys setulosus<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Praomys rostratus<br>Number tested: 39<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Rattus rattus<br>Number tested: 26<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Crocidura sp.<br>Number tested: 27<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Dasymys rufulus<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Grammomys buntingi<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hybomys sp.<br>Number tested: 4<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hylomyscus simus<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lophuromys sikapusi<br>Number tested: 33<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Malacomys edwardsi<br>Number tested: 3<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys erythroleucus<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys natalensis<br>Number tested: 16<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys minutoides<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys setulosus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Praomys rostratus<br>Number tested: 21<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Rattus rattus<br>Number tested: 40<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Crocidura sp.<br>Number tested: 54<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hybomys sp.<br>Number tested: 3<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hylomyscus simus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lemniscomys striatus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lophuromys sikapusi<br>Number tested: 25<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Malacomys edwardsi<br>Number tested: 4<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys erythroleucus<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys natalensis<br>Number tested: 100<br>Number positive (%): 1 (1%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys minutoides<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys setulosus<br>Number tested: 3<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Praomys rostratus<br>Number tested: 102<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Rattus rattus<br>Number tested: 58<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Crocidura sp.<br>Number tested: 35<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Dasymys rufulus<br>Number tested: 4<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hybomys sp.<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hylomyscus simus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lemniscomys striatus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lophuromys sikapusi<br>Number tested: 31<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Malacomys edwardsi<br>Number tested: 6<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys erythroleucus<br>Number tested: 3<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys natalensis<br>Number tested: 34<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Praomys rostratus<br>Number tested: 48<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Rattus rattus<br>Number tested: 42<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Uranomys ruddi<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Crocidura sp.<br>Number tested: 28<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Dasymys rufulus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hybomys sp.<br>Number tested: 5<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hylomyscus simus<br>Number tested: 6<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lemniscomys striatus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lophuromys sikapusi<br>Number tested: 30<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Malacomys edwardsi<br>Number tested: 9<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys erythroleucus<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys natalensis<br>Number tested: 51<br>Number positive (%): 3 (5.88%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys minutoides<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys setulosus<br>Number tested: 11<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Praomys rostratus<br>Number tested: 92<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Rattus rattus<br>Number tested: 32<br>Number positive (%): 0 (NA)<br>"],null,["Pathogen family: Hantaviridae<br>Pathogen species: Dobrava-Belgrade orthohantavirus<br>Host species: Apodemus agrarius<br>Number tested: 42<br>Number positive (%): 8 (19.05%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Dobrava-Belgrade orthohantavirus<br>Host species: Apodemus flavicollis<br>Number tested: 2<br>Number positive (%): 1 (50%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Asama virus<br>Host species: Sorex caecutiens<br>Number tested: 3<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Asama virus<br>Host species: Sorex gracillimus<br>Number tested: 5<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Asama virus<br>Host species: Sorex unguiculatus<br>Number tested: 6<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Asama virus<br>Host species: Crocidura dsinezumi<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Asama virus<br>Host species: Urotrichis talpoides<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Asama virus<br>Host species: Urotrichis talpoides<br>Number tested: 2<br>Number positive (%): 1 (50%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Asama virus<br>Host species: Urotrichis talpoides<br>Number tested: 3<br>Number positive (%): 2 (66.67%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Seoul hantavirus<br>Host species: Rattus sp.<br>Number tested: 30<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Crocidura sp.<br>Number tested: 42<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Grammomys buntingi<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Graphiurus sp.<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hylomyscus simus<br>Number tested: 8<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lemniscomys striatus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lophuromys sikapusi<br>Number tested: 41<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Malacomys edwardsi<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys erythroleucus<br>Number tested: 4<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys natalensis<br>Number tested: 60<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys minutoides<br>Number tested: 4<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys setulosus<br>Number tested: 7<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Praomys rostratus<br>Number tested: 44<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Rattus rattus<br>Number tested: 61<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Crocidura sp.<br>Number tested: 24<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hybomys sp.<br>Number tested: 3<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hylomyscus simus<br>Number tested: 5<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lophuromys sikapusi<br>Number tested: 14<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys erythroleucus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys natalensis<br>Number tested: 96<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys minutoides<br>Number tested: 3<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys setulosus<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Praomys rostratus<br>Number tested: 39<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Rattus rattus<br>Number tested: 26<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Crocidura sp.<br>Number tested: 27<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Dasymys rufulus<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Grammomys buntingi<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hybomys sp.<br>Number tested: 4<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hylomyscus simus<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lophuromys sikapusi<br>Number tested: 33<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Malacomys edwardsi<br>Number tested: 3<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys erythroleucus<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys natalensis<br>Number tested: 16<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys minutoides<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys setulosus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Praomys rostratus<br>Number tested: 21<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Rattus rattus<br>Number tested: 40<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Crocidura sp.<br>Number tested: 54<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hybomys sp.<br>Number tested: 3<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hylomyscus simus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lemniscomys striatus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lophuromys sikapusi<br>Number tested: 25<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Malacomys edwardsi<br>Number tested: 4<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys erythroleucus<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys natalensis<br>Number tested: 100<br>Number positive (%): 1 (1%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys minutoides<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys setulosus<br>Number tested: 3<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Praomys rostratus<br>Number tested: 102<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Rattus rattus<br>Number tested: 58<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Crocidura sp.<br>Number tested: 35<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Dasymys rufulus<br>Number tested: 4<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hybomys sp.<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hylomyscus simus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lemniscomys striatus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lophuromys sikapusi<br>Number tested: 31<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Malacomys edwardsi<br>Number tested: 6<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys erythroleucus<br>Number tested: 3<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys natalensis<br>Number tested: 34<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Praomys rostratus<br>Number tested: 48<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Rattus rattus<br>Number tested: 42<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Uranomys ruddi<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Crocidura sp.<br>Number tested: 28<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Dasymys rufulus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hybomys sp.<br>Number tested: 5<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hylomyscus simus<br>Number tested: 6<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lemniscomys striatus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lophuromys sikapusi<br>Number tested: 30<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Malacomys edwardsi<br>Number tested: 9<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys erythroleucus<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys natalensis<br>Number tested: 51<br>Number positive (%): 3 (5.88%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys minutoides<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys setulosus<br>Number tested: 11<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Praomys rostratus<br>Number tested: 92<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Rattus rattus<br>Number tested: 32<br>Number positive (%): 0 (NA)<br>"],{"interactive":false,"permanent":false,"direction":"auto","opacity":1,"offset":[0,0],"textsize":"10px","textOnly":false,"className":"","sticky":true},null]},{"method":"addCircleMarkers","args":[[31.833135,31.833135,31.833135,31.833135,31.833135,31.833135,31.833135,31.833135,31.833135,31.833135,33.446953,33.446953,33.446953,33.446953,33.446953,33.446953,33.446953,33.446953,33.446953,33.446953,33.446953,33.446953,33.446953,33.446953,33.4469529999999,33.4469529999999,33.4469529999999,33.4469529999999,33.4469529999999,33.4469529999998,33.4469529999998,33.4469529999998,33.4469529999998,33.4469529999998,33.4469529999998,33.4469529999998,33.4469529999997,33.4469529999997,33.4469529999997,33.4469529999997,33.4469529999997,33.4469529999997,48.719,48.719,63.8258,40.75,40.75,40.75,40.75,40.75,41,41,41,41,41,41,41.0167,41.0167,41.0167,41.0167,41.0167,40.75,40.75,40.75,40.75,40.75,41,41,41,41,41,41,41.0167,41.0167,41.0167,41.0167,41.0167,38.11002,38.11002,38.11002,38.11002,38.11002,38.11002,38.11002,38.051592,38.051592,38.051592,38.051592,38.051592,38.051592,38.051592,38.051592,38.051592,49.283333,7.945028,7.945028,7.945028,7.945028,7.945028,7.945028,7.945028,7.945028,7.945028,7.945028,7.945028,7.945028,7.945028,8.005569,8.005569,8.005569,8.005569,8.005569,8.005569,8.005569,8.005569,8.005569,8.005569,7.757033,7.757033,7.757033,7.757033,7.757033,7.75703299999999,7.75703299999999,7.75703299999999,7.75703299999999,7.75703299999999,7.75703299999999,7.75703299999999,7.75703299999999,7.762217,7.762217,7.762217,7.762217,7.762217,7.762217,7.762217,7.762217,7.762217,7.762217,7.762217,7.762217,7.771753,7.771753,7.771753,7.771753,7.771753,7.771753,7.771753,7.771753,7.771753,7.771753,7.771753,7.771753,7.973069,7.973069,7.973069,7.973069,7.973069,7.973069,7.973069,7.973069,7.973069,7.973069,7.973069,7.973069,7.973069],[-110.852966,-110.852966,-110.852966,-110.852966,-110.852966,-110.852966,-110.852966,-110.852966,-110.852966,-110.852966,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,-117.622952,21.2611,21.2611,20.263,39.616,39.616,39.616,39.616,39.616,41.05,41.05,41.05,41.05,41.05,41.05,40.983,40.983,40.983,40.983,40.983,39.616,39.616,39.616,39.616,39.616,41.05,41.05,41.05,41.05,41.05,41.05,40.983,40.983,40.983,40.983,40.983,127.07705,127.07705,127.07705,127.07705,127.07705,127.07705,127.07705,127.1069191,127.1069191,127.1069191,127.1069191,127.1069191,127.1069191,127.1069191,127.1069191,127.1069191,-123.1,-11.431461,-11.431461,-11.431461,-11.431461,-11.431461,-11.431461,-11.431461,-11.431461,-11.431461,-11.431461,-11.431461,-11.431461,-11.431461,-11.427842,-11.427842,-11.427842,-11.427842,-11.427842,-11.427842,-11.427842,-11.427842,-11.427842,-11.427842,-11.770189,-11.770189,-11.770189,-11.770189,-11.770189,-11.770189,-11.770189,-11.770189,-11.770189,-11.770189,-11.770189,-11.770189,-11.770189,-11.742564,-11.742564,-11.742564,-11.742564,-11.742564,-11.742564,-11.742564,-11.742564,-11.742564,-11.742564,-11.742564,-11.742564,-11.726172,-11.726172,-11.726172,-11.726172,-11.726172,-11.726172,-11.726172,-11.726172,-11.726172,-11.726172,-11.726172,-11.726172,-11.3957,-11.3957,-11.3957,-11.3957,-11.3957,-11.3957,-11.3957,-11.3957,-11.3957,-11.3957,-11.3957,-11.3957,-11.3957],[18.1383571472171,7.14142842854285,10.8627804912002,9.89949493661167,1.4142135623731,4.58257569495584,3.46410161513775,2.23606797749979,2.64575131106459,2.64575131106459,3.46410161513775,1.4142135623731,4.12310562561766,5.29150262212918,8.94427190999916,7.48331477354788,6.08276253029822,13.6747943311773,11.5758369027902,22.4722050542442,14.1067359796659,1,1.73205080756888,1,1.4142135623731,5.65685424949238,6.6332495807108,3.74165738677394,2.44948974278318,2.82842712474619,4.58257569495584,8.77496438739212,2.64575131106459,5.65685424949238,1.73205080756888,4.69041575982343,6.40312423743285,18.5741756210067,13.8202749610853,6.78232998312527,1.4142135623731,4,17.4928556845359,7.41619848709566,46.6154480832267,1.73205080756888,3.16227766016838,8.48528137423857,2,2.82842712474619,5.29150262212918,3.16227766016838,9.05538513813742,1,3.3166247903554,2.64575131106459,2.64575131106459,7.21110255092798,5,1,3,1.73205080756888,3.16227766016838,8.48528137423857,2,2.82842712474619,5.29150262212918,3.16227766016838,9.05538513813742,1,3.3166247903554,2.64575131106459,2.64575131106459,7.21110255092798,5,1,3,22.8035085019828,1,2.44948974278318,2,1,1,4.89897948556636,25.4361946839538,2,1.73205080756888,3.3166247903554,1,2.44948974278318,1.4142135623731,1,7.48331477354788,23.5159520326097,6.48074069840786,1,1,2.82842712474619,1,6.40312423743285,1,2,7.74596669241483,2,2.64575131106459,6.6332495807108,7.81024967590665,4.89897948556636,1.73205080756888,2.23606797749979,3.74165738677394,1,9.79795897113271,1.73205080756888,1.4142135623731,6.2449979983984,5.09901951359278,5.19615242270663,1.4142135623731,1,2,1.4142135623731,5.74456264653803,1.73205080756888,1.4142135623731,4,1.4142135623731,1,4.58257569495584,6.32455532033676,7.34846922834953,1.73205080756888,1,1,5,2,1.4142135623731,10,1,1.73205080756888,10.0995049383621,7.61577310586391,5.91607978309962,2,1,1,1,5.56776436283002,2.44948974278318,1.73205080756888,5.8309518948453,6.92820323027551,6.48074069840786,1.4142135623731,5.29150262212918,1,2.23606797749979,2.44948974278318,1,5.47722557505166,3,1.4142135623731,7.14142842854285,1,3.3166247903554,9.59166304662544,5.65685424949238],null,"Antibody detection",{"interactive":true,"className":"","stroke":true,"color":["#BEBEBE","#BEBEBE","#FFFFFF","#FFFFFF","#BEBEBE","#FFFFFF","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#FFFFFF","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#FFFFFF","#FFFFFF","#FFFFFF","#FFFFFF","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#FFFFFF","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#FFFFFF","#BEBEBE","#FFFFFF","#FFFFFF","#BEBEBE","#BEBEBE","#BEBEBE","#FFFFFF","#FFFFFF","#FFFFFF","#BEBEBE","#BEBEBE","#FFFFFF","#BEBEBE","#FFFFFF","#FFFFFF","#FFFFFF","#FFFFFF","#BEBEBE","#FFFFFF","#BEBEBE","#BEBEBE","#FFFFFF","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#FFFFFF","#FFFFFF","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#FFFFFF","#BEBEBE","#BEBEBE","#FFFFFF","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#FFFFFF","#FFFFFF","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#FFFFFF","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#FFFFFF","#BEBEBE","#BEBEBE","#FFFFFF","#BEBEBE","#BEBEBE","#BEBEBE","#FFFFFF","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#FFFFFF","#BEBEBE","#BEBEBE","#FFFFFF","#FFFFFF","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#FFFFFF","#BEBEBE","#FFFFFF","#BEBEBE","#BEBEBE","#FFFFFF","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#FFFFFF","#BEBEBE","#BEBEBE","#FFFFFF","#BEBEBE","#BEBEBE","#FFFFFF","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#FFFFFF","#FFFFFF","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE","#FFFFFF","#BEBEBE","#BEBEBE","#BEBEBE","#BEBEBE"],"weight":2,"opacity":1,"fill":true,"fillColor":["#F8A29E","#F8A29E","#F8A29E","#F8A29E","#F8A29E","#F8A29E","#F8A29E","#F8A29E","#F8A29E","#F8A29E","#241211","#241211","#241211","#241211","#241211","#241211","#241211","#241211","#241211","#241211","#241211","#241211","#241211","#241211","#241211","#241211","#241211","#241211","#241211","#241211","#241211","#241211","#241211","#241211","#241211","#241211","#241211","#241211","#241211","#241211","#241211","#241211","#0087C7","#0087C7","#4B201D","#091824","#091824","#091824","#091824","#091824","#091824","#091824","#091824","#091824","#091824","#091824","#091824","#091824","#091824","#091824","#091824","#7F3C37","#7F3C37","#7F3C37","#7F3C37","#7F3C37","#7F3C37","#7F3C37","#7F3C37","#7F3C37","#7F3C37","#7F3C37","#7F3C37","#7F3C37","#7F3C37","#7F3C37","#7F3C37","#005682","#005682","#005682","#005682","#005682","#005682","#005682","#005682","#005682","#005682","#005682","#005682","#005682","#005682","#005682","#005682","#C3625B","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C","#00314C"],"fillOpacity":0.8},{"showCoverageOnHover":true,"zoomToBoundsOnClick":true,"spiderfyOnMaxZoom":true,"removeOutsideVisibleBounds":true,"spiderLegPolylineOptions":{"length":6},"freezeAtZoom":false,"spiderfyDistanceMultiplier":2},null,["Pathogen family: Hantaviridae<br>Pathogen species: Sin Nombre orthohantavirus<br>Host species: Chaetodipus baileyi<br>Number tested: 329<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Sin Nombre orthohantavirus<br>Host species: Neotoma albigula<br>Number tested: 51<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Sin Nombre orthohantavirus<br>Host species: Peromyscus eremicus<br>Number tested: 118<br>Number positive (%): 1 (< 1%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Sin Nombre orthohantavirus<br>Host species: Peromyscus boylii<br>Number tested: 98<br>Number positive (%): 13 (13.27%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Sin Nombre orthohantavirus<br>Host species: Peromyscus maniculatus<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Sin Nombre orthohantavirus<br>Host species: Peromyscus leucopus<br>Number tested: 21<br>Number positive (%): 3 (14.29%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Sin Nombre orthohantavirus<br>Host species: Reithrodontomys fulvescens<br>Number tested: 12<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Sin Nombre orthohantavirus<br>Host species: Sigmodon ochrognathus<br>Number tested: 5<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Sin Nombre orthohantavirus<br>Host species: Chaetodipus penicillatus<br>Number tested: 7<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Sin Nombre orthohantavirus<br>Host species: Onychomys torridue<br>Number tested: 7<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Chaetodipus californicus<br>Number tested: 12<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Chaetodipus fallax<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Microtus californicus<br>Number tested: 17<br>Number positive (%): 1 (5.88%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Mus musculus<br>Number tested: 28<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Neotoma fiscipes<br>Number tested: 80<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Neotoma lepida<br>Number tested: 56<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Peromyscus boylii<br>Number tested: 37<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Peromyscus californicus<br>Number tested: 187<br>Number positive (%): 29 (15.51%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Peromyscus eremicus<br>Number tested: 134<br>Number positive (%): 1 (< 1%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Peromyscus maniculatus<br>Number tested: 505<br>Number positive (%): 49 (9.7%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Reithrodontomys megalotis<br>Number tested: 199<br>Number positive (%): 36 (18.09%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Rattus rattus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Spermophilus beecheyi<br>Number tested: 3<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Thomomys bottae<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Microtus californicus<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Mus musculus<br>Number tested: 32<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Peromyscus maniculatus<br>Number tested: 44<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Reithrodontomys megalotis<br>Number tested: 14<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Rattus rattus<br>Number tested: 6<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Chaetodipus californicus<br>Number tested: 8<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Microtus californicus<br>Number tested: 21<br>Number positive (%): 4 (19.05%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Mus musculus<br>Number tested: 77<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Neotoma fiscipes<br>Number tested: 7<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Neotoma lepida<br>Number tested: 32<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Peromyscus boylii<br>Number tested: 3<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Peromyscus californicus<br>Number tested: 22<br>Number positive (%): 1 (4.55%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Peromyscus eremicus<br>Number tested: 41<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Peromyscus maniculatus<br>Number tested: 345<br>Number positive (%): 37 (10.72%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Reithrodontomys megalotis<br>Number tested: 191<br>Number positive (%): 13 (6.81%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Rattus rattus<br>Number tested: 46<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Rattus norvegicus<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Spermophilus beecheyi<br>Number tested: 16<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Dobrava-Belgrade orthohantavirus<br>Host species: Apodemus agrarius<br>Number tested: 306<br>Number positive (%): 42 (13.73%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Dobrava-Belgrade orthohantavirus<br>Host species: Apodemus flavicollis<br>Number tested: 55<br>Number positive (%): 2 (3.64%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Puumala virus<br>Host species: Clethrionomys glareolus<br>Number tested: 2173<br>Number positive (%): 382 (17.58%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Apodemus flavicollis<br>Number tested: 3<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Apodemus mystacinus<br>Number tested: 10<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Apodemus sylvaticus s.l.<br>Number tested: 72<br>Number positive (%): 2 (2.78%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Microtus roberti<br>Number tested: 4<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Microtus rossiaemeridonalis<br>Number tested: 8<br>Number positive (%): 1 (12.5%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Apodemus flavicollis<br>Number tested: 28<br>Number positive (%): 1 (3.57%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Apodemus mystacinus<br>Number tested: 10<br>Number positive (%): 1 (10%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Apodemus sylvaticus s.l.<br>Number tested: 82<br>Number positive (%): 1 (1.22%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Dryomys nitedula<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Microtus roberti<br>Number tested: 11<br>Number positive (%): 1 (9.09%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Microtus rossiaemeridonalis<br>Number tested: 7<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Apodemus mystacinus<br>Number tested: 7<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Apodemus sylvaticus s.l.<br>Number tested: 52<br>Number positive (%): 1 (1.92%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Microtus guentheri lydius<br>Number tested: 25<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Microtus roberti<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Microtus rossiaemeridonalis<br>Number tested: 9<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Puumala virus/Saaremaa virus<br>Host species: Apodemus flavicollis<br>Number tested: 3<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Puumala virus/Saaremaa virus<br>Host species: Apodemus mystacinus<br>Number tested: 10<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Puumala virus/Saaremaa virus<br>Host species: Apodemus sylvaticus s.l.<br>Number tested: 72<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Puumala virus/Saaremaa virus<br>Host species: Microtus roberti<br>Number tested: 4<br>Number positive (%): 1 (25%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Puumala virus/Saaremaa virus<br>Host species: Microtus rossiaemeridonalis<br>Number tested: 8<br>Number positive (%): 2 (25%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Puumala virus/Saaremaa virus<br>Host species: Apodemus flavicollis<br>Number tested: 28<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Puumala virus/Saaremaa virus<br>Host species: Apodemus mystacinus<br>Number tested: 10<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Puumala virus/Saaremaa virus<br>Host species: Apodemus sylvaticus s.l.<br>Number tested: 82<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Puumala virus/Saaremaa virus<br>Host species: Dryomys nitedula<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Puumala virus/Saaremaa virus<br>Host species: Microtus roberti<br>Number tested: 11<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Puumala virus/Saaremaa virus<br>Host species: Microtus rossiaemeridonalis<br>Number tested: 7<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Puumala virus/Saaremaa virus<br>Host species: Apodemus mystacinus<br>Number tested: 7<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Puumala virus/Saaremaa virus<br>Host species: Apodemus sylvaticus s.l.<br>Number tested: 52<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Puumala virus/Saaremaa virus<br>Host species: Microtus guentheri lydius<br>Number tested: 25<br>Number positive (%): 1 (4%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Puumala virus/Saaremaa virus<br>Host species: Microtus roberti<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Puumala virus/Saaremaa virus<br>Host species: Microtus rossiaemeridonalis<br>Number tested: 9<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Hantaan virus<br>Host species: Apodemus agrarius<br>Number tested: 520<br>Number positive (%): 121 (23.27%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Hantaan virus<br>Host species: Micromys minutus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Hantaan virus<br>Host species: Mus musculus<br>Number tested: 6<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Hantaan virus<br>Host species: Rattus norvegicus<br>Number tested: 4<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Hantaan virus<br>Host species: Microtus fortis<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Hantaan virus<br>Host species: Myodes regulus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Hantaan virus<br>Host species: Crocidura lasiura<br>Number tested: 24<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Hantaan virus<br>Host species: Apodemus agrarius<br>Number tested: 647<br>Number positive (%): 94 (14.53%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Hantaan virus<br>Host species: Micromys minutus<br>Number tested: 4<br>Number positive (%): 1 (25%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Hantaan virus<br>Host species: Mus musculus<br>Number tested: 3<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Hantaan virus<br>Host species: Rattus norvegicus<br>Number tested: 11<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Hantaan virus<br>Host species: Rattus rattus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Hantaan virus<br>Host species: Tscherskia triton<br>Number tested: 6<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Hantaan virus<br>Host species: Microtus fortis<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Hantaan virus<br>Host species: Myodes regulus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Hantaan virus<br>Host species: Crocidura lasiura<br>Number tested: 56<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Seoul hantavirus<br>Host species: Rattus sp.<br>Number tested: 553<br>Number positive (%): 6 (1.08%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Crocidura sp.<br>Number tested: 42<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Grammomys buntingi<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Graphiurus sp.<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hylomyscus simus<br>Number tested: 8<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lemniscomys striatus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lophuromys sikapusi<br>Number tested: 41<br>Number positive (%): 1 (2.44%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Malacomys edwardsi<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys erythroleucus<br>Number tested: 4<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys natalensis<br>Number tested: 60<br>Number positive (%): 5 (8.33%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys minutoides<br>Number tested: 4<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys setulosus<br>Number tested: 7<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Praomys rostratus<br>Number tested: 44<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Rattus rattus<br>Number tested: 61<br>Number positive (%): 1 (1.64%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Crocidura sp.<br>Number tested: 24<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hybomys sp.<br>Number tested: 3<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hylomyscus simus<br>Number tested: 5<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lophuromys sikapusi<br>Number tested: 14<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys erythroleucus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys natalensis<br>Number tested: 96<br>Number positive (%): 2 (2.08%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys minutoides<br>Number tested: 3<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys setulosus<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Praomys rostratus<br>Number tested: 39<br>Number positive (%): 1 (2.56%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Rattus rattus<br>Number tested: 26<br>Number positive (%): 1 (3.85%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Crocidura sp.<br>Number tested: 27<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Dasymys rufulus<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Grammomys buntingi<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hybomys sp.<br>Number tested: 4<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hylomyscus simus<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lophuromys sikapusi<br>Number tested: 33<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Malacomys edwardsi<br>Number tested: 3<br>Number positive (%): 1 (33.33%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys erythroleucus<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys natalensis<br>Number tested: 16<br>Number positive (%): 2 (12.5%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys minutoides<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys setulosus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Praomys rostratus<br>Number tested: 21<br>Number positive (%): 3 (14.29%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Rattus rattus<br>Number tested: 40<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Crocidura sp.<br>Number tested: 54<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hybomys sp.<br>Number tested: 3<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hylomyscus simus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lemniscomys striatus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lophuromys sikapusi<br>Number tested: 25<br>Number positive (%): 1 (4%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Malacomys edwardsi<br>Number tested: 4<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys erythroleucus<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys natalensis<br>Number tested: 100<br>Number positive (%): 12 (12%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys minutoides<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys setulosus<br>Number tested: 3<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Praomys rostratus<br>Number tested: 102<br>Number positive (%): 1 (< 1%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Rattus rattus<br>Number tested: 58<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Crocidura sp.<br>Number tested: 35<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Dasymys rufulus<br>Number tested: 4<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hybomys sp.<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hylomyscus simus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lemniscomys striatus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lophuromys sikapusi<br>Number tested: 31<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Malacomys edwardsi<br>Number tested: 6<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys erythroleucus<br>Number tested: 3<br>Number positive (%): 2 (66.67%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys natalensis<br>Number tested: 34<br>Number positive (%): 3 (8.82%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Praomys rostratus<br>Number tested: 48<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Rattus rattus<br>Number tested: 42<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Uranomys ruddi<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Crocidura sp.<br>Number tested: 28<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Dasymys rufulus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hybomys sp.<br>Number tested: 5<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hylomyscus simus<br>Number tested: 6<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lemniscomys striatus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lophuromys sikapusi<br>Number tested: 30<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Malacomys edwardsi<br>Number tested: 9<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys erythroleucus<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys natalensis<br>Number tested: 51<br>Number positive (%): 5 (9.8%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys minutoides<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys setulosus<br>Number tested: 11<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Praomys rostratus<br>Number tested: 92<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Rattus rattus<br>Number tested: 32<br>Number positive (%): 0 (NA)<br>"],null,["Pathogen family: Hantaviridae<br>Pathogen species: Sin Nombre orthohantavirus<br>Host species: Chaetodipus baileyi<br>Number tested: 329<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Sin Nombre orthohantavirus<br>Host species: Neotoma albigula<br>Number tested: 51<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Sin Nombre orthohantavirus<br>Host species: Peromyscus eremicus<br>Number tested: 118<br>Number positive (%): 1 (< 1%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Sin Nombre orthohantavirus<br>Host species: Peromyscus boylii<br>Number tested: 98<br>Number positive (%): 13 (13.27%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Sin Nombre orthohantavirus<br>Host species: Peromyscus maniculatus<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Sin Nombre orthohantavirus<br>Host species: Peromyscus leucopus<br>Number tested: 21<br>Number positive (%): 3 (14.29%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Sin Nombre orthohantavirus<br>Host species: Reithrodontomys fulvescens<br>Number tested: 12<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Sin Nombre orthohantavirus<br>Host species: Sigmodon ochrognathus<br>Number tested: 5<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Sin Nombre orthohantavirus<br>Host species: Chaetodipus penicillatus<br>Number tested: 7<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Sin Nombre orthohantavirus<br>Host species: Onychomys torridue<br>Number tested: 7<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Chaetodipus californicus<br>Number tested: 12<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Chaetodipus fallax<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Microtus californicus<br>Number tested: 17<br>Number positive (%): 1 (5.88%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Mus musculus<br>Number tested: 28<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Neotoma fiscipes<br>Number tested: 80<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Neotoma lepida<br>Number tested: 56<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Peromyscus boylii<br>Number tested: 37<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Peromyscus californicus<br>Number tested: 187<br>Number positive (%): 29 (15.51%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Peromyscus eremicus<br>Number tested: 134<br>Number positive (%): 1 (< 1%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Peromyscus maniculatus<br>Number tested: 505<br>Number positive (%): 49 (9.7%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Reithrodontomys megalotis<br>Number tested: 199<br>Number positive (%): 36 (18.09%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Rattus rattus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Spermophilus beecheyi<br>Number tested: 3<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Thomomys bottae<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Microtus californicus<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Mus musculus<br>Number tested: 32<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Peromyscus maniculatus<br>Number tested: 44<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Reithrodontomys megalotis<br>Number tested: 14<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Rattus rattus<br>Number tested: 6<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Chaetodipus californicus<br>Number tested: 8<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Microtus californicus<br>Number tested: 21<br>Number positive (%): 4 (19.05%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Mus musculus<br>Number tested: 77<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Neotoma fiscipes<br>Number tested: 7<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Neotoma lepida<br>Number tested: 32<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Peromyscus boylii<br>Number tested: 3<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Peromyscus californicus<br>Number tested: 22<br>Number positive (%): 1 (4.55%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Peromyscus eremicus<br>Number tested: 41<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Peromyscus maniculatus<br>Number tested: 345<br>Number positive (%): 37 (10.72%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Reithrodontomys megalotis<br>Number tested: 191<br>Number positive (%): 13 (6.81%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Rattus rattus<br>Number tested: 46<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Rattus norvegicus<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Multiple<br>Host species: Spermophilus beecheyi<br>Number tested: 16<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Dobrava-Belgrade orthohantavirus<br>Host species: Apodemus agrarius<br>Number tested: 306<br>Number positive (%): 42 (13.73%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Dobrava-Belgrade orthohantavirus<br>Host species: Apodemus flavicollis<br>Number tested: 55<br>Number positive (%): 2 (3.64%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Puumala virus<br>Host species: Clethrionomys glareolus<br>Number tested: 2173<br>Number positive (%): 382 (17.58%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Apodemus flavicollis<br>Number tested: 3<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Apodemus mystacinus<br>Number tested: 10<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Apodemus sylvaticus s.l.<br>Number tested: 72<br>Number positive (%): 2 (2.78%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Microtus roberti<br>Number tested: 4<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Microtus rossiaemeridonalis<br>Number tested: 8<br>Number positive (%): 1 (12.5%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Apodemus flavicollis<br>Number tested: 28<br>Number positive (%): 1 (3.57%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Apodemus mystacinus<br>Number tested: 10<br>Number positive (%): 1 (10%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Apodemus sylvaticus s.l.<br>Number tested: 82<br>Number positive (%): 1 (1.22%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Dryomys nitedula<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Microtus roberti<br>Number tested: 11<br>Number positive (%): 1 (9.09%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Microtus rossiaemeridonalis<br>Number tested: 7<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Apodemus mystacinus<br>Number tested: 7<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Apodemus sylvaticus s.l.<br>Number tested: 52<br>Number positive (%): 1 (1.92%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Microtus guentheri lydius<br>Number tested: 25<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Microtus roberti<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Microtus rossiaemeridonalis<br>Number tested: 9<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Puumala virus/Saaremaa virus<br>Host species: Apodemus flavicollis<br>Number tested: 3<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Puumala virus/Saaremaa virus<br>Host species: Apodemus mystacinus<br>Number tested: 10<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Puumala virus/Saaremaa virus<br>Host species: Apodemus sylvaticus s.l.<br>Number tested: 72<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Puumala virus/Saaremaa virus<br>Host species: Microtus roberti<br>Number tested: 4<br>Number positive (%): 1 (25%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Puumala virus/Saaremaa virus<br>Host species: Microtus rossiaemeridonalis<br>Number tested: 8<br>Number positive (%): 2 (25%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Puumala virus/Saaremaa virus<br>Host species: Apodemus flavicollis<br>Number tested: 28<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Puumala virus/Saaremaa virus<br>Host species: Apodemus mystacinus<br>Number tested: 10<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Puumala virus/Saaremaa virus<br>Host species: Apodemus sylvaticus s.l.<br>Number tested: 82<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Puumala virus/Saaremaa virus<br>Host species: Dryomys nitedula<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Puumala virus/Saaremaa virus<br>Host species: Microtus roberti<br>Number tested: 11<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Puumala virus/Saaremaa virus<br>Host species: Microtus rossiaemeridonalis<br>Number tested: 7<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Puumala virus/Saaremaa virus<br>Host species: Apodemus mystacinus<br>Number tested: 7<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Puumala virus/Saaremaa virus<br>Host species: Apodemus sylvaticus s.l.<br>Number tested: 52<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Puumala virus/Saaremaa virus<br>Host species: Microtus guentheri lydius<br>Number tested: 25<br>Number positive (%): 1 (4%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Puumala virus/Saaremaa virus<br>Host species: Microtus roberti<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Puumala virus/Saaremaa virus<br>Host species: Microtus rossiaemeridonalis<br>Number tested: 9<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Hantaan virus<br>Host species: Apodemus agrarius<br>Number tested: 520<br>Number positive (%): 121 (23.27%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Hantaan virus<br>Host species: Micromys minutus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Hantaan virus<br>Host species: Mus musculus<br>Number tested: 6<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Hantaan virus<br>Host species: Rattus norvegicus<br>Number tested: 4<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Hantaan virus<br>Host species: Microtus fortis<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Hantaan virus<br>Host species: Myodes regulus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Hantaan virus<br>Host species: Crocidura lasiura<br>Number tested: 24<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Hantaan virus<br>Host species: Apodemus agrarius<br>Number tested: 647<br>Number positive (%): 94 (14.53%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Hantaan virus<br>Host species: Micromys minutus<br>Number tested: 4<br>Number positive (%): 1 (25%)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Hantaan virus<br>Host species: Mus musculus<br>Number tested: 3<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Hantaan virus<br>Host species: Rattus norvegicus<br>Number tested: 11<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Hantaan virus<br>Host species: Rattus rattus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Hantaan virus<br>Host species: Tscherskia triton<br>Number tested: 6<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Hantaan virus<br>Host species: Microtus fortis<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Hantaan virus<br>Host species: Myodes regulus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Hantaan virus<br>Host species: Crocidura lasiura<br>Number tested: 56<br>Number positive (%): 0 (NA)<br>","Pathogen family: Hantaviridae<br>Pathogen species: Seoul hantavirus<br>Host species: Rattus sp.<br>Number tested: 553<br>Number positive (%): 6 (1.08%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Crocidura sp.<br>Number tested: 42<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Grammomys buntingi<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Graphiurus sp.<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hylomyscus simus<br>Number tested: 8<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lemniscomys striatus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lophuromys sikapusi<br>Number tested: 41<br>Number positive (%): 1 (2.44%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Malacomys edwardsi<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys erythroleucus<br>Number tested: 4<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys natalensis<br>Number tested: 60<br>Number positive (%): 5 (8.33%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys minutoides<br>Number tested: 4<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys setulosus<br>Number tested: 7<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Praomys rostratus<br>Number tested: 44<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Rattus rattus<br>Number tested: 61<br>Number positive (%): 1 (1.64%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Crocidura sp.<br>Number tested: 24<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hybomys sp.<br>Number tested: 3<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hylomyscus simus<br>Number tested: 5<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lophuromys sikapusi<br>Number tested: 14<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys erythroleucus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys natalensis<br>Number tested: 96<br>Number positive (%): 2 (2.08%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys minutoides<br>Number tested: 3<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys setulosus<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Praomys rostratus<br>Number tested: 39<br>Number positive (%): 1 (2.56%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Rattus rattus<br>Number tested: 26<br>Number positive (%): 1 (3.85%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Crocidura sp.<br>Number tested: 27<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Dasymys rufulus<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Grammomys buntingi<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hybomys sp.<br>Number tested: 4<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hylomyscus simus<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lophuromys sikapusi<br>Number tested: 33<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Malacomys edwardsi<br>Number tested: 3<br>Number positive (%): 1 (33.33%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys erythroleucus<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys natalensis<br>Number tested: 16<br>Number positive (%): 2 (12.5%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys minutoides<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys setulosus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Praomys rostratus<br>Number tested: 21<br>Number positive (%): 3 (14.29%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Rattus rattus<br>Number tested: 40<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Crocidura sp.<br>Number tested: 54<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hybomys sp.<br>Number tested: 3<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hylomyscus simus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lemniscomys striatus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lophuromys sikapusi<br>Number tested: 25<br>Number positive (%): 1 (4%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Malacomys edwardsi<br>Number tested: 4<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys erythroleucus<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys natalensis<br>Number tested: 100<br>Number positive (%): 12 (12%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys minutoides<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys setulosus<br>Number tested: 3<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Praomys rostratus<br>Number tested: 102<br>Number positive (%): 1 (< 1%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Rattus rattus<br>Number tested: 58<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Crocidura sp.<br>Number tested: 35<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Dasymys rufulus<br>Number tested: 4<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hybomys sp.<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hylomyscus simus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lemniscomys striatus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lophuromys sikapusi<br>Number tested: 31<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Malacomys edwardsi<br>Number tested: 6<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys erythroleucus<br>Number tested: 3<br>Number positive (%): 2 (66.67%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys natalensis<br>Number tested: 34<br>Number positive (%): 3 (8.82%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Praomys rostratus<br>Number tested: 48<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Rattus rattus<br>Number tested: 42<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Uranomys ruddi<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Crocidura sp.<br>Number tested: 28<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Dasymys rufulus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hybomys sp.<br>Number tested: 5<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Hylomyscus simus<br>Number tested: 6<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lemniscomys striatus<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Lophuromys sikapusi<br>Number tested: 30<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Malacomys edwardsi<br>Number tested: 9<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys erythroleucus<br>Number tested: 2<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys natalensis<br>Number tested: 51<br>Number positive (%): 5 (9.8%)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys minutoides<br>Number tested: 1<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys setulosus<br>Number tested: 11<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Praomys rostratus<br>Number tested: 92<br>Number positive (%): 0 (NA)<br>","Pathogen family: Arenaviridae<br>Pathogen species: Lassa mammarenavirus<br>Host species: Rattus rattus<br>Number tested: 32<br>Number positive (%): 0 (NA)<br>"],{"interactive":false,"permanent":false,"direction":"auto","opacity":1,"offset":[0,0],"textsize":"10px","textOnly":false,"className":"","sticky":true},null]},{"method":"addLayersControl","args":[[],["Viral detection","Antibody detection"],{"collapsed":false,"autoZIndex":true,"position":"topright"}]},{"method":"addLegend","args":[{"colors":["#7FBFF5","#0087C7","#005682","#00314C","#091824","#241211","#4B201D","#7F3C37","#C3625B","#F8A29E"],"labels":["Asama virus","Dobrava-Belgrade orthohantavirus","Hantaan virus","Lassa mammarenavirus","Lymphocytic choriomeningitis virus","Multiple","Puumala virus","Puumala virus/Saaremaa virus","Seoul hantavirus","Sin Nombre orthohantavirus"],"na_color":null,"na_label":"NA","opacity":0.8,"position":"bottomright","type":"factor","title":"Pathogen species","extra":null,"layerId":null,"className":"info legend","group":null}]}],"limits":{"lat":[7.75703299999999,63.8258],"lng":[-123.1,138.252]}},"evals":[],"jsHooks":[]}</script>
<p>An interactive map showing the locations of pathogen sampling. Selecting clustered points will expand the data. Each individual circle represents a single host species pathogen combination at that sampling location. The number of samples tested is associated with circle size. Selecting the point will display information about the host-pathogen association, including number of samples tested and number of positive samples. A white border of the circle designates host-pathogen associations with at least one positive result. The colour of the circle indicates the pathogen species (where known). Two layers are available, for viral detection and antibody detection.</p>
</div>
</div>
</section>
<section id="host-pathogen-associations" class="level3">
<h3 class="anchored" data-anchor-id="host-pathogen-associations">Host pathogen associations</h3>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb5"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb5-1"><a href="#cb5-1" aria-hidden="true" tabindex="-1"></a>pathogen_summary <span class="ot">&lt;-</span> combined_data<span class="sc">$</span>pathogen <span class="sc">%&gt;%</span></span>
<span id="cb5-2"><a href="#cb5-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(<span class="at">genus =</span> host_genus,</span>
<span id="cb5-3"><a href="#cb5-3" aria-hidden="true" tabindex="-1"></a>         <span class="at">species =</span> host_species,</span>
<span id="cb5-4"><a href="#cb5-4" aria-hidden="true" tabindex="-1"></a>         pathogen_family, pathogen_species,</span>
<span id="cb5-5"><a href="#cb5-5" aria-hidden="true" tabindex="-1"></a>         number_tested, number_positive,</span>
<span id="cb5-6"><a href="#cb5-6" aria-hidden="true" tabindex="-1"></a>         detection_method) <span class="sc">%&gt;%</span></span>
<span id="cb5-7"><a href="#cb5-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">number_tested =</span> <span class="fu">as.numeric</span>(number_tested)) <span class="sc">%&gt;%</span></span>
<span id="cb5-8"><a href="#cb5-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(number_tested <span class="sc">&gt;=</span> <span class="dv">1</span>) <span class="sc">%&gt;%</span></span>
<span id="cb5-9"><a href="#cb5-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">left_join</span>(combined_data<span class="sc">$</span>zoonoses <span class="sc">%&gt;%</span></span>
<span id="cb5-10"><a href="#cb5-10" aria-hidden="true" tabindex="-1"></a>              <span class="fu">select</span>(pathogen_species, known_zoonosis),</span>
<span id="cb5-11"><a href="#cb5-11" aria-hidden="true" tabindex="-1"></a>            <span class="at">by =</span> <span class="fu">c</span>(<span class="st">"pathogen_species"</span>))</span>
<span id="cb5-12"><a href="#cb5-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-13"><a href="#cb5-13" aria-hidden="true" tabindex="-1"></a>hp_associations <span class="ot">&lt;-</span> combined_data<span class="sc">$</span>rodent <span class="sc">%&gt;%</span></span>
<span id="cb5-14"><a href="#cb5-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(genus, species) <span class="sc">%&gt;%</span></span>
<span id="cb5-15"><a href="#cb5-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">left_join</span>(pathogen_summary <span class="sc">%&gt;%</span></span>
<span id="cb5-16"><a href="#cb5-16" aria-hidden="true" tabindex="-1"></a>              <span class="fu">group_by</span>(detection_method, species, pathogen_species, known_zoonosis) <span class="sc">%&gt;%</span></span>
<span id="cb5-17"><a href="#cb5-17" aria-hidden="true" tabindex="-1"></a>              <span class="fu">mutate</span>(<span class="at">number_tested =</span> <span class="fu">sum</span>(number_tested),</span>
<span id="cb5-18"><a href="#cb5-18" aria-hidden="true" tabindex="-1"></a>                     <span class="at">number_positive =</span> <span class="fu">sum</span>(number_positive)), <span class="at">by =</span> <span class="fu">c</span>(<span class="st">"genus"</span>, <span class="st">"species"</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb5-19"><a href="#cb5-19" aria-hidden="true" tabindex="-1"></a>  <span class="fu">drop_na</span>(number_tested) <span class="sc">%&gt;%</span></span>
<span id="cb5-20"><a href="#cb5-20" aria-hidden="true" tabindex="-1"></a>  <span class="fu">distinct</span>() <span class="sc">%&gt;%</span></span>
<span id="cb5-21"><a href="#cb5-21" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(<span class="sc">!</span>pathogen_species <span class="sc">%in%</span> <span class="fu">c</span>(<span class="st">"Multiple"</span>, <span class="st">"Puumala virus/Saaremaa virus"</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb5-22"><a href="#cb5-22" aria-hidden="true" tabindex="-1"></a>  <span class="fu">arrange</span>(pathogen_family, pathogen_species, genus, species) <span class="sc">%&gt;%</span></span>
<span id="cb5-23"><a href="#cb5-23" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">association =</span> <span class="fu">case_when</span>(number_positive <span class="sc">&gt;=</span> <span class="dv">1</span> <span class="sc">~</span> <span class="cn">TRUE</span>,</span>
<span id="cb5-24"><a href="#cb5-24" aria-hidden="true" tabindex="-1"></a>                                 <span class="cn">TRUE</span> <span class="sc">~</span> <span class="cn">FALSE</span>),</span>
<span id="cb5-25"><a href="#cb5-25" aria-hidden="true" tabindex="-1"></a>         <span class="at">detection_method =</span> <span class="fu">case_when</span>(detection_method <span class="sc">==</span> <span class="st">"pcr"</span> <span class="sc">~</span> <span class="st">"Viral"</span>,</span>
<span id="cb5-26"><a href="#cb5-26" aria-hidden="true" tabindex="-1"></a>                                      detection_method <span class="sc">==</span> <span class="st">"antibody"</span> <span class="sc">~</span> <span class="st">"Antibody"</span>))</span>
<span id="cb5-27"><a href="#cb5-27" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-28"><a href="#cb5-28" aria-hidden="true" tabindex="-1"></a>scaled_n <span class="ot">&lt;-</span> <span class="fu">preProcess</span>(<span class="fu">as.data.frame</span>(<span class="fu">log10</span>(hp_associations<span class="sc">$</span>number_tested)), <span class="at">method =</span> <span class="st">"range"</span>)</span>
<span id="cb5-29"><a href="#cb5-29" aria-hidden="true" tabindex="-1"></a>hp_associations<span class="sc">$</span>alpha <span class="ot">&lt;-</span> <span class="fu">predict</span>(scaled_n, <span class="fu">as.data.frame</span>(<span class="fu">log10</span>(hp_associations<span class="sc">$</span>number_tested)))[, <span class="dv">1</span>]</span>
<span id="cb5-30"><a href="#cb5-30" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-31"><a href="#cb5-31" aria-hidden="true" tabindex="-1"></a>host_pathogen_plot <span class="ot">&lt;-</span> hp_associations <span class="sc">%&gt;%</span></span>
<span id="cb5-32"><a href="#cb5-32" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggplot</span>(<span class="fu">aes</span>(<span class="at">x =</span> pathogen_species, <span class="at">y =</span> species)) <span class="sc">+</span></span>
<span id="cb5-33"><a href="#cb5-33" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_tile</span>(<span class="fu">aes</span>(<span class="at">fill =</span> association, <span class="at">colour =</span> known_zoonosis, <span class="at">alpha =</span> alpha,</span>
<span id="cb5-34"><a href="#cb5-34" aria-hidden="true" tabindex="-1"></a>                <span class="at">text =</span> <span class="fu">paste0</span>(<span class="st">"Pathogen species: "</span>, pathogen_species, <span class="st">"&lt;br&gt;"</span>,</span>
<span id="cb5-35"><a href="#cb5-35" aria-hidden="true" tabindex="-1"></a>                              <span class="st">"Host species: "</span>, species, <span class="st">"&lt;br&gt;"</span>,</span>
<span id="cb5-36"><a href="#cb5-36" aria-hidden="true" tabindex="-1"></a>                              <span class="st">"Number tested: "</span>, number_tested, <span class="st">"&lt;br&gt;"</span>,</span>
<span id="cb5-37"><a href="#cb5-37" aria-hidden="true" tabindex="-1"></a>                              <span class="st">"Number positive: "</span>, number_positive, <span class="st">"&lt;br&gt;"</span>,</span>
<span id="cb5-38"><a href="#cb5-38" aria-hidden="true" tabindex="-1"></a>                              <span class="st">"Known zoonosis: "</span>, known_zoonosis))) <span class="sc">+</span></span>
<span id="cb5-39"><a href="#cb5-39" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_viridis_d</span>() <span class="sc">+</span></span>
<span id="cb5-40"><a href="#cb5-40" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_colour_manual</span>(<span class="at">values =</span> <span class="fu">c</span>(<span class="st">"white"</span>, <span class="st">"black"</span>)) <span class="sc">+</span></span>
<span id="cb5-41"><a href="#cb5-41" aria-hidden="true" tabindex="-1"></a>  <span class="fu">facet_wrap</span>(<span class="sc">~</span> detection_method) <span class="sc">+</span></span>
<span id="cb5-42"><a href="#cb5-42" aria-hidden="true" tabindex="-1"></a>  <span class="fu">guides</span>(<span class="at">alpha =</span> <span class="st">"none"</span>,</span>
<span id="cb5-43"><a href="#cb5-43" aria-hidden="true" tabindex="-1"></a>         <span class="at">colour =</span> <span class="st">"none"</span>) <span class="sc">+</span></span>
<span id="cb5-44"><a href="#cb5-44" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">fill =</span> <span class="st">"H-P association"</span>,</span>
<span id="cb5-45"><a href="#cb5-45" aria-hidden="true" tabindex="-1"></a>       <span class="at">x =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb5-46"><a href="#cb5-46" aria-hidden="true" tabindex="-1"></a>       <span class="at">y =</span> <span class="fu">element_blank</span>()) <span class="sc">+</span></span>
<span id="cb5-47"><a href="#cb5-47" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_bw</span>() <span class="sc">+</span></span>
<span id="cb5-48"><a href="#cb5-48" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">axis.text.x =</span> <span class="fu">element_text</span>(<span class="at">angle =</span> <span class="dv">45</span>, <span class="at">vjust =</span> <span class="dv">1</span>, <span class="at">hjust =</span> <span class="dv">1</span>))</span>
<span id="cb5-49"><a href="#cb5-49" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb5-50"><a href="#cb5-50" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplotly</span>(host_pathogen_plot, <span class="at">tooltip =</span> <span class="st">"text"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div id="htmlwidget-7525b11e97e6b011530f" style="width:100%;height:866px;" class="plotly html-widget"></div>
<script type="application/json" data-for="htmlwidget-7525b11e97e6b011530f">{"x":{"data":[{"x":[2.5,2.5,3.5,3.5,2.5],"y":[38.5,39.5,39.5,38.5,38.5],"text":"Pathogen species: Hantaan virus<br>Host species: Rattus rattus<br>Number tested: 1<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.1)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.1)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":true,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.5,3.5,4.5,4.5,3.5],"y":[13.5,14.5,14.5,13.5,13.5],"text":"Pathogen species: Lassa mammarenavirus<br>Host species: Graphiurus sp.<br>Number tested: 1<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.1)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.1)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[4.5,4.5,5.5,5.5,4.5],"y":[11.5,12.5,12.5,11.5,11.5],"text":"Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Dryomys nitedula<br>Number tested: 1<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.1)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.1)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.5,3.5,4.5,4.5,3.5],"y":[13.5,14.5,14.5,13.5,13.5],"text":"Pathogen species: Lassa mammarenavirus<br>Host species: Graphiurus sp.<br>Number tested: 1<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.1)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.1)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x2","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0.5,0.5,1.5,1.5,0.5],"y":[7.5,8.5,8.5,7.5,7.5],"text":"Pathogen species: Asama virus<br>Host species: Crocidura dsinezumi<br>Number tested: 1<br>Number positive: 0<br>Known zoonosis: FALSE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(255,255,255,0.1)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.1)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x2","yaxis":"y","hoverinfo":"text","frame":null},{"x":[2.5,2.5,3.5,3.5,2.5],"y":[27.5,28.5,28.5,27.5,27.5],"text":"Pathogen species: Hantaan virus<br>Host species: Myodes regulus<br>Number tested: 2<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.1812)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.1812)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.5,3.5,4.5,4.5,3.5],"y":[12.5,13.5,13.5,12.5,12.5],"text":"Pathogen species: Lassa mammarenavirus<br>Host species: Grammomys buntingi<br>Number tested: 2<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.1812)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.1812)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.5,3.5,4.5,4.5,3.5],"y":[45.5,46.5,46.5,45.5,45.5],"text":"Pathogen species: Lassa mammarenavirus<br>Host species: Uranomys ruddi<br>Number tested: 2<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.1812)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.1812)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[6.5,6.5,7.5,7.5,6.5],"y":[35.5,36.5,36.5,35.5,35.5],"text":"Pathogen species: Sin Nombre orthohantavirus<br>Host species: Peromyscus maniculatus<br>Number tested: 2<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.1812)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.1812)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.5,3.5,4.5,4.5,3.5],"y":[12.5,13.5,13.5,12.5,12.5],"text":"Pathogen species: Lassa mammarenavirus<br>Host species: Grammomys buntingi<br>Number tested: 2<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.1812)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.1812)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x2","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.5,3.5,4.5,4.5,3.5],"y":[45.5,46.5,46.5,45.5,45.5],"text":"Pathogen species: Lassa mammarenavirus<br>Host species: Uranomys ruddi<br>Number tested: 2<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.1812)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.1812)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x2","yaxis":"y","hoverinfo":"text","frame":null},{"x":[2.5,2.5,3.5,3.5,2.5],"y":[22.5,23.5,23.5,22.5,22.5],"text":"Pathogen species: Hantaan virus<br>Host species: Microtus fortis<br>Number tested: 3<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.2287)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.2287)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0.5,0.5,1.5,1.5,0.5],"y":[41.5,42.5,42.5,41.5,41.5],"text":"Pathogen species: Asama virus<br>Host species: Sorex caecutiens<br>Number tested: 3<br>Number positive: 0<br>Known zoonosis: FALSE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(255,255,255,0.2287)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.2287)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x2","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.5,3.5,4.5,4.5,3.5],"y":[16.5,17.5,17.5,16.5,16.5],"text":"Pathogen species: Lassa mammarenavirus<br>Host species: Lemniscomys striatus<br>Number tested: 4<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.2624)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.2624)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.5,3.5,4.5,4.5,3.5],"y":[16.5,17.5,17.5,16.5,16.5],"text":"Pathogen species: Lassa mammarenavirus<br>Host species: Lemniscomys striatus<br>Number tested: 4<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.2624)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.2624)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x2","yaxis":"y","hoverinfo":"text","frame":null},{"x":[6.5,6.5,7.5,7.5,6.5],"y":[40.5,41.5,41.5,40.5,40.5],"text":"Pathogen species: Sin Nombre orthohantavirus<br>Host species: Sigmodon ochrognathus<br>Number tested: 5<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.2885)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.2885)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0.5,0.5,1.5,1.5,0.5],"y":[42.5,43.5,43.5,42.5,42.5],"text":"Pathogen species: Asama virus<br>Host species: Sorex gracillimus<br>Number tested: 5<br>Number positive: 0<br>Known zoonosis: FALSE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(255,255,255,0.2885)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.2885)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x2","yaxis":"y","hoverinfo":"text","frame":null},{"x":[2.5,2.5,3.5,3.5,2.5],"y":[44.5,45.5,45.5,44.5,44.5],"text":"Pathogen species: Hantaan virus<br>Host species: Tscherskia triton<br>Number tested: 6<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.3099)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.3099)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0.5,0.5,1.5,1.5,0.5],"y":[43.5,44.5,44.5,43.5,43.5],"text":"Pathogen species: Asama virus<br>Host species: Sorex unguiculatus<br>Number tested: 6<br>Number positive: 0<br>Known zoonosis: FALSE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(255,255,255,0.3099)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.3099)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x2","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.5,3.5,4.5,4.5,3.5],"y":[10.5,11.5,11.5,10.5,10.5],"text":"Pathogen species: Lassa mammarenavirus<br>Host species: Dasymys rufulus<br>Number tested: 7<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.3279)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.3279)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[6.5,6.5,7.5,7.5,6.5],"y":[5.5,6.5,6.5,5.5,5.5],"text":"Pathogen species: Sin Nombre orthohantavirus<br>Host species: Chaetodipus penicillatus<br>Number tested: 7<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.3279)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.3279)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[6.5,6.5,7.5,7.5,6.5],"y":[31.5,32.5,32.5,31.5,31.5],"text":"Pathogen species: Sin Nombre orthohantavirus<br>Host species: Onychomys torridue<br>Number tested: 7<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.3279)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.3279)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.5,3.5,4.5,4.5,3.5],"y":[10.5,11.5,11.5,10.5,10.5],"text":"Pathogen species: Lassa mammarenavirus<br>Host species: Dasymys rufulus<br>Number tested: 7<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.3279)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.3279)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x2","yaxis":"y","hoverinfo":"text","frame":null},{"x":[2.5,2.5,3.5,3.5,2.5],"y":[26.5,27.5,27.5,26.5,26.5],"text":"Pathogen species: Hantaan virus<br>Host species: Mus musculus<br>Number tested: 9<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.3574)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.3574)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.5,3.5,4.5,4.5,3.5],"y":[28.5,29.5,29.5,28.5,28.5],"text":"Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys minutoides<br>Number tested: 11<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.3809)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.3809)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.5,3.5,4.5,4.5,3.5],"y":[28.5,29.5,29.5,28.5,28.5],"text":"Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys minutoides<br>Number tested: 11<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.3809)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.3809)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x2","yaxis":"y","hoverinfo":"text","frame":null},{"x":[6.5,6.5,7.5,7.5,6.5],"y":[39.5,40.5,40.5,39.5,39.5],"text":"Pathogen species: Sin Nombre orthohantavirus<br>Host species: Reithrodontomys fulvescens<br>Number tested: 12<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.3911)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.3911)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.5,3.5,4.5,4.5,3.5],"y":[19.5,20.5,20.5,19.5,19.5],"text":"Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys erythroleucus<br>Number tested: 14<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.4091)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.4091)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x2","yaxis":"y","hoverinfo":"text","frame":null},{"x":[2.5,2.5,3.5,3.5,2.5],"y":[37.5,38.5,38.5,37.5,37.5],"text":"Pathogen species: Hantaan virus<br>Host species: Rattus norvegicus<br>Number tested: 15<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.4172)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.4172)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.5,3.5,4.5,4.5,3.5],"y":[14.5,15.5,15.5,14.5,14.5],"text":"Pathogen species: Lassa mammarenavirus<br>Host species: Hybomys sp.<br>Number tested: 16<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.4247)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.4247)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.5,3.5,4.5,4.5,3.5],"y":[14.5,15.5,15.5,14.5,14.5],"text":"Pathogen species: Lassa mammarenavirus<br>Host species: Hybomys sp.<br>Number tested: 16<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.4247)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.4247)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x2","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.5,3.5,4.5,4.5,3.5],"y":[15.5,16.5,16.5,15.5,15.5],"text":"Pathogen species: Lassa mammarenavirus<br>Host species: Hylomyscus simus<br>Number tested: 23<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.4673)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.4673)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.5,3.5,4.5,4.5,3.5],"y":[15.5,16.5,16.5,15.5,15.5],"text":"Pathogen species: Lassa mammarenavirus<br>Host species: Hylomyscus simus<br>Number tested: 23<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.4673)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.4673)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x2","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.5,3.5,4.5,4.5,3.5],"y":[18.5,19.5,19.5,18.5,18.5],"text":"Pathogen species: Lassa mammarenavirus<br>Host species: Malacomys edwardsi<br>Number tested: 23<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.4673)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.4673)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x2","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.5,3.5,4.5,4.5,3.5],"y":[29.5,30.5,30.5,29.5,29.5],"text":"Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys setulosus<br>Number tested: 24<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.4722)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.4722)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.5,3.5,4.5,4.5,3.5],"y":[29.5,30.5,30.5,29.5,29.5],"text":"Pathogen species: Lassa mammarenavirus<br>Host species: Nannomys setulosus<br>Number tested: 24<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.4722)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.4722)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x2","yaxis":"y","hoverinfo":"text","frame":null},{"x":[4.5,4.5,5.5,5.5,4.5],"y":[23.5,24.5,24.5,23.5,23.5],"text":"Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Microtus guentheri lydius<br>Number tested: 25<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.477)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.477)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[6.5,6.5,7.5,7.5,6.5],"y":[30.5,31.5,31.5,30.5,30.5],"text":"Pathogen species: Sin Nombre orthohantavirus<br>Host species: Neotoma albigula<br>Number tested: 51<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.5605)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.5605)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[2.5,2.5,3.5,3.5,2.5],"y":[8.5,9.5,9.5,8.5,8.5],"text":"Pathogen species: Hantaan virus<br>Host species: Crocidura lasiura<br>Number tested: 80<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.6133)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.6133)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.5,3.5,4.5,4.5,3.5],"y":[17.5,18.5,18.5,17.5,17.5],"text":"Pathogen species: Lassa mammarenavirus<br>Host species: Lophuromys sikapusi<br>Number tested: 174<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.7043)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.7043)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x2","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.5,3.5,4.5,4.5,3.5],"y":[9.5,10.5,10.5,9.5,9.5],"text":"Pathogen species: Lassa mammarenavirus<br>Host species: Crocidura sp.<br>Number tested: 210<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.7263)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.7263)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.5,3.5,4.5,4.5,3.5],"y":[9.5,10.5,10.5,9.5,9.5],"text":"Pathogen species: Lassa mammarenavirus<br>Host species: Crocidura sp.<br>Number tested: 210<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.7263)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.7263)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x2","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.5,3.5,4.5,4.5,3.5],"y":[38.5,39.5,39.5,38.5,38.5],"text":"Pathogen species: Lassa mammarenavirus<br>Host species: Rattus rattus<br>Number tested: 259<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.7509)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.7509)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x2","yaxis":"y","hoverinfo":"text","frame":null},{"x":[6.5,6.5,7.5,7.5,6.5],"y":[4.5,5.5,5.5,4.5,4.5],"text":"Pathogen species: Sin Nombre orthohantavirus<br>Host species: Chaetodipus baileyi<br>Number tested: 329<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.7789)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.7789)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.5,3.5,4.5,4.5,3.5],"y":[36.5,37.5,37.5,36.5,36.5],"text":"Pathogen species: Lassa mammarenavirus<br>Host species: Praomys rostratus<br>Number tested: 346<br>Number positive: 0<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.7848)","dash":"solid"},"fill":"toself","fillcolor":"rgba(68,1,84,0.7848)","hoveron":"fills","name":"FALSE","legendgroup":"FALSE","showlegend":false,"xaxis":"x2","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.5,1.5,2.5,2.5,1.5],"y":[1.5,2.5,2.5,1.5,1.5],"text":"Pathogen species: Dobrava-Belgrade orthohantavirus<br>Host species: Apodemus flavicollis<br>Number tested: 2<br>Number positive: 1<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.1812)","dash":"solid"},"fill":"toself","fillcolor":"rgba(253,231,37,0.1812)","hoveron":"fills","name":"TRUE","legendgroup":"TRUE","showlegend":true,"xaxis":"x2","yaxis":"y","hoverinfo":"text","frame":null},{"x":[2.5,2.5,3.5,3.5,2.5],"y":[21.5,22.5,22.5,21.5,21.5],"text":"Pathogen species: Hantaan virus<br>Host species: Micromys minutus<br>Number tested: 5<br>Number positive: 1<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.2885)","dash":"solid"},"fill":"toself","fillcolor":"rgba(253,231,37,0.2885)","hoveron":"fills","name":"TRUE","legendgroup":"TRUE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[0.5,0.5,1.5,1.5,0.5],"y":[46.5,47.5,47.5,46.5,46.5],"text":"Pathogen species: Asama virus<br>Host species: Urotrichis talpoides<br>Number tested: 6<br>Number positive: 3<br>Known zoonosis: FALSE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(255,255,255,0.3099)","dash":"solid"},"fill":"toself","fillcolor":"rgba(253,231,37,0.3099)","hoveron":"fills","name":"TRUE","legendgroup":"TRUE","showlegend":false,"xaxis":"x2","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.5,3.5,4.5,4.5,3.5],"y":[19.5,20.5,20.5,19.5,19.5],"text":"Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys erythroleucus<br>Number tested: 14<br>Number positive: 2<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.4091)","dash":"solid"},"fill":"toself","fillcolor":"rgba(253,231,37,0.4091)","hoveron":"fills","name":"TRUE","legendgroup":"TRUE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[4.5,4.5,5.5,5.5,4.5],"y":[24.5,25.5,25.5,24.5,24.5],"text":"Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Microtus roberti<br>Number tested: 16<br>Number positive: 1<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.4247)","dash":"solid"},"fill":"toself","fillcolor":"rgba(253,231,37,0.4247)","hoveron":"fills","name":"TRUE","legendgroup":"TRUE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[6.5,6.5,7.5,7.5,6.5],"y":[34.5,35.5,35.5,34.5,34.5],"text":"Pathogen species: Sin Nombre orthohantavirus<br>Host species: Peromyscus leucopus<br>Number tested: 21<br>Number positive: 3<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.4566)","dash":"solid"},"fill":"toself","fillcolor":"rgba(253,231,37,0.4566)","hoveron":"fills","name":"TRUE","legendgroup":"TRUE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.5,3.5,4.5,4.5,3.5],"y":[18.5,19.5,19.5,18.5,18.5],"text":"Pathogen species: Lassa mammarenavirus<br>Host species: Malacomys edwardsi<br>Number tested: 23<br>Number positive: 1<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.4673)","dash":"solid"},"fill":"toself","fillcolor":"rgba(253,231,37,0.4673)","hoveron":"fills","name":"TRUE","legendgroup":"TRUE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[4.5,4.5,5.5,5.5,4.5],"y":[25.5,26.5,26.5,25.5,25.5],"text":"Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Microtus rossiaemeridonalis<br>Number tested: 24<br>Number positive: 1<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.4722)","dash":"solid"},"fill":"toself","fillcolor":"rgba(253,231,37,0.4722)","hoveron":"fills","name":"TRUE","legendgroup":"TRUE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[4.5,4.5,5.5,5.5,4.5],"y":[2.5,3.5,3.5,2.5,2.5],"text":"Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Apodemus mystacinus<br>Number tested: 27<br>Number positive: 1<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.486)","dash":"solid"},"fill":"toself","fillcolor":"rgba(253,231,37,0.486)","hoveron":"fills","name":"TRUE","legendgroup":"TRUE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[4.5,4.5,5.5,5.5,4.5],"y":[1.5,2.5,2.5,1.5,1.5],"text":"Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Apodemus flavicollis<br>Number tested: 31<br>Number positive: 1<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.5022)","dash":"solid"},"fill":"toself","fillcolor":"rgba(253,231,37,0.5022)","hoveron":"fills","name":"TRUE","legendgroup":"TRUE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.5,1.5,2.5,2.5,1.5],"y":[0.5,1.5,1.5,0.5,0.5],"text":"Pathogen species: Dobrava-Belgrade orthohantavirus<br>Host species: Apodemus agrarius<br>Number tested: 42<br>Number positive: 8<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.5378)","dash":"solid"},"fill":"toself","fillcolor":"rgba(253,231,37,0.5378)","hoveron":"fills","name":"TRUE","legendgroup":"TRUE","showlegend":false,"xaxis":"x2","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.5,1.5,2.5,2.5,1.5],"y":[1.5,2.5,2.5,1.5,1.5],"text":"Pathogen species: Dobrava-Belgrade orthohantavirus<br>Host species: Apodemus flavicollis<br>Number tested: 55<br>Number positive: 2<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.5694)","dash":"solid"},"fill":"toself","fillcolor":"rgba(253,231,37,0.5694)","hoveron":"fills","name":"TRUE","legendgroup":"TRUE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[6.5,6.5,7.5,7.5,6.5],"y":[32.5,33.5,33.5,32.5,32.5],"text":"Pathogen species: Sin Nombre orthohantavirus<br>Host species: Peromyscus boylii<br>Number tested: 98<br>Number positive: 13<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.637)","dash":"solid"},"fill":"toself","fillcolor":"rgba(253,231,37,0.637)","hoveron":"fills","name":"TRUE","legendgroup":"TRUE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[6.5,6.5,7.5,7.5,6.5],"y":[33.5,34.5,34.5,33.5,33.5],"text":"Pathogen species: Sin Nombre orthohantavirus<br>Host species: Peromyscus eremicus<br>Number tested: 118<br>Number positive: 1<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.6588)","dash":"solid"},"fill":"toself","fillcolor":"rgba(253,231,37,0.6588)","hoveron":"fills","name":"TRUE","legendgroup":"TRUE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.5,3.5,4.5,4.5,3.5],"y":[17.5,18.5,18.5,17.5,17.5],"text":"Pathogen species: Lassa mammarenavirus<br>Host species: Lophuromys sikapusi<br>Number tested: 174<br>Number positive: 2<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.7043)","dash":"solid"},"fill":"toself","fillcolor":"rgba(253,231,37,0.7043)","hoveron":"fills","name":"TRUE","legendgroup":"TRUE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[4.5,4.5,5.5,5.5,4.5],"y":[3.5,4.5,4.5,3.5,3.5],"text":"Pathogen species: Lymphocytic choriomeningitis virus<br>Host species: Apodemus sylvaticus s.l.<br>Number tested: 206<br>Number positive: 4<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.724)","dash":"solid"},"fill":"toself","fillcolor":"rgba(253,231,37,0.724)","hoveron":"fills","name":"TRUE","legendgroup":"TRUE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.5,3.5,4.5,4.5,3.5],"y":[38.5,39.5,39.5,38.5,38.5],"text":"Pathogen species: Lassa mammarenavirus<br>Host species: Rattus rattus<br>Number tested: 259<br>Number positive: 2<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.7509)","dash":"solid"},"fill":"toself","fillcolor":"rgba(253,231,37,0.7509)","hoveron":"fills","name":"TRUE","legendgroup":"TRUE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[1.5,1.5,2.5,2.5,1.5],"y":[0.5,1.5,1.5,0.5,0.5],"text":"Pathogen species: Dobrava-Belgrade orthohantavirus<br>Host species: Apodemus agrarius<br>Number tested: 306<br>Number positive: 42<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.7704)","dash":"solid"},"fill":"toself","fillcolor":"rgba(253,231,37,0.7704)","hoveron":"fills","name":"TRUE","legendgroup":"TRUE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.5,3.5,4.5,4.5,3.5],"y":[36.5,37.5,37.5,36.5,36.5],"text":"Pathogen species: Lassa mammarenavirus<br>Host species: Praomys rostratus<br>Number tested: 346<br>Number positive: 5<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.7848)","dash":"solid"},"fill":"toself","fillcolor":"rgba(253,231,37,0.7848)","hoveron":"fills","name":"TRUE","legendgroup":"TRUE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.5,3.5,4.5,4.5,3.5],"y":[20.5,21.5,21.5,20.5,20.5],"text":"Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys natalensis<br>Number tested: 357<br>Number positive: 29<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.7885)","dash":"solid"},"fill":"toself","fillcolor":"rgba(253,231,37,0.7885)","hoveron":"fills","name":"TRUE","legendgroup":"TRUE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[3.5,3.5,4.5,4.5,3.5],"y":[20.5,21.5,21.5,20.5,20.5],"text":"Pathogen species: Lassa mammarenavirus<br>Host species: Mastomys natalensis<br>Number tested: 357<br>Number positive: 4<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.7885)","dash":"solid"},"fill":"toself","fillcolor":"rgba(253,231,37,0.7885)","hoveron":"fills","name":"TRUE","legendgroup":"TRUE","showlegend":false,"xaxis":"x2","yaxis":"y","hoverinfo":"text","frame":null},{"x":[2.5,2.5,3.5,3.5,2.5],"y":[0.5,1.5,1.5,0.5,0.5],"text":"Pathogen species: Hantaan virus<br>Host species: Apodemus agrarius<br>Number tested: 1167<br>Number positive: 215<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,0.9272)","dash":"solid"},"fill":"toself","fillcolor":"rgba(253,231,37,0.9272)","hoveron":"fills","name":"TRUE","legendgroup":"TRUE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null},{"x":[5.5,5.5,6.5,6.5,5.5],"y":[6.5,7.5,7.5,6.5,6.5],"text":"Pathogen species: Puumala virus<br>Host species: Clethrionomys glareolus<br>Number tested: 2173<br>Number positive: 382<br>Known zoonosis: TRUE","type":"scatter","mode":"lines","line":{"width":0.377952755905512,"color":"rgba(0,0,0,1)","dash":"solid"},"fill":"toself","fillcolor":"rgba(253,231,37,1)","hoveron":"fills","name":"TRUE","legendgroup":"TRUE","showlegend":false,"xaxis":"x","yaxis":"y","hoverinfo":"text","frame":null}],"layout":{"margin":{"t":33.1689497716895,"r":7.30593607305936,"b":113.705627635071,"l":168.767123287671},"plot_bgcolor":"rgba(255,255,255,1)","paper_bgcolor":"rgba(255,255,255,1)","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187},"xaxis":{"domain":[0,0.496829020801624],"automargin":true,"type":"linear","autorange":false,"range":[0.4,7.6],"tickmode":"array","ticktext":["Asama virus","Dobrava-Belgrade orthohantavirus","Hantaan virus","Lassa mammarenavirus","Lymphocytic choriomeningitis virus","Puumala virus","Sin Nombre orthohantavirus"],"tickvals":[1,2,3,4,5,6,7],"categoryorder":"array","categoryarray":["Asama virus","Dobrava-Belgrade orthohantavirus","Hantaan virus","Lassa mammarenavirus","Lymphocytic choriomeningitis virus","Puumala virus","Sin Nombre orthohantavirus"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-45,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"yaxis":{"domain":[0,1],"automargin":true,"type":"linear","autorange":false,"range":[0.4,47.6],"tickmode":"array","ticktext":["Apodemus agrarius","Apodemus flavicollis","Apodemus mystacinus","Apodemus sylvaticus s.l.","Chaetodipus baileyi","Chaetodipus penicillatus","Clethrionomys glareolus","Crocidura dsinezumi","Crocidura lasiura","Crocidura sp.","Dasymys rufulus","Dryomys nitedula","Grammomys buntingi","Graphiurus sp.","Hybomys sp.","Hylomyscus simus","Lemniscomys striatus","Lophuromys sikapusi","Malacomys edwardsi","Mastomys erythroleucus","Mastomys natalensis","Micromys minutus","Microtus fortis","Microtus guentheri lydius","Microtus roberti","Microtus rossiaemeridonalis","Mus musculus","Myodes regulus","Nannomys minutoides","Nannomys setulosus","Neotoma albigula","Onychomys torridue","Peromyscus boylii","Peromyscus eremicus","Peromyscus leucopus","Peromyscus maniculatus","Praomys rostratus","Rattus norvegicus","Rattus rattus","Reithrodontomys fulvescens","Sigmodon ochrognathus","Sorex caecutiens","Sorex gracillimus","Sorex unguiculatus","Tscherskia triton","Uranomys ruddi","Urotrichis talpoides"],"tickvals":[1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47],"categoryorder":"array","categoryarray":["Apodemus agrarius","Apodemus flavicollis","Apodemus mystacinus","Apodemus sylvaticus s.l.","Chaetodipus baileyi","Chaetodipus penicillatus","Clethrionomys glareolus","Crocidura dsinezumi","Crocidura lasiura","Crocidura sp.","Dasymys rufulus","Dryomys nitedula","Grammomys buntingi","Graphiurus sp.","Hybomys sp.","Hylomyscus simus","Lemniscomys striatus","Lophuromys sikapusi","Malacomys edwardsi","Mastomys erythroleucus","Mastomys natalensis","Micromys minutus","Microtus fortis","Microtus guentheri lydius","Microtus roberti","Microtus rossiaemeridonalis","Mus musculus","Myodes regulus","Nannomys minutoides","Nannomys setulosus","Neotoma albigula","Onychomys torridue","Peromyscus boylii","Peromyscus eremicus","Peromyscus leucopus","Peromyscus maniculatus","Praomys rostratus","Rattus norvegicus","Rattus rattus","Reithrodontomys fulvescens","Sigmodon ochrognathus","Sorex caecutiens","Sorex gracillimus","Sorex unguiculatus","Tscherskia triton","Uranomys ruddi","Urotrichis talpoides"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-0,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"x","title":"","hoverformat":".2f"},"shapes":[{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":0.496829020801624,"y0":0,"y1":1},{"type":"rect","fillcolor":"rgba(217,217,217,1)","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0,"x1":0.496829020801624,"y0":0,"y1":23.37899543379,"yanchor":1,"ysizemode":"pixel"},{"type":"rect","fillcolor":"transparent","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.503170979198377,"x1":1,"y0":0,"y1":1},{"type":"rect","fillcolor":"rgba(217,217,217,1)","line":{"color":"rgba(51,51,51,1)","width":0.66417600664176,"linetype":"solid"},"yref":"paper","xref":"paper","x0":0.503170979198377,"x1":1,"y0":0,"y1":23.37899543379,"yanchor":1,"ysizemode":"pixel"}],"annotations":[{"text":"Antibody","x":0.248414510400812,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(26,26,26,1)","family":"","size":11.689497716895},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"},{"text":"Viral","x":0.751585489599188,"y":1,"showarrow":false,"ax":0,"ay":0,"font":{"color":"rgba(26,26,26,1)","family":"","size":11.689497716895},"xref":"paper","yref":"paper","textangle":-0,"xanchor":"center","yanchor":"bottom"}],"xaxis2":{"type":"linear","autorange":false,"range":[0.4,7.6],"tickmode":"array","ticktext":["Asama virus","Dobrava-Belgrade orthohantavirus","Hantaan virus","Lassa mammarenavirus","Lymphocytic choriomeningitis virus","Puumala virus","Sin Nombre orthohantavirus"],"tickvals":[1,2,3,4,5,6,7],"categoryorder":"array","categoryarray":["Asama virus","Dobrava-Belgrade orthohantavirus","Hantaan virus","Lassa mammarenavirus","Lymphocytic choriomeningitis virus","Puumala virus","Sin Nombre orthohantavirus"],"nticks":null,"ticks":"outside","tickcolor":"rgba(51,51,51,1)","ticklen":3.65296803652968,"tickwidth":0.66417600664176,"showticklabels":true,"tickfont":{"color":"rgba(77,77,77,1)","family":"","size":11.689497716895},"tickangle":-45,"showline":false,"linecolor":null,"linewidth":0,"showgrid":true,"domain":[0.503170979198377,1],"gridcolor":"rgba(235,235,235,1)","gridwidth":0.66417600664176,"zeroline":false,"anchor":"y","title":"","hoverformat":".2f"},"showlegend":true,"legend":{"bgcolor":"rgba(255,255,255,1)","bordercolor":"transparent","borderwidth":1.88976377952756,"font":{"color":"rgba(0,0,0,1)","family":"","size":11.689497716895},"title":{"text":"H-P association","font":{"color":"rgba(0,0,0,1)","family":"","size":14.6118721461187}}},"hovermode":"closest","barmode":"relative"},"config":{"doubleClick":"reset","modeBarButtonsToAdd":["hoverclosest","hovercompare"],"showSendToCloud":false},"source":"A","attrs":{"a3cc7cc07029":{"x":{},"y":{},"fill":{},"colour":{},"alpha":{},"text":{},"type":"scatter"}},"cur_data":"a3cc7cc07029","visdat":{"a3cc7cc07029":["function (y) ","x"]},"highlight":{"on":"plotly_click","persistent":false,"dynamic":false,"selectize":false,"opacityDim":0.2,"selected":{"opacity":1},"debounce":0},"shinyEvents":["plotly_hover","plotly_click","plotly_selected","plotly_relayout","plotly_brushed","plotly_brushing","plotly_clickannotation","plotly_doubleclick","plotly_deselect","plotly_afterplot","plotly_sunburstclick"],"base_url":"https://plot.ly"},"evals":[],"jsHooks":[]}</script>
<p>An interactive plot displaying observed host pathogen associations. Facets are produced for antibody based assays and viral detection assays. The rodent species are listed in alphabetical order on the y-axis, detected pathogens are displayed in alphabetical order on the x-axis. Purple tiles represent host-pathogen associations that were not observed for that detection type, yellow tiles represent observed host-pathogen associations. The strength of the colour (the alpha) is scaled to the number of assays performed for that host-pathogen pair. A black line surrounding the tile indicates the pathogen as a known zoonosis. Hovering over the relavent tile will highlight the number of assays and the number of positive samples.</p>
</div>
</div>
</section>
</section>
</section>

</main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>